{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "E:\\covid-19-nlp\n"
     ]
    }
   ],
   "source": [
    "import nltk\n",
    "import os\n",
    "import json\n",
    "import pandas as pd\n",
    "from tqdm import tqdm\n",
    "import string\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.stem.snowball import SnowballStemmer\n",
    "from nltk.tokenize import word_tokenize\n",
    "from nltk.tokenize import sent_tokenize\n",
    "import gensim\n",
    "from gensim import corpora\n",
    "from gensim.corpora import Dictionary\n",
    "import numpy as np\n",
    "fileDir = os.path.dirname(os.path.realpath('__file__'))\n",
    "print(fileDir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "metadata": {},
   "outputs": [],
   "source": [
    "dirs=[\"biorxiv_medrxiv\"]\n",
    "docs=[]\n",
    "\n",
    "\n",
    "for d in dirs:\n",
    "    for file in os.listdir(f\"{d}/{d}/pdf_json\"):\n",
    "        #print(file)\n",
    "        file_path=f\"{d}/{d}/pdf_json/{file}\"\n",
    "        j=json.load(open(file_path,\"rb\"))\n",
    "        full_text=\"\"\n",
    "        abstract=j['abstract']\n",
    "        title=j['metadata']['title']\n",
    "        for text in j['body_text']:\n",
    "            full_text+=text['text']+'\\n\\n'\n",
    "        docs.append([title,abstract,full_text])\n",
    "\n",
    "df=pd.DataFrame(docs,columns=['title','abstract','full_text'])\n",
    "dfinal = pd.DataFrame(docs,columns=['title','abstract','full_text'])\n",
    "\n",
    "new_abstract = [] \n",
    "for value in df[\"abstract\"]: \n",
    "    if (len(value) > 0): \n",
    "        new_abstract.append(value[0]['text'])\n",
    "    else: \n",
    "        new_abstract.append(\"\") \n",
    "       \n",
    "df['abstract'] = new_abstract\n",
    "#df.head()\n",
    "\n",
    "def remove_punctuations(text):\n",
    "    for punctuation in string.punctuation:\n",
    "        text = text.replace(punctuation, '')\n",
    "    return text\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Clean data-lower the dataframe, remove stopwords,tokenized text into sentence then into words\n",
    "xtitleLower = df['title'].to_string(na_rep='').lower()\n",
    "df['title'] = df['title'].str.lower()\n",
    "xtitleLower = df['abstract'].to_string(na_rep='').lower()\n",
    "df['abstract'] = df['abstract'].str.lower()\n",
    "xtitleLower = df['full_text'].to_string(na_rep='').lower()\n",
    "df['full_text'] = df['full_text'].str.lower()\n",
    "stop = stopwords.words('english')\n",
    "stemmer = SnowballStemmer('english')\n",
    "df['title'] = df['title'].apply(lambda x: ' '.join([word for word in x.split() if word not in (stop)]))\n",
    "df['abstract'] = df['abstract'].apply(lambda x: ' '.join([word for word in x.split() if word not in (stop)]))\n",
    "df['full_text'] = df['full_text'].apply(lambda x: ' '.join([word for word in x.split() if word not in (stop)]))\n",
    "df[\"title\"] = df['title'].apply(remove_punctuations)\n",
    "df[\"abstract\"] = df['abstract'].apply(remove_punctuations)\n",
    "#df[\"full_text\"] = df['full_text'].apply(remove_punctuations)\n",
    "df['title'] = df['title'].str.replace('\\d+', '')\n",
    "df['abstract'] = df['abstract'].str.replace('\\d+', '')\n",
    "df['full_text'] = df['full_text'].str.replace('\\d+', '')\n",
    "#df['stemmed'] = df['unstemmed'].apply(lambda x: [stemmer.stem(y) for y in x]) # Stem every word.\n",
    "#df = df.drop(columns=['unstemmed'])\n",
    "#print(df.head())\n",
    "arr = df.to_numpy()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {},
   "outputs": [],
   "source": [
    "dirs=[\"biorxiv_medrxiv\"]\n",
    "docs=[]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "for d in dirs:\n",
    "    for file in os.listdir(f\"{d}/{d}/pdf_json\"):\n",
    "        #print(file)\n",
    "        file_path=f\"{d}/{d}/pdf_json/{file}\"\n",
    "        j=json.load(open(file_path,\"rb\"))\n",
    "        full_text=\"\"\n",
    "        abstract=j['abstract']\n",
    "        title=j['metadata']['title']\n",
    "        for text in j['body_text']:\n",
    "            full_text+=text['text']+'\\n\\n'\n",
    "        docs.append([title,abstract,full_text])\n",
    "\n",
    "df=pd.DataFrame(docs,columns=['title','abstract','full_text'])\n",
    "dfinal = pd.DataFrame(docs,columns=['title','abstract','full_text'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [],
   "source": [
    "new_abstract = [] \n",
    "for value in df[\"abstract\"]: \n",
    "    if (len(value) > 0): \n",
    "        new_abstract.append(value[0]['text'])\n",
    "    else: \n",
    "        new_abstract.append(\"\") \n",
    "       \n",
    "df['abstract'] = new_abstract\n",
    "#df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [],
   "source": [
    "def remove_punctuations(text):\n",
    "    for punctuation in string.punctuation:\n",
    "        text = text.replace(punctuation, '')\n",
    "    return text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [],
   "source": [
    "#tokenization\n",
    "for i in range(len(arr)):\n",
    "    for j in range(len(arr[i])):\n",
    "        arr[i][j] = word_tokenize(arr[i][j])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [],
   "source": [
    "arrayDictionaries = np.empty(len(arr), dtype=object)\n",
    "result = []\n",
    "for i in range(len(arr)):\n",
    "    result1 = arr[i][0] + arr[i][1] +arr[i][2] \n",
    "    arrayDictionaries[i] = np.array(result1)  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {},
   "outputs": [],
   "source": [
    "#build the dictionary\n",
    "dictionary = gensim.corpora.Dictionary(arrayDictionaries)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "#create corpus \n",
    "corpus = [dictionary.doc2bow(arrayDictionary) for arrayDictionary in arrayDictionaries]\n",
    "tf_idf = gensim.models.TfidfModel(corpus)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Similarity index with 2278 documents in 0 shards (stored under E:\\covid-19-nlp/Index)\n",
      "<class 'gensim.similarities.docsim.Similarity'>\n"
     ]
    }
   ],
   "source": [
    "# building the index\n",
    "sims = gensim.similarities.Similarity(fileDir+'/Index',tf_idf[corpus],num_features=len(dictionary))\n",
    "#print to check that the sims were created\n",
    "print(sims)\n",
    "print(type(sims))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of documents: 1\n"
     ]
    }
   ],
   "source": [
    "file2_docs = []\n",
    "\n",
    "with open ('Queries.txt') as f:\n",
    "    tokens = sent_tokenize(f.read())\n",
    "    for line in tokens:\n",
    "        file2_docs.append(line)\n",
    "\n",
    "print(\"Number of documents:\",len(file2_docs))  \n",
    "for line in file2_docs:\n",
    "    query_doc = [w.lower() for w in word_tokenize(line)]\n",
    "    query_doc_bow = dictionary.doc2bow(query_doc) #update an existing dictionary andcreate bag of words"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Comparing Result: [0. 0. 0. ... 0. 0. 0.]\n"
     ]
    }
   ],
   "source": [
    "query_doc_tf_idf = tf_idf[query_doc_bow]\n",
    "# print(document_number, document_similarity)\n",
    "print('Comparing Result:', sims[query_doc_tf_idf])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3.8611813\n"
     ]
    }
   ],
   "source": [
    "sum_of_sims =(np.sum(sims[query_doc_tf_idf], dtype=np.float32))\n",
    "print(sum_of_sims)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.65469474\n"
     ]
    }
   ],
   "source": [
    "#the maximum value\n",
    "print(np.amax(sims[query_doc_tf_idf]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[0.108917   0.11822368 0.12335537 0.14555898 0.14591417 0.15535428\n",
      " 0.18939789 0.1907245  0.22733808 0.65469474]\n"
     ]
    }
   ],
   "source": [
    "#the top 10 similarities\n",
    "print (sims[query_doc_tf_idf][np.argsort(sims[query_doc_tf_idf])[-10:]])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 891  835  846 1407  933  864  799  789  615 1112]\n"
     ]
    }
   ],
   "source": [
    "#return the id of the top 10 similarities\n",
    "print( np.argsort(sims[query_doc_tf_idf])[-10:])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The 1st best search\n",
      "Title : not Mentioned\n",
      "There is an ongoing pandemic of viral pneumonia called Coronavirus Disease 2019 which is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection. 44\n",
      "\n",
      "At the time of this writing, the disease has been reported to affect 634,835 people in more than 45 200 countries, territories or areas, and cause 29,891 deaths (1). Understanding the clinical and 46 epidemiological characteristics of the disease is important for informing public health decision 47 making, which would enable improvement of surveillance and effective planning of treatment. In in China and in other parts of the world (3-12). However, most of these reports were limited by a 52 small sample size, and the characteristics reported appeared to be inconsistent. For example, in 53 a report involving 99 patients, Chen et al.\n",
      "\n",
      "(3) noted a much higher proportion of men than 54 women, and suggested that men were generally more susceptible to SARS-CoV-2 infection. On 55 the other hand, Shi et al. (4) found that the male-to-female ratio among the 81 patients included 56 was close to 1:1, indicating that both genders were equally susceptible to 57 the World Health Organization (WHO) has declared COVID-19 a global pandemic and the 58 contagion shows no sign of slowing down (13). Thus, it is a timely prompt to obtain a more 59 precise understanding of the disease by combining all available data in the literature. In this 60 study, a systematic review and pooled analysis was performed to characterize the clinical and 61 epidemiological features of COVID-19 patients. 62\n",
      "\n",
      "Three separate searches were performed on PubMed database on March 2, 2020, using the 65 keywords \"COVID-19\", \"2019-nCoV\" and \"SARS-CoV-2\". No language restriction was applied. 66\n",
      "\n",
      "After removal of duplicated records, screening by title and abstract was performed to identify 67 potentially relevant studies. The full-texts of these potentially relevant studies were then 68 evaluated. The reference lists of these studies were also hand-searched to identify additional 69 records. Studies were included if they reported any clinical and/or epidemiological data of 70 confirmed COVID-19 patients, regardless of their study design. However, review papers or 71 studies employing secondary analysis of the previously available data were excluded from the 72 analysis. In case of overlapping studies, the ones which the largest sample size or the most 73 complete data set were included. 74 75\n",
      "\n",
      "The following data were extracted from the included studies: name of first author, country, date 77 of diagnosis, demographic data, smoking status, comorbidities, signs and symptoms, 78 laboratory/biochemical data, CT findings, and complications of the disease. A simple pooling of 79 data was performed to provide an overall summary of the clinical and epidemiological 80 characteristics of the patients. All data are reported as absolute number and/or mean. As 81 patient-level data were not available in majority of the studies, standard deviation could not be 82 calculated. 83 84\n",
      "\n",
      "There was no funding source for this study. The corresponding author had full access to all the 86 data in the study and had final responsibility for the decision to submit for publication. 87\n",
      "\n",
      "The flowchart of study selection is depicted in Figure 1 . A total of 644 records were identified 89 showed decreased and increased leukocyte counts. Besides, 51.39% of the patients had a 114 decreased lymphocyte count and 9.63% had a decreased platelet count. Increased levels of 115 aspartate aminotransferase, creatinine, creatine kinase, lactate dehydrogenase, hypersensitive 116 troponin I, procalcitonin and C-reactive protein were observed in 27. 45%, 5.19%, 13.25%, 117 45.76%, 3.57%, 69.08% and 72.30% of the patients, respectively. 118 119 Analysis of computed tomography (CT) data revealed that most of the patients (78.50%) 120 had bilateral lung involvements, while 15.65% had unilateral lung involvements and 5.86% 121 showed no sign of viral pneumonia. In 41.18% of the patients, all five lung lobes were affected. 122\n",
      "\n",
      "On the other hand, as much as 17.03%, 11.46%, 12.69% and 17.65% patients respectively had 123 one, two, three and four lobes affected. 124 125 It was unknown how many patients had complications following SARS-CoV-2 infection. 126\n",
      "\n",
      "Nonetheless, among the wide spectrum of complications observed, acute respiratory distress 127 syndrome (ARDS) was most frequently documented (28.36%), followed by acute cardiac injury 128 (7.89%) and acute kidney injury (7.60%). 129 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 Discussion 130 COVID-19 poses a significant burden on the healthcare system all over the world. A complete 131 understanding of the characteristics of the disease is important for effective surveillance and 132 public health response measures to be implemented in a timely manner. Currently, although we 133 have some basic understanding of the clinical and epidemiological features of COVID-19 134 patients, our knowledge is insufficient. This is because inconsistencies still exist in the findings 135 of many published reports, and the sample sizes in most of these reports were too small for a 136 reliable summary to be made. In this work, a systematic review and pooled analysis was 137 performed to combine data from 69 previous reports, in order to yield a more accurate summary 138 of the clinical and epidemiological characteristics of COVID-19 patients. 139\n",
      "\n",
      "In many instances, susceptibility to viral infections may be related to factors such as 141 gender and smoking habits (72-74). For the former, it is believed that X chromosome 142 inactivation in females may cause cellular mosaicism which ensures the presence of at least 143 one functional copy of X-linked immune genes, thus conferring women an increased resistance 144 against viral infections (75). In addition, estrogen, the major female sex hormone, is known to 145 promote adaptive immune response (76), while testosterone, the primary sex hormone in men, 146 could contribute to the suppression of the innate immune response, rendering men more 147 susceptible to viral infections (77, 78). On the other hand, cigarette smoking may reduce the 148 level of circulating immunoglobulins, immune cells, and pro-inflammatory cytokines, as well as 149 disrupt the response of antibodies to antigens (72). For these reasons, some studies have 150 suggested that men and smokers are more susceptible to SARS-CoV-2 infections (3, 79). In the 151 present work, we noted that the ratios of male to female and smokers to non-smokers were 152 close to 1:1. Although the relative risk or odds ratio of the association between these variables 153 and SARS-CoV-2 infection could not be computed due to the lack of a comparison group, a 154\n",
      "\n",
      "proportion of approximately 1:1 suggests that susceptibility to SARS-CoV-2 infection is universal. 155 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "The present work also showed that the symptoms of COVID-19 were generally non-157 specific, thus the disease cannot be reliably distinguished from other infectious diseases based 158 on the symptoms alone. The most commonly observed symptoms were similar to those of the 159 previous coronavirus disease outbreaks (MERS and SARS), i.e. fever, cough, and myalgia or 160 fatigue. Nevertheless, compared to MERS and SARS, the proportion of afebrile patients was 161 much higher in COVID-19 (16.79% cf. 2% in MERS and 0-1% in SARS) (80). This indicates that 162 a significant number of COVID-19 patients would be missed if surveillance and monitoring 163 systems focus largely on temperature screening, as commonly practiced in airports (81, 82). 164\n",
      "\n",
      "The present work also showed that dyspnea was observed in only 16.97% of the patients. This 165 contradicts with advisories and guidelines published by many health authorities, which suggest 166 that dyspnea is a commonly observed symptom in . Besides, it was observed 167 that 0.80% of the patients were asymptomatic. Currently, whether asymptomatic patients can 168 transmit the virus to other individuals is not fully known, but it is highly possible (35). Thus, 169 although the number of asymptomatic patients was low, identifying and isolating such patients 170 to prevent uncontrolled disease spread would prove very challenging. It is therefore important 171 for diagnostic tests to be performed on asymptomatic medium-and high-risk individuals to 172 facilitate early detection and prevention of SARS-CoV-2 transmission. In addition to nucleic acid 173 testing using real-time reverse transcription polymerase chain reaction, some studies have 174 suggested the potential usability of chest CT for COVID-19 diagnosis (90, 91). However, in the 175 present work, we found that 5.86% of the patients did not have abnormalities on CT scans. Thus, 176 although CT findings have substantial accuracy in identifying the disease (90, 91), the results need 177 to be interpreted with caution. 178 179 SARS-CoV-2 is known to infect a cell by first binding to its angiotensin converting 180 enzyme 2 (ACE2) receptor (92). Apart from the lung, high expression of ACE2 receptor is also 181 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 observed in several other organs such as the heart, kidney, and intestine, as well as the in 182 lymphocytes (93, 94) . Several previous studies reported that a decreased lymphocyte count 183 was a common feature of SARS-CoV-2 infection (2, 3, 7, 30, 41) . In the present work, 184 lymphopenia was observed in 51.39% of the patients. A decreased lymphocyte count implies a 185 weakening adaptive immune system. Considering the high expression of ACE2 in lymphocytes, 186\n",
      "\n",
      "it has been postulated that SARS-CoV-2 may directly infect and attack lymphocytes, thus 187 impairing the immune system (95). Besides the decrease in lymphocyte count, the present 188 study also showed that many COVID-19 patients had increased levels of C-reactive protein, 189 creatine kinase, lactate dehydrogenase and procalcitonin. High levels of C-reactive protein and there is a possibility that some overlapping patients were not removed from our analysis and 204 their characteristics were overreported. Besides, since patient-level data were not reported in 205 most of the studies, median values and standard deviations, which understandably provide 206 more meaningful information, could not be computed. Finally, as mentioned above, various 207 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.02.20050989 doi: medRxiv preprint comparisons among the patients (e.g. severe vs. mild, and death vs. survivor) could not be 208 analyzed due to insufficient data available. 209 210\n",
      "\n",
      "In conclusion, this report has successfully provided a more complete picture of the clinical and 212 epidemiological characteristics of COVID-19 patients. A wide variation exists in the clinical 213 manifestation of the disease. As the outbreak continues to escalate and SARS-CoV-2 continues 214 to mutate (97), it is important to consistently update the characteristics of the patients in order to 215 monitor whether evolving strains of the virus could cause the disease differently. Sharing of 216 clinical and epidemiological data among the scientific community is highly important for 217 informing public health decision making for controlling the spread of the disease. 218\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 Contributor 219\n",
      "\n",
      "SCT contributed solely to this work. 220 221\n",
      "\n",
      "The author declares no competing interests. 223 224\n",
      "\n",
      "No funding was obtained for this work. 226 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "multicenter study]. Zhonghua gan zang bing za zhi = Zhonghua ganzangbing zazhi = 325\n",
      "\n",
      "Chinese journal of hepatology. 2020;28 (2) . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.02.20050989 doi: medRxiv preprint 494 Figure 1 : Flow chart of study selection 495 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 2nd best search\n",
      "Title : Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV\n",
      "In early December 2019, several cases of pneumonia with unknown causes were reported in Wuhan, Hubei, China (1) . Later, a novel coronavirus, 2019-nCoV was identified based on sequencing of the patient respiratory tract samples. As of February 1, 2020, more than 10,000 confirmed patients and more than 15,000 suspected cases were reported in China, with additional patients being identified in a rapidly growing number internationally. Analyzing of the genome of 2019-nCoV showed that this new virus shared around 80~90% sequence identity to SARS-CoV (2) . Both bioinformatics modeling and in vitro experiments indicated that 2019-nCoV was more likely to use the angiotensin-converting enzyme 2 (ACE2) as the entry receptor (3, 4) . ACE2 is previously known as the receptor for SARS-CoV and HCoV-NL63 (5, 6) . Considering the sequence of 2019-nCoV is similar to the SRAS-CoV and they are reported to share the same molecular as the entry point, analyses of ACE2 expression and distribution in lung and related biological processes may help us understand the pathogenesis and design therapeutic strategies for 2019-nCoV. Recent advances in bioinformatics enable researchers to reveal the underlying mechanisms of various diseases. In this study, we used bioinformatics approaches to identify the ACE2 expression features in lung and mine associated regulating networks. These results may help us understand the pathogenesis and design therapeutic strategies for 2019-nCoV.\n",
      "\n",
      "Among the 2019-nCoV infected patients, there are many patients with underlying diseases, such as chronic respiratory disease and cardiovascular disease. Therefore, it is important to evaluate whether patients with underlying diseases are more susceptible to coronavirus infection than healthy population. Firstly, we analyzed the expression of ACE2 in lung in the different populations. The expression level of ACE2 was not significantly altered between healthy populations and patients with chronic respiratory diseases including chronic obstructive pulmonary diseases (COPD) and asthma ( Figure 1A , 1B, and 1C, p-value > 0.05). It indicated that 2019-nCoV . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.03.20020206 doi: medRxiv preprint could have the same ability to bind the respiratory tract epithelial cells of healthy population and patients with chronic respiratory diseases through ACE2. Thus, there may be no significant difference in the susceptibility of 2019-nCov infection between healthy population and patients with chronic respiratory diseases.\n",
      "\n",
      "Secondly, we found that the level of ACE2 expression was markedly upregulated in long-term smokers ( Figure 1D , p-value < 0.05). However, the p-value between baseline and post-acute smoke exposure (ASE) group (24 hours after smoking 3 cigarettes) was 0.073 ( Figure 1E ). It was assumed that the long-term smoking could elevate the expression of ACE2 and thus, increase the susceptibility to 2019-nCoV infection.\n",
      "\n",
      "Thirdly, we analyzed the expression of ACE2 in airway epithelial cells after being infected with SARS-CoV. The result suggested that 24 hours after SARS-CoV infection, the expression of ACE2 dramatically increased compared to 12 hours. After 48 hours, the expression of ACE2 remained at a high level. This indicated that ACE2 not only played a critical role in viral susceptibility but also may be involved in post-infection regulation.\n",
      "\n",
      "To investigate the virus-related potential biological processes associated with ACE2, we extracted the expression profiles of 15 healthy non-smokers from dataset GSE89809 and divided them into two groups: high and low-expression level of ACE2.\n",
      "\n",
      "We performed GSEA (Gene Set Enrichment Analysis) and found that the expression of ACE2 was mainly associated with innate and acquired immune responses, regulation of B cell mediated immunity, cytokine and IL-1，IL-10，IL-6，IL-8 secretion ( . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint . https://doi.org/10.1101/2020.02.03.20020206 doi: medRxiv preprint\n",
      "\n",
      "To further explore whether above activities could be triggered after coronavirus infection, we analyzed the expression profiles of epithelial cells which was infected by SARS-CoV. The biological process scores were obtained using GSVA (Gene Set Variation Analysis) and the pearson correlation between process scores and ACE2 expression was calculated. Similarly, while ACE2 increased after infection, production of cytokines, IL-1, IL-10 and IL-6 also increased, and the regulation of B cell activation was triggered to response. After 48 hours, virus activities such as viral entry into the host cell, virus life cycle, and viral transcription were enhanced. Besides, the T-cell cytokine secretion was increased, and T-cell activation was promoted ( Figure 2B ). Then, we used ssGSEA (Single Sample Gene Set Enrichment Analysis) to quantify the immune infiltrates in healthy people epithelial cells and cells after infection. We found that in normal samples, the immune cells were not activated so that the correlation between infiltration and ACE2 expression was not significant ( Figure 2C ). However, in the SARS-CoV infected cells, the ACE2 was significantly correlated with neutrophils, NK cells, Th17 cells, Th2 cells, Th1 cells, DC ( Figure   2D ).\n",
      "\n",
      "Considering higher expression of ACE2 was involved in mediating inflammatory responses, immune activities, and viral replication in both healthy individuals and infected cells, we further explored protein-protein regulation network in viral activity-related proteins and cytokine secretion, respectively. We extracted the viral activity-associated and cytokine secretion-associated proteins from the Gene ontology (GO) biological process gmt file. Construction of the Protein-protein interaction (PPI) showed that these proteins were tightly interactive among each other and we here exhibited the most significant PPI. In terms of viral activity, we found that \"viral gene expression\" proteins such as RPS3, RPS8 and PRS25 were the most important genes in the PPI network. In addition, VCP, LARP1, UBA52, PRKN, EIF3A and EIF3L\n",
      "\n",
      "were also important in the network ( Figure 3A ). In terms of the cytokine secretion associated proteins, we found that proteins such as SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1 and PRKCD were Hub proteins in the regulating network of . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint . https://doi.org/10.1101/2020.02.03.20020206 doi: medRxiv preprint cytokine secretion after coronavirus infection ( Figure 3B ), indicating that these molecules were critically important in ACE2-induced inflammatory response. Previous studies have found that ACE2 was related to the severity of acute respiratory syndrome induced by coronavirus infection, which regulated the . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint . https://doi.org/10.1101/2020.02.03.20020206 doi: medRxiv preprint production of acute respiratory distress syndrome-associated cytokines (7) . ACE2 is also related to adaptive immune responses (8) . In this current study, the GSEA analysis showed that the high expression of ACE2 was related to innate immune response, acquired immune response, B cell regulatory immunity and cytokine secretion, and enhanced the inflammatory response induced by IL-1, IL-10, IL-6, IL-8 cytokines. We speculated that the immune system dysfunction involved in the high expression of . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint . https://doi.org /10.1101 /10. /2020 In order to further analyze the core genes involved in viral replication and inflammatory response mediated by ACE2, we constructed a protein-protein interaction network. It was found that RPS3 played a key role in viral replication, and SRC non-receptor protein kinase was the hub gene in inflammatory response. It has been shown that RPS3 plays an important role in cell mitosis by interacting with αtubulin. When cells are infected with HIV-1, HIV-1 Tat protein can interact with RPS3 to disturb its localization on the spindle, and ultimately inhibit cell proliferation (10) .\n",
      "\n",
      "Moreover, the function of RPS3 depends on its own nuclear translocation, which is regulated by protein kinase like SRC (11) . We speculated that the increased expression of ACE2 affected RPS3 and SRC, which were the two hub genes involved in viral replication and inflammatory response.\n",
      "\n",
      "In conclusion, 2019-nCoV infection is a serious public health problem, endangers human health seriously. Due to the strong human to human infectivity, it needs to be further studied. Our current study found that there was no significant difference in Note all results are based on data mining, thus, further experiments are needed.\n",
      "\n",
      "Six independent studies (GSE37758, GSE73395, GSE89809, GSE63127, GSE97010, and GSE14700), covering healthy volunteers, COPD patients, asthma patients, smoking individuals were collected from GEO data repository (https://www.ncbi.nlm.nih.gov/geo/). The samples consist of lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small airway epithelial cells, and SARS infected cells.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.03.20020206 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.03.20020206 doi: medRxiv preprint\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 3rd best search\n",
      "Title : No evidence that androgen regulation of pulmonary TMPRSS2 explains sex-discordant COVID-19 outcomes\n",
      "The December 2019 outbreak caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in Wuhan, China, has led to the coronavirus disease 2019 pandemic Zhou et al., 2020) . The highly contagious nature of the disease as well as the lack of vaccines or clinically approved treatments has caused a worldwide public health emergency. Therefore, improved and timely understanding of the human susceptibilities to SARS-CoV-2 will prove invaluable in controlling the pandemic and treatment of those affected.\n",
      "\n",
      "SARS-CoV-2 enters cells using two host cellular proteins: angiotensin converting enzyme-2 (ACE2) and transmembrane serine protease 2 (TMPRSS2). The virus first employs ACE2 as a cell entry protein, followed by TMPRSS2-mediated proteolytic processing of the SARS-2 spike protein, further facilitating viral entry (Hoffmann et al., 2020) . Targeting the activity or expression of both factors by a plethora of approaches has been proposed as potential treatment (Hoffmann et al., 2020; Stopsack et al., 2020; Zhang et al., 2020) .\n",
      "\n",
      "TMPRSS2 is also a widely studied androgen-regulated gene in prostate tissue, contributing to prostate cancer pathogenesis by way of aberrantly driving oncogene expression.\n",
      "\n",
      "Approximately half of all prostate cancers harbor a fusion that juxtaposes a TMPRSS2 transcriptional regulatory element, which is stimulated by potent androgens and the androgen receptor (AR), in front of an ERG oncogene. (Tomlins et al., 2005) . The end result is AR stimulation of oncogene expression.\n",
      "\n",
      "This raises the possibility that the physiological roles of androgens may, at least partially, account for the sex-specific clinical outcomes (Grasselli et al., 2020; Sharon Moalem, 2020) . Due to its androgen-regulated nature in the prostate and its essential role in SARS-Cov-2 etiology, TMPRSS2 expression has been postulated to follow a similar pattern of regulation in pulmonary cells by the potent androgens testosterone and dihydrotestosterone (Mikkonen et al., 2010) . If this link proves correct, it could pave the path to novel strategies, including re-purposing of FDA-approved potent androgen synthesis inhibitors or AR antagonists, such as enzalutamide (Enz) and apalutamide, for the treatment of COVID-19.\n",
      "\n",
      "These strategies have been the subject of clinical trial discussions (Sharifi and Ryan, 2020; Stopsack et al., 2020) .\n",
      "\n",
      "Here, we show that there is no evidence for increased TMPRSS2 expression in the lungs of males. On the other hand, cigarette smoking increases the levels of pulmonary TMPRSS2 mRNA, which drop back to never-smoker levels after quitting smoking. We provide in vivo evidence that neither mRNA nor protein levels of TMPRSS2 vary by sex. In author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint addition, AR-antagonist therapy with Enz does not suppress pulmonary TMPRSS2 expression in eugonadal male mice. Interestingly, despite similar transcript levels, we found AR protein levels are substantially higher in lungs of male mice compared with females. This finding, conditional upon validation in human samples, may yet help explain sex differences in COVID-19 outcomes, but likely in a manner that is independent of TMPRSS2 regulation.\n",
      "\n",
      "TMPRSS2 and AR transcript expression in human lung are not higher in males than in females Considering the poorer clinical outcome of COVID-19 in men, underlying androgen-related causes are suspected but not presently known. The SARS-Cov-2 co-receptor, TMPRSS2\n",
      "\n",
      "harbors an AR-responsive enhancer that is induced by male gonadal hormones in prostate tissue (Wang et al., 2007) , raising the possibility of a similar mode of regulation in the respiratory system. Furthermore, certain pulmonary disease process outcomes, including asthma, are sex steroid associated (Zein et al., 2020) . Therefore, we sought to investigate whether male sex was associated with higher expression of TMPRSS2 or AR in human lung.\n",
      "\n",
      "To this end, we acquired the publicly available expression datasets for TMPRSS2 and AR in non-cancerous lung and associated respiratory tissues from the Genomic Expression Omnibus (GEO). Across all tissue-type comparisons, we found no evidence for elevated TMPRSS2 or AR mRNA expression ( Fig.1 ) in males compared with females. Together, these results suggest that worse clinical manifestations of men with COVID-19 are unlikely to be associated with TMPRSS2. However, this conclusion is limited to transcriptional information, and further studies are required to rule out the possibility of protein level differences in the human respiratory system.\n",
      "\n",
      "To further explore sex differences in TMPRSS2 expression in animal models, we harvested lungs from female and male mice treated with control diet or Enz for > 10 days and analyzed them for protein and mRNA expression. In the case of TMPRSS2, this yielded results consistent with human expression data: i.e., there were no sex-specific changes. Additionally, Enz treatment did not downregulate TMPRSS2 ( Fig. 2A and B) . Given the unchanged AR protein levels (despite a modest but statistically significant mRNA increase) in Enz-treated males, we infer that TMPRSS2 is not regulated by AR in the lung. Nevertheless, further author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint single-cell level experiments are needed to explore the expression pattern of TMPRSS2 in cellular subsets rather than in bulk lung and respiratory tract.\n",
      "\n",
      "Similar to TMPRSS2 and also consistent with human data, we found no evidence of sexspecific changes in AR transcript levels of mice ( Fig. 2A) . However, the lungs of male mice express considerably higher amounts of AR protein compared with females ( Fig. 2B ). This finding begs the question whether human lungs also follow a sex-discordant pattern of AR protein synthesis or if this is a mouse-specific phenomenon. If the former, androgen-ARmediated changes may yet explain sex-specific differences in COVID-19 outcomes by TMPRSS2-independent mechanisms.\n",
      "\n",
      "In addition to male sex, smoking is a risk factor for COVID-19 susceptibility and poor clinical outcomes . One recent study of 1,099 COVID-19-positive patients reported a more than two-fold increased risk for intensive care unit admission and death in smokers as compared with non-smokers (Guan et al., 2020) . We identified human expression GEO datasets of bronchial/airway epithelial cells containing subject smoking status and asked whether smoking is associated with TMPRSS2 expression. Our analysis indicated a consistent pattern where expression of both TMPRSS2 (Fig. 3A ) and the primary SARS-CoV-2 receptor ACE2 ( Fig.3B ) was modestly but significantly increased in smokers compared with non-smokers. Interestingly, the levels were downregulated to never-smoker levels in former smokers. The results of our analysis are in keeping with several recent reports on ACE2 and smoking (Brake et al., 2020; Leung et al., 2020; Smith and Sheltzer, 2020) .\n",
      "\n",
      "Although the p-values range widely across different data sets, almost all the data sets show changes in a consistently increased direction for current smokers including multiple data sets with small p-values ( Fig. 3A and B ).\n",
      "\n",
      "For both TMPRSS2 (Fig. 3C) and ACE2 (Fig. 3D) , there was no correlation between smoking pack years and mRNA expression in both current and former smokers, suggesting that the change does not build up over time but is instead a rapid process -akin to a switch.\n",
      "\n",
      "Sex-associated clinical outcomes have been long observed in a variety of infectious and inflammatory conditions. Sex steroids (i.e., androgens and estrogens) are possible mediators author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint of these biologic differences. For COVID-19, potential androgen-mediated biologic differences include 1) TMPRSS2 regulation and 2) immune modulation (Sharifi and Ryan, 2020; Stopsack et al., 2020) . Our study addresses the first of these possibilities.\n",
      "\n",
      "Although AR protein is clearly expressed in the male mouse lung at levels higher than in females, we find no evidence for androgen regulation of TMPRSS2. This evidence includes 1) the absence of any TMPRSS2 increase in male compared with female human lung 2) no TMPRSS2 increase in male compared with female mouse lung 3) no evidence for TMPRSS2 suppression with next generation AR antagonist treatment. Therefore, given that AR protein is higher in male lungs, it is possible that AR-dependent transcription in the male lung mediates biologic differences independent of TMPRSS2 expression that are important for driving sex-dependent differences in clinical outcomes. This has yet to be determined.\n",
      "\n",
      "Limitations of our study include the fact that we cannot rule out cell type-specific differences in sex-associated or androgen-dependent TMPRSS2 regulation in pulmonary tissues. Nevertheless, early autopsy studies of COVID-19 appear to show diffuse viral damage that may suggest that the expression of viral host cell receptors is likely not limited to an isolated pulmonary subset of cells (Barton et al., 2020) . Although we did perform immunohistochemical staining studies for TMPRSS2 expression in the mouse lung, we encountered different patterns compared to human prostate tissue, in which TMPRSS2 is a membrane/cell surface protein. The mouse lung lacked cytoplasmic and membrane staining but expressed significant nuclear stain. This led us to conclude that much of the antibody binding in mouse was non-specific.\n",
      "\n",
      "The possibility of suppressing androgen-regulated TMPRSS2 expression as a required SARS-CoV-2 host protein was an attractive hypothesis that may also be tested with clinical trials of next-generation anti-androgens (Sharifi and Ryan, 2020; Stopsack et al., 2020) .\n",
      "\n",
      "However, our data do not support this hypothesis and we would urge caution in prioritizing COVID-19 clinical trials.\n",
      "\n",
      "Sexually dimorphic AR expression in an organ not associated with sexual differentiation -the male lung -raises the question of function. This is reminiscent of sexually dimorphic AR protein expression in the male human kidney, in which AR function includes regulating glucocorticoid metabolism and downstream steroid receptor activity . Whether this physiology also occurs in the lung has yet to be determined.\n",
      "\n",
      "In conclusion, we find no evidence for androgen regulation of TMPRSS2 in the male lung. Therefore, TMPRSS2 regulation in the lung appears to fundamentally differ from a author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint clear androgen-dependent effect in prostatic tissues. Pulmonary TMPRSS2 regulation appears not to account for the sex-discordance in COVID-19 clinical outcomes. In contrast, both TMPRSS2 and ACE2 appear to have increased expression in current smokers. It is not yet known whether these changes in expression have a functional impact on COVID-19 infection, but if they do, the results suggest smokers could partly mitigate their increased risk by quitting smoking. This also raises the question of whether other hazards such as high levels of air pollution could have similar effects. Nevertheless, a recent study reports that ACE2 expression is not upregulated by lung disease or exposure to carcinogens (Smith and Sheltzer, 2020) . Together, these data suggest that induction of ACE2, and perhaps TMPRSS2, is mediated by specific types of tissue injury.\n",
      "\n",
      "Mouse studies were performed under a protocol approved by the Institutional Animal Care and Use Committee (IACUC) of the Cleveland Clinic Lerner Research Institute. A cohort of adult NSG mice (> 6 weeks old) were obtained from Cleveland Clinic Biological Resources Unit. The male mice were arbitrarily divided between two groups receiving control chow or Enz diet 62.5mg/kg. Following 11 days on diet, the mice were sacrificed using a lethal dose of Nembutal followed by cardiac puncture. Once sacrificed, the abdominal and thoracic cavities of the mice were opened, the inferior vena cava was cut, and the lungs were gently perfused with warm saline via the right ventricle. Next, the lungs were removed, and the individual lobes were either fixed with 10% formalin for paraffin embedding or snap frozen for subsequent RNA or protein analysis.\n",
      "\n",
      "Approximately 40-50 mg freshly frozen lung was added to soft tissue homogenizing CK14 tubes (Betin Technologies) with 200 ml RIPA buffer containing HALT protease and phosphate inhibitor cocktail. Lung tissues were then homogenized with a homogenizer (Minilys, Betin Technologies) three times (60 s each time) at the highest speed, with 5-10 minute intervals on ice to cool lysates. The lysates were then centrifuged for 15 min at 16,000\n",
      "\n",
      "x g and the supernatants were collected for immunoblot analysis with antibodies for TMPRSS2 (Abcam: ab92323 and Proteintech: 14437-1-AP), AR (EMD Millipore: PG-21 and Santa Cruz Biotechnology: N-20) and GAPDH (D16H11). author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint Total RNA was harvested by homogenizing 25 mg lung tissue in 350 µl RLT buffer (RNeasy kit, Qiagen) following the manufacturer's instructions. cDNA synthesis were then carried out with the iScript cDNA Synthesis Kit (Bio-Rad). Quantitative PCR (qPCR) analysis was conducted in triplicate in an ABI 7500 Real-Time PCR machine (Applied Biosystems) using iTaq Fast SYBR Green Supermix with ROX (Bio-Rad) with the following primer sets:\n",
      "\n",
      "forward: 5'-gtcatccacacacatcccaagtc-3' reverse: 5'-tcccagaacctccaaagcaaga-3'\n",
      "\n",
      "Ar forward: 5'-ggcagcagtgaagcaggtag-3' reverse: 5'-cggacagagccagcggaa-3'\n",
      "\n",
      "forward: 5'-gacctccttcttccaggctttg -3' reverse: 5'-ctcccaccttgtctccagtcttta-3'.\n",
      "\n",
      "The public genomics data repository Gene Expression Omnibus (GEO, ncbi.nlm.nih.gov/geo) was searched for data sets containing expression profiling of samples from non-cancerous human lung and bronchial/airway epithelial cells with samples identified by gender and/or smoking status of subjects. The following data sets were identified: GSE994 (airway epithelial cells from current/former/never smokers), GSE4115 (histologically normal bronchial epithelial cells from smokers with and without lung cancer), GSE7895 (airway epithelial cells from current/former/never smokers), GSE16008 (bronchial epithelial cells from healthy current and never smokers), GSE18385 (large and small airway epithelial cells from healthy current and never smokers), GSE37147 (bronchial epithelial cells from current and former smokers with and without COPD), GSE43696 (bronchial epithelial cells from asthma patients and healthy controls), GSE63127 (small airway epithelial cells from healthy current and never smokers), GSE103174 (lung tissue from smokers and nonsmokers with and without COPD), and GSE123352 (non-involved lung parenchyma from ever and never smokers with lung adenocarcinoma). TMPRSS2, AR, and ACE2 gene expression values were obtained from each data set and analyses for comparisons between groups (for each data set for which gender and/or current smoking status information was available) were performed using R.\n",
      "\n",
      "author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint (normalized so that the mean within each data set equals 1) from publicly available GEO data sets. N for each data set: GSE16008 = 13 current smoker/13 never smoker; GSE18385 large airway = 32 current smoker/20 never smoker; GSE18385 small airway = 58 current smoker/51 never smoker; GSE37147 = 99 current smoker/139 former smoker; GSE61327 = 112 current smoker/71 never smoker; GSE7895 = 52 current smoker/31 former smoker/21 author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint never smoker; GSE994 = 34 current smoker/18 former smoker/23 never smoker. Data sets 18385, 63127, 7895, and 994 contained multiple TMPRSS2 reference sequences and the expression values of the different sequences were added together within each data set. In all of the data sets that contained four TMPRSS2 reference sequences (18385 both large and small airway and 63127), the difference between groups was largest for reference sequence AI660243, so expression using that reference sequence alone is also shown for those data sets. For data sets with two groups, p-values from t-tests are shown. For data sets with three groups, Tukey HSD p-values are shown that were obtained after one-way ANOVA (ANOVA expression and smoking pack years. Adjusted R-squared and p-values from linear fits for each data set: GSE16008 R 2 = 0.03 and p = 0.19; GSE18385 large airway R 2 = -0.03 and p = 0.76; GSE18385 small airway R 2 = -0.01 and p = 0.41; GSE37147 (current smokers) R 2 = -0.01 and p = 0.67; GSE37147 (former smokers) R 2 = 0.001 and p = 0.30; GSE7895 (current smokers) R 2 = 0.01 and p = 0.24; GSE7895 (former smokers) R 2 = -0.02 and p = 0.60. author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.21.051201 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allo\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 4th best search\n",
      "Title : Integrated analyses of single-cell atlases reveal age, gender, and smoking status associations with cell type-specific expression of mediators of SARS-CoV-2 viral entry and highlights inflammatory programs in putative target cells\n",
      "the receptor for both SARS-CoV 40 and SARS-CoV-2. The receptor-binding domain of the SARS-CoV-2 S-protein has a higher binding affinity for human ACE2 than SARS-CoV 36, 41 , whereas the interaction with CD147 (encoded by the gene BSG), another reported receptor for the SARS-CoV-2 S-protein, is weak (CD147: Kd, 0.185 µM vs hACE2: Kd, ~15 nM) 42 . Following receptor binding, the virus gains access to the host cell cytosol through acid-dependent proteolytic cleavage of the S protein. For SARS-CoV, a number of proteases including TMPRSS2 and CTSL cleave at the S1 and S2 boundary and S2 domain (S2') 43 to mediate membrane fusion and virus infectivity. For SARS-CoV-2, both pharmacological inhibition of endogenous TMPRSS2 protein and TMPRSS2 overexpression support a role for TMPRSS2-mediated cellular entry 37, 44 .\n",
      "\n",
      "The identification of the specific cell types that can be infected by SARS-CoV-2 will inform our understanding of disease transmission and pathogenesis, which are often cell context-specific. Studies suggest that key infection routes involve the nasal passages, airways, and alveoli, where epithelial cells play a key barrier role. Identifying putative target cells in other organs could inform our understanding of extra-pulmonary COVID-19 associated organ failure or of potential placental transmission. Early analyses of the Human Lung Cell Atlas revealed that some of the cells of the nasal passages, airways, alveoli, and gut co-express ACE2 and TMPRSS2 45, 46 .\n",
      "\n",
      "Here, we perform integrated analysis of 107 single-cell and single-nucleus RNA-Seq studies, including 22 studies of the lung and airways, and 85 additional studies of other diverse tissues, spanning both published and unpublished datasets. We comprehensively define the expression patterns of the ACE2 viral receptor and accessory proteases genes. We test how their expression is related to age (from prenatal to old age), sex, and smoking status. We identify gene expression programs associated with cells that can be infected by the virus and compare these programs across specific cell types, organs, and species. To further inform future studies, we assess the conservation of these human features in mouse models and explore the expression of other proteases that may play a role in the viral replication cycle.\n",
      "\n",
      "line with the kidney's role in the renin-angiotensin-aldosterone system, dual-positive cells are enriched in the proximal tubular cells and in principal cells of the collecting duct (Fig. 1a,c) . Interestingly, brain oligodendrocytes, multiciliated and sustentacular cells in olfactory epithelium, AT1 cells in non-smoker lung, and ductal cells in pancreas -all ACE2 + TMPRSS2 + -were also all enriched for MYRF, a transcription factor necessary for myelination in the brain and sufficient to induce expression of the myelin proteins MOG (myelin oligodendrocyte glycoprotein) and MBP (myelin basic protein) 54 . ACE2 + CTSL + cells were enriched in additional subsets, associated with COVID-19 pathology, most notably the olfactory epithelium, ventricular cardiomyocytes, heart macrophages, and pericytes in multiple tissues, including the heart, lung, and kidney (Fig. 1d) .\n",
      "\n",
      "The presence of dual-positive cells in the lung, heart, and kidney may reflect that cells in these organs may be direct targets of viral infection and pathology 5, 55 . Dual-positive cells in the sustentacular and basal cells of the olfactory epithelium (Fig. 1c) may be associated with a loss of the sense of smell 56 . Dual-positive cells in the corneal and conjunctival epithelium, may contribute to viral transmission 4, 57 . Dual positive cardiomyocytes may be related to \"direct\" cardiomyocyte damage (see Tucker et al. companion manuscript 58 ), whereas heart pericytes may indicate a vascular component to the cardiac dysfunction, and could contribute to increased troponin leak in patients without coronary artery disease. Notably, ACE2-expressing heart pericytes in another dataset (Tucker et al) is even higher (30%) than any other tissue dataset analyzed here (max. 21 .7% in kidney, Extended Data Fig. 4) . Despite the lymphopenia observed with COVID-19 2,4,59,60 , we did not typically observe ACE2 mRNA expression in scRNA-seq profiles in the bone marrow or cord blood (Fig. 1a,b) , although there was ACE2 expression in some tissue macrophages, including alveolar and heart macrophages (Extended Data Fig. 4) . Further studies of ACE2 RNA and protein expression in COVID-19 disease tissue will help elucidate its expression in immune cells 61 .\n",
      "\n",
      "To validate our findings from scRNA-seq analysis and to determine the spatial expression patterns of ACE2, TMPRSS2, and CTSL, and their corresponding proteins we performed fluorescence in situ hybridization and immunohistochemistry on tissue sections of airway and alveoli from healthy donor lungs that were rejected for lung transplantation.\n",
      "\n",
      "First, we performed triple fluorescence in situ hybridization to identify ACE2, CTSL and TMPRSS2 on alveolar sections. We observed co-expression, albeit at low levels, of all three genes in alveolar cells (Fig. 1e) . We then performed co-staining with cell type-specific markers. We observed ACE2 transcripts in a subset of type 2 (AT2) cells identified by canonical AT2 protein markers, HTII-280 and pro-SFTPC (Fig. 1f,g) . Similarly, we observed TMPRSS2 gene expression in HTII-280+ AT2 cells (Fig. 1f) . Immunostaining for TMPRSS2 protein further confirmed AT2 cell expression (Extended Data Fig. 5a) . We also observed TMPRSS2 protein expression at low levels in some AT1 cells identified by the canonical AT1 protein marker AGER (Extended Data Fig. 5a) . Of note, some non-epithelial cells also expressed these three genes. We further validated the expression of ACE2 by bulk mRNA-seq in sorted AT2 cells, including those from long-term cultured alveolar organoids (Extended Data Fig. 5b) . We then performed immunohistochemistry and deployed three different available putative ACE2 antibodies to establish ACE2 protein expression (Supplementary Table 4 ). One of these 3 antibodies, the one used previously to functionally block cellular viral entry, specifically labeled adult pro-SFTPC-positive AT2 cells (Extended Data Fig. 5c) . As a cautionary note, the lack of agreement between antibody staining patterns suggests that some of these antibodies may be non-specific.\n",
      "\n",
      "Previous studies have revealed that ACE2 is highly enriched in mucous cells of the nasal and lower respiratory tract epithelium . In healthy lungs, both large and small airways contain mucous cells in surface airway epithelium, albeit at low numbers. However, submucosal glands (SMGs) that reside deep within the airway tissues are composed of abundant mucous cells. To test whether these mucous cells also express ACE2, TMPRSS2, and CTSL, we performed an integrated analysis on scRNA-seq datasets obtained from microdissected SMGs of the healthy donors. We observed overlapping expression and relatively high enrichment of AEC2, TMPRSS2 and CTSL in mucous cells of the SMGS (Extended Data Fig. 5e ). In situ transcript analysis for ACE2 further confirmed the presence of transcripts in acinar epithelial cells of the SMGs (Fig. 1h) , and cells expressing ACE2 in the large airway epithelium (Fig. 1i) .\n",
      "\n",
      "We next sought to understand how the expression of each of these three key genes --ACE2, TMPRSS2, and CTSL --in specific cell subsets may relate to three key covariates that have been associated with disease severity: age (older individuals are more severely affected), sex (males are more severely affected), and smoking (smokers are more severely affected) 62 . We integrated samples across many studies, as no single dataset generated to date is sufficiently large to address this question. We assembled 22 datasets (Supplementary Table 2 , Supplementary Data File D3), comprised of 1,176,683 cells from 164 individuals, spanning 282 healthy nasal, lung, and airway samples profiled by scRNA-seq or snRNA-seq from either biopsies, resections, entire lungs that could not be used for transplant, or post mortem examinations, allowing us to study a diversity of respiratory regions and cell types (Fig. 3a) . These included 6 published datasets 63-68 and 16 datasets that are not yet published [69] [70] [71] [72] [73] . In the case of unpublished data, we only obtained singlecell expression counts for the three genes, as well as the total UMI counts per cell, cell identity annotations, and the relevant anonymous clinical variables (age and sex, as well as smoking status when ascertained). Cell identity annotations were manually harmonized using an ontology with three levels of annotation specificity (Fig. 3b, Supplementary Table 2 ); focusing on levels 2 and 3 allowed us to include a large number of datasets, while retaining relatively high cell subset specificity (Fig. 3a,b) . To facilitate rapid data sharing, we analyzed data pre-processed by each data-generating team at the level of gene counts, using total counts as a size factor. We used Poisson regression (diffxpy package; Methods) to model the association between the expression counts of the three genes and age, sex, and smoking status, and their possible pair-wise interactions (Fig. 3c) , using total counts as an offset, and dataset as a technical covariate to capture sampling and processing differences. It should be noted that modeling interaction terms was crucial as their omission resulted in reversed effects for age and sex for particular cell types (Discussion). This model was fitted to non-fetal lung data (761,693 cells, 165 samples, 77 donors, 10 datasets) within each cell type to assess cell-type specific association of these covariates with the three genes. To further validate sex and age associations, we fit a simplified version of the model without smoking status covariates to the full non-fetal lung data (986,342 cells, 225 samples, 125 donors, 16 datasets).\n",
      "\n",
      "Uncertainty is challenging to model in our single-cell meta-analysis as variability exists on the levels of both donors and cells. For simplicity, we modeled the overall variance with both contributions covered implicitly by treating each cell as an independent observation. As cells from the same donor cannot be typically regarded as independent observations, this can result in inflated p-values, especially when there are few donors for a particular cell type. To counteract this limitation, we employed three approaches: (1) We used a simple noise model (Poisson) to reduce the chance of overfitting donor variability to obtain spurious associations; (2) We confirmed significant associations from the single-cell model in a pseudo-bulk analysis to ensure effect directions are consistent when modeling only donor variation (Methods, Fig. 3d , Extended Data Fig. 6, 7, 8, Supplementary Data D1) ; and, (3) We investigated whether significant associations change direction when holding out any one dataset to ensure that the effect is not dominated by the inclusion of many cells from only one source (Methods, Fig. 3f, Supplementary Data D1) . We regarded an association that passes all of these validations as a robust trend, while associations that appear dominated by a single dataset (often because this dataset is a major contributor of a given cell type) were denoted as indications.\n",
      "\n",
      "We focused on trends or indications in those cell types where both TMPRSS2 and ACE2 are predominantly expressed in the lung: airway epithelial cells (basal, multiciliated, and secretory cells), alveolar AT2 cells, and submucosal gland secretory cells (Fig. 3e) . Strikingly, we find robust trends of ACE2 expression with age, sex, and smoking status in these cell types (Fig. 3d , Extended Data Fig. 6 and MALTE4): ACE2 expression increases with age in basal and multiciliated cells. ACE2 expression is elevated in males in airway secretory cells and alveolar AT2 cells. Furthermore, we find strongly elevated levels of ACE2 in past or current smokers in multiciliated cells (log fold change (log FC): 1.52, Fig. 3d ). Significant associations of ACE2 expression indicate increased expression with age in AT2 (largest age effect: slope of log expression per year of 0.020) and secretory cells, and increased expression of ACE2 in males in multiciliated cells. Further indications associate past or current smoking with decreased ACE2 expression in AT2 cells, and increased ACE2 expression in basal cells. However, these last five indications are not robust trends and depend on the inclusion of a single dataset (Fig. 3f) , often because that dataset contributes a large number of cells of a particular type. Specifically, when we held out the largest declined donor transplant dataset (Supplementary Table 2 , \"Regev-Rajagopal\", most cells and most samples), a declined donor tracheal epithelium dataset (\"Seibold\", Supplementary Table 2 , most donors in the smoking analysis), or a further declined donor lung dataset (\"Kropski-Banovich\", Supplementary Table 2 ) respectively, the effect is no longer present (Methods, Fig. 3f , Supplemenatry Data D1).\n",
      "\n",
      "The above trends and indications for sex and age were further validated in the simplified model on the full non-fetal lung dataset (Extended Data Fig. 8, Supplementary Data D2) . With the exception of the age association in basal and secretory cells, all associations were found to be significant at a false discovery rate (FDR) threshold of 5%, confirmed by pseudo-bulk analysis, and were supported at least at the level of indication. Indeed, all robust trends were also supported as robust trends in the simplified model. Fitting the simplified model on the smoking data subset shows that modeling smoking status is crucial to detect the basal and secretory age association. Without modeling the effect of smoking status on basal and secretory cell ACE2 expression, this variance is captured as uncertainty against which the age effect is evaluated as not significant.\n",
      "\n",
      "Taking into account smoking status is of particular importance as the effect sizes associated with smoking tend to be much larger than age effects, and tend to be larger than sex effects. For example, in multiciliated cells the effect sizes assigned to smoking status, sex, and age associations with ACE2 expression are β=1.52, β=0.51, and β=0.0107 respectively, where β represents the log FC and the slope of log expression per year.\n",
      "\n",
      "Examining joint trends of ACE2 and the protease genes within the same cell type, there are indications of up-regulation of both ACE2 and TMPRSS2 in multiciliated cells (ACE2 indication dependent on \"Seibold\" dataset) in males and with age (both indications dependent on \"Regev-Rajagopal\" dataset). In AT2 cells, there is an indication of joint up-regulation of ACE2 and TMPRSS2 with age (dependent on \"Regev-Rajagopal\" dataset), and an indication of ACE2 and CTSL down-regulation in smokers (dependent on \"Kropski-Banovich\" dataset). All above joint trends for age and sex covariates were confirmed on the full non-fetal lung data using the simple model without smoking covariates.\n",
      "\n",
      "In aggregate, elevated levels of cell-type specific ACE2 and associated proteases are correlated to increasing age, in smokers, and in males.\n",
      "\n",
      "The age associations highlighted particularly low expression in samples from very young children (newborn to 3 years old). As reports suggest that most infants and young children cases do not display severe disease 11, 74 , we inspected the subsets of studies of human development and pediatric samples from our integrated analysis (Supplementary Table 2 ). These included cells from 12 first trimester samples (8 donors) of fetal lung (8.5 weeks post conception; wpc), 29 fetal lung samples (26 donors) from the second trimester (13-24 weeks) 68 , 13 lung samples (10 donors) spanning from third trimester premature births (n=4), full term newborns (n=1), ~3-month old (n=4), 3-year-old (n=3), and 10-year-old (n=1) children. Because the number of samples here is small, all observations must be interpreted with caution.\n",
      "\n",
      "The extent of ACE2 expression in lung cells changes during development (Fig. 3g , Supplementary Table 5 ). There are dual-positive cells present in the very early first trimester lungs by scRNA-seq, and some ACE2 expression in epithelial cells in the second trimester samples (Extended Data Fig. 3a,b) . Notably, spatial transcriptomics of 8.5 pcw fetal lung did not capture any ACE2 expression (data not shown). In lungs from third trimester pre-term births, ACE2 expression is high, with ACE2 + TMPRSS2 + cells observed in alveolar AT2 cell populations (Extended Data Fig. 3b) .\n",
      "\n",
      "Strikingly, ACE2 expression is very low in normal lungs of newborns, the one ~3 months' lung sample, and the 3-year-old lungs (Fig. 3g) . This is further supported by single-cell chromatin accessibility by transposome hypersensitive sites sequencing (scTHS-Seq 75 ) from human pediatric samples (full gestation, no known lung disease) collected at day 1 of life, 14 months, 3 years, and 9 years (n=1 at each time point) (Extended Data Fig. 9a) . ACE2 gene activity scores (Methods), when present, were in the AT1/AT2 population, but no signal was present at birth, it was low in the 9-year-old and 3-year-old sample, and higher in the 14 month-old (Extended Data Fig. 9b-d) .\n",
      "\n",
      "Notably, immunohistochemistry (IHC) of the ~3 month-old infant lung also showed fewer ACE2immunoreactive AT2 cells (Extended Data Fig. 5d ).\n",
      "\n",
      "We also assessed whether ACE2, TMPRSS2, and CTSL are expressed in the human placenta during pregnancy, using data from three published scRNA-seq studies: two from the first trimester (64,000 cells and 14,000 cells) and one from full-term placenta (20,518 cells) [76] [77] [78] . ACE2 was expressed (1.4%) in maternal decidual/stromal cells, maternal pericytes, and fetal extravillous trophoblasts, cytotrophoblasts, and syncytiotrophoblast in both first-trimester and term placenta (Fig. 1d) . Note that there was little expression of TMPRSS2 (0.2%) in the placenta and accordingly few ACE2 + TMPRSS2 + dual-positive cells (as we previously reported 45 ). However, CTSL is expressed in most cells (56%) in the maternal-fetal interface, and there are ACE2 + CTSL + dualpositive cells (1.3%) among maternal decidual/stromal cells, pericytes, and fetal trophoblasts (HLA-A, B negative) in both first-trimester and term placentae.\n",
      "\n",
      "Overall, these patterns may be important in understanding why children are more resistant to COVID-19 and and in considering risk during pregnancy.\n",
      "\n",
      "Our Human Lung Cell Atlas analyses have revealed immune signaling genes that co-vary with ACE2 and TMPRSS2 in airway and lung cells 45, 46 . These analyses identified antiviral response genes that are enriched in ACE2 + TMPRSS2 + cells (e.g., IDO1, IRAK3, NOS2, TNFSF10, OAS1, MX1), and suggested that ACE2 itself is interferon regulated 45, 46 .\n",
      "\n",
      "To explore such gene programs in a broader context, we identified signatures for dual-positive ACE2 + TMPRSS2 + cells compared to dual-negative ACE2 -TMPRSS2cells in the nasal epithelium, lung, and gut (Supplementary Tables 6, 7 8) with two complementary approaches. The first aimed to find features that characterize programs of dual positive cells that are shared by different cell types in one tissue (\"tissue programs\"). The second aimed to find features that are associated with dual positive cells compared to other cells of the same type, and may or may not be shared with other types (\"cell programs\") (Methods). To infer tissue programs, we trained a random forest classifier to discriminate between dual-positive and dual-negative cells (excluding ACE2 and TMPRSS2; 75:25 class balanced test-train split), generalizing across multiple cell types in one tissue, and ranked genes according to their importance scores in the classifier (Methods). To infer cell programs, we performed differential expression analysis between dual-positive and dualnegative cells within each cell subset. We note that ACE2 + TMPRSS2 + cells have more unique transcripts detected (Extended Data Fig. 10b) : this can reflect a technical confounder, biological features, or both. We conservatively controlled for these differences (by sampling dual positive and dual negative cells from matched gene complexity bins; Methods; Extended Data Fig. 10b , Extended Data Fig. 11) . Importantly, these methods do not assume that ACE2 + TMPRSS2 + cells form a distinct subset within each cell type. Rather, our goal is to leverage the variation among single cells within a single type to identify gene programs that are co-regulated with ACE2 and TMPRSS2 within each expressing cell subset.\n",
      "\n",
      "Tissue programs (Fig. 4a, Extended Data Fig. 10a 10a for visualization of selected genes, and Supplementary Tables 6 for the full list). These include phagosome structure, antigen processing and presentation, and apoptosis. Among the tissue program genes we highlight: CEACAM5 (lung, nasal, gut programs) and CEACAM6 79 (lung), surface attachment factors for coronavirus spike protein; SLPI (lung, nasal), a secreted protease inhibitor that is associated with virus resistance 80 ; PIGR (lung, gut), the polymeric immunoglobulin receptor that may promote antibody-dependent enhancement via IgA 81 ; and, CXCL17 (lung, nasal), a mucosal chemokine that attracts dendritic cells and monocytes to the lungs 82 . In addition, the tissue programs yielded multiple genes that are associated with cholesterol and lipid metabolic pathways and endocytosis (DHCR24, LCN2, FASN); high expression of both MHC I (B2M, HLA-B) and MHC II (HLA-DRA, DRB1, CTSS, CD74) pathways, expanding on prior evidence that AT2 cells can serve as antigen presenting cells in infection 83 ; genes indicating preparation against cellular injury through ROS inhibition, preventing proteases, or through antiviral responses (interferons, extracellular RNAse, etc: PLAC8, TXNIP); complement genes (C3, C4BPA); genes involved in immune modulation (BTG1); and, tight junction genes (DST, CLDN3, CLDN4).\n",
      "\n",
      "The cell programs (Fig. 4c, Extended Data Fig. 11a ,b, Supplementary Table 6 ) were enriched in many of the same genes and pathways as tissue-specific programs (Fig. 4d , Supplementary Table 6 ,7,8,9,10), and highlight a potential role for TNF signaling in ACE2 regulation. We first confirmed that the cell programs were not merely associated with the number of transcripts per cell (Extended Data Fig. 11c) . While some genes were shared between the tissue and cell programs (e.g., many virus-related genes, such as CEACAM5, CXCL17, SLPI, and HLA-DRA), the cell programs further captured unique biological functions and activities. For example, dual positive lung secretory cells differentially expressed genes involved in TNF signaling including RIPK3, a key regulator of inflammatory cell death via necroptosis, previously implicated in SARS-CoV pathogenesis 84 . Both lung dual positive secretory and multiciliated cells differentially expressed lysosomal genes (MFSD8, CTSS, CTNS, CTSH), potentially relevant for endolysosomal entry of coronaviruses 85 . Dual positive AT1 cell programs included genes involved in immunoproteasome (PSMB8, PSMB9, Fig. 4c) , class I and II antigen presentation (HLA-DMA, HLA-DRB5, HLA-DPB1, HLA-DRA, HLA-DPA1), and phagocytosis. Dual-positive nasal goblet cells differentially expressed several cytokines and chemokines, including granulocyte-colony stimulating factor (CSF3), which may impact hematopoiesis, the recruitment of neutrophils, and inflammatory pathology; CXCL1 and CXCL3, chemoattractants for neutrophils; interleukin-19 (IL19), which induces the production of IL-6 and TNF 86 ; and CCL20, which is upregulated by TNF 87 . The AT2 cell program included the surfactant proteins, SFTPA1 and SFTPA2; the IL-1 receptor (IL1R1), which may promote antiviral immune responses (below); and, multiple components of MHC-II (e.g., HLA-DPA1, HLA-DPB1), congruent with a role in antigen presentation.\n",
      "\n",
      "Cell programs from multiple tissues (Fig. 4c,d) included genes related to TNF signaling (e.g., BIRC3, CCL20, CXCL1, CXCL2, JUN, NFKB1), raising the possibility that anti-TNF therapy may impact the expression of ACE2 and/or TMPRSS2. Consistent with this hypothesis, ACE2 expression in enterocytes was significantly lower in ulcerative colitis patients treated with anti-TNF compared to untreated patients (mean = 0.22 and 0.13 log2(transcripts per 10,000 (TP10K)+1) in treated vs. untreated; adjusted P < 1e-3). However, we could not control for many important features, including disease severity, which is strongly associated with anti-TNF treatment, raising the need for future work. Some of the genes are targets of known drugs 88 . For example, dualpositive lung secretory cells expressed, in addition to ACE2 (targeted by ACE inhibitors), other drug targets, including C3, HDAC9, IL23A, PIK3CA, RAMP1, and SLC7A11. Other program genes were shown to interact with SARS-CoV-2 proteins via affinity purification mass spectrometry 89 . Among those was GDF15, which was identified as a putative interaction partner for the SARS-CoV-2 protein Orf8 89 , is a central regulator of inflammation 90 , and was a member of the dual-positive cell programs of both lung basal cells and nasal multiciliated cells.\n",
      "\n",
      "Some program genes may be particularly related to COVID19 pathological features and may indicate putative therapeutic targets. For example, MUC1 is especially highly induced in dualpositive cells (in tissue and specific cell programs), which may be associated with respiratory secretions 91 . Importantly, the lung tissue and gut enterocyte programs include the gene encoding the IL6 co-receptor (IL6ST), and the AT2 cell program includes IL6. IL6 signaling has been implicated in uncontrolled immune responses in the lungs of COVID19 patients, elevated serum IL6 levels are associated with the need for mechanical ventilation 92 , and anti-IL6R antibodies (tocilizumab) are being tested for clinical efficacy in COVID-19 patients. Indeed, IL6ST and IL6 are higher in dual positive vs. dual negative AT2 cells (Extended Data Fig. 11d) , although IL6 expression is relatively low in these cells from healthy tissue. Additional cell types, such as heart pericytes, are enriched for cells with co-expression of ACE2 with IL6R or IL6ST (Extended Data Fig. 12 ).\n",
      "\n",
      "The immune-like features of ACE2 + epithelial cells are also reflected in the regulatory features of the ACE2 locus by scATAC-Seq (Fig. 4f) . Epithelial cells (AT1, AT2, secretory, multiciliated, ionocytes, and neuroendocrine cells) with open chromatin at the ACE2 locus show high motif activity scores 93 for interferon-regulatory factors such as IRF1/7 (MA0050.1, adj p-val=0.02, MA0772.1, adj p-val=0.08), STAT1::STAT2 (MA0517.1, adj p-val=0.15), FOXA1 (MA0148.2, adj p-val=0.24) and FOXD2 (MA0847.2, adj p-val=0.27), compared to epithelial cells with no sign of open chromatin at the ACE2 locus, though the difference is statistically significant (adj p-val <0.05) only for IRF1. Interestingly, Forkhead Box Transcription Factors have been implicated in regulation of ACE2 transcription 94 . Note that because epithelial cells with an accessible ACE2 locus tend to have a higher number of fragments in peaks than cells with inaccessible ACE2 (Extended Data Fig. 11f ), consistent also with higher UMIs in scRNA-seq, some of the cells with inaccessible ACE2 could be false negatives, reducing our power.\n",
      "\n",
      "Previous studies in the healthy lung predicted that interactions between AT2 cells and myeloidlineage macrophages may be important for immune regulation and surfactant homeostasis 95 . To explore this possibility, we predicted interactions between AT2 cells (in general, or ACE2 + TMPRSS2 + dual-positives) and myeloid cells (Methods 96 ), using our large declined donor transplant dataset (\"Regev/Rajagopal\"; 41 samples, 10 patients, 2-6 locations each). AT2 cells and myeloid cells were present in lung lobes samples from all 10 patients, whereas samples from 5 patients contained both ACE2 + TMPRSS2 + dual-positive AT2 cells and myeloid cells. We identified significant predicted interactions involving Oncostatin M (OSM), an IL6-type cytokine expressed by myeloid cells 97 , with the Oncostatin M Receptor (OSMR) and its paralog receptor LIF Receptor Subunit Alpha (LIFR) expressed in both for AT2 cells in general, and in double positive ones). Interactions involving the complement pathway were also predicted (for all AT2 and dual positives) between Complement C3 and C5 expressed by AT2 cells and their cognate receptor expressed in myeloid cells. Three samples had interactions between the IL1 receptor on AT2 cells and IL1B or IL1RN in myeloid cells. The IL1-receptor interactions were identified mostly (in 2 out of 3 samples) involving only dual-positive AT2 cells, suggesting a possible role of ACE2 + TMPRSS2 + dual-positive cells in IL1-mediated processes. Finally, we identified interactions between CSF1, 2, or 3 expressed in AT2 cells (including double positives) and their receptors expressed in myeloid cells. These predicted interactions further support the previously identified roles for cross-talk between AT2 and myeloid cells, such as macrophages, in immune regulation (OSM, complement, IL1) and surfactant homeostasis (CSF), as previously highlighted 95 .\n",
      "\n",
      "We next asked whether human cell types of interest were present in animal models. While such analyses cannot address molecular compatibility (due to sequence variation in ACE2 across species, as shown for lower compatibility of SARS-CoV and mouse ACE2 98 ), they can help determine if dual-positive cells are present in commonly employed models, and if their characteristics, proportions, and programs are similar to those of their human counterparts. In a separate study 46 , our lung network showed strong similarities to the human data in a macaque model. Here, we focused on the more distant, but commonly used, mouse model. Ace2 + Tmprss2 + and Ace2 + Ctsl + dual-positive cells were present primarily in club and multiciliated cells in the airway epithelia of healthy mice 99 (Ace2 + Tmprss2 + club 5.1% [4.7%, 5.4%] and multiciliated 2.8% [2.3%, 3.5%], Ace2 + Ctsl + club 11.1% [10.6%, 11.6%] and multiciliated 3.5% [2.9%, 4.3%]), consistent with the expression patterns found in human airways (Fig. 5a) . Furthermore, Ace2 expression increased over a 2-month time-course of healthy mouse aging in both club (p=1.16e-06) and goblet (p=0.01911) cells (Fig. 5a) . The proportion of Ace2 + Tmprss2 + dual-positive cells did not significantly increase with age during this time course (data not shown), but the proportion of Ace2 + Ctsl + dual-positive cells significantly increased in club cells during this time-course (Fig. 5b) . Interestingly, the mice were aged between 2-4 months, a 2-month period that is reported to reflect the maturation period from early to mature adults 100 . Examining bulk RNA-Seq profiles of sorted populations of alveolar AT2 cells (SFTPC + ), airway basal cells (KRT5 + ), alveolar endothelial cells (CD45-CD31 + ), alveolar epithelial cells (Epcam + ), whole lung and whole trachea from a KRT5-CreER/LSL-TdTomato/SFTPC-eGFP transgenic mouse model, and across tissues from ENCODE, showed that Ace2, Tmprss2 and Ctsl are expressed in sorted AT2 cells, whole trachea and whole lung, as well as in stomach, intestine, kidney and bladder.\n",
      "\n",
      "In human smokers, statistical modeling uncovered a robust trend of increased ACE2 expression in airway epithelial cells, while expression in AT2 cells was reduced (Fig. 3d, Extended Data Fig.  6 ). To experimentally confirm these findings, we examined cell profiles from mice exposed daily to cigarette smoke for two months, followed by scRNA-seq of whole lungs (Fig. 5c) . Epithelial specific expression patterns of mouse Ace2 and the Ace2 + Tmprss2 + and Ace2 + Ctsl + dual-positive cells were largely consistent with the human data (Fig. 5d) . Upon smoke exposure, there was a significant increase in Ace2 + airway secretory cell numbers, while the fraction of Ace2 + AT2 cells was unaltered (Fig. 5e) . Moreover, the expression levels of Ace2 were significantly increased in airway secretory cells (Fig. 5f ), but not in AT2 cells (Fig. 5g) . This was in agreement with bulk RNA-seq of mouse lungs exposed to different doses of cigarette smoke 101 , in which Ace2 levels increased in a dose-dependent manner by daily cigarette smoke over 5 months (Fig. 5h) . Notably, the COVID-19 relevant proteases Tmprss2 and Ctsl were also significantly increased by smoke exposure in mice (Fig. 5i,j) . Thus, mouse smoking data shows similar trends as observed in humans and experimentally confirms the association of ACE2 levels with smoking.\n",
      "\n",
      "We also compared the patterns between the mouse and human placenta, analyzing Ace2, Tmprss2, and Ctsl expression across 100,000 cells from scRNA-seq data during mouse placenta development from embryonic days 9.5 to 18 (Shu et al., unpublished). We find Ace2 + Tmprss2 + dual-positive cells (1.4%) in a large fraction of fetal trophoblasts with strong epithelial signatures. Ace2 + Tmprss2 + dual-positive cells express signatures of AT2 cells and hepatocytes, and many also express Ctsl. Ace2 + Ctsl + dual-positive cells (3.4%) are also present among fibroblasts, stromal cells, and fetal trophoblasts in both mice and humans (Fig. 5k, Extended Data Fig. 13) . Notably, while ACE2 + CTSL + dual-positive fibroblasts and stromal cells in humans are of maternal origin, Ace2 + Ctsl + dual-positive fibroblasts and stromal cells are of fetal origin in mice.\n",
      "\n",
      "TMPRSS2 has been demonstrated to mediate SARS-CoV-2 infection in vitro 37, 44 , but SARS-CoV-2 also infects cells in the absence of TMPRSS2 37 . Thus, additional proteases likely play roles in proteolytic cleavage events of spike and other viral proteins that underlie entry (fusion) and egress. To systematically predict proteases potentially involved in SARS-CoV-2 pathogenesis, we tested for co-expression of each of 625 annotated human protease genes 102 with ACE2 in the large declined donor transplant dataset (\"Regev/Rajagopal\") from 10 patients. The analysis recovered TMPRSS2 as one of the significantly co-expressed in multiple lung epithelial cell types (Fig. 6a , Supplementary Table 11, 12 ). In addition, multiple members of the proprotein convertase subtilisin kexin (PCSK) family were also significantly co-expressed with ACE2 in both proximal and distal airway epithelial cells (Fig. 6a,b) , including FURIN, PCSK2, PCSK5, PCSK6 and PCSK7 in AT2 cells. Proprotein convertases have known roles in coronavirus S-protein priming 43, 103, 104 . We obtained similar results in an independent dataset of 182,952 cells from 40 donors (Extended Data Fig. 14a ,b, \"aggregated lung\").\n",
      "\n",
      "To further investigate the role of proprotein convertases as candidates for SARS-CoV-2 S-protein processing we analyzed the SARS-CoV-2 spike protein sequence. Multiple sequence alignment of S-protein sequences of SARS-CoV-2 and other beta-coronaviruses revealed a polybasic insert at the S1/S2 junction present only in SARS-CoV-2 spike (Extended Data Fig. 14c ). While polybasic sites are found in multiple members of betacoronavirus lineages A and C (e.g., MERS-CoV), SARS-CoV-2 is the only known member of lineage B harboring a polybasic motif in the S1/S2 region (Extended Data Fig. 14c ). As previously reported, this polybasic sequence corresponds well to cleavage motifs of multiple PCSK family proteases (Extended Data Fig. 14d ) [105] [106] [107] , and has a high probability for its PCSK-mediated cleavage (at amino acid 685) (by ProP and PROSPERous 108,109 ) as well as additional sites including the S2' position (at amino acid 815), which would release predicted fusion-mediating peptides (Extended Data Fig. 14e ) 110 .\n",
      "\n",
      "We next examined PCSKs expression and co-expression with ACE2 across lung cell subsets (Fig.  6c, Extended Data Fig. 14f) . FURIN, PCSK5 and PCSK7 were broadly expressed across multiple lung cell types, and PCSK1 and PCSK2 were largely restricted to neuroendocrine cells, as previously reported 107 , with PCSK2 further detected in 11.2% of AT2 cells (Fig. 6d, Extended  Data Fig. 14g ). In many cell subsets we observed dual expression with ACE2 at fractions comparable to or higher than those of ACE2 + TMPRSS2 + cells (Fig. 6e, Extended Data Fig. 14h ). These include AT2 cells (ACE2 + TMPRSS2 + , ACE2 + FURIN + and ACE2 + PCSK2 + at 0.90%, 0.78%, and 0.56%, Fig. 6e) ; multiciliated cells in the proximal airway (ACE2 + TMPRSS2 + , ACE2 + FURIN + , ACE2 + PCSK7 + , and ACE2 + PCSK5 + at 0.91%, 0.37%, 1.02%, and 0.91%), and basal cells (ACE2 + TMPRSS2 + , ACE2 + FURIN + , and ACE2 + PCSK7 + at 0.19%, 0.21%, and 0.74%). Co-expression is present across tissues in addition to the lung (Extended Data Fig. 14i,j) , including the liver, ileum, kidney and nasal airways, with the highest percentages of ACE2 + PCSK + dual positive cells in nasal airways (ACE2 + PCSK7 + 1.36%, ACE2 + FURIN + 0.67%), bladder (ACE2 + PCSK5 + 0.45%) and testis (ACE2 + PCSK7 + 0.41%).\n",
      "\n",
      "Because different host proteases may contribute to different stages of the viral life cycle 107,111 , we also examined the prevalence of ACE2 + TMPRSS2 + PCSK + triple-positive cells (TPs) in the lung dataset. ACE2 + TMPRSS2 + PCSK7 + were the main triple positive cells in multiciliated (0.75%) and secretory cells (0.72%) of proximal airways, and ACE2 + TMPRSS2 + FURIN + TPs were the most common within AT2 cells (0.36%) (Extended Data Fig. 14k) . Finally, when we examined all known human proteases for co-expression with ACE2 in major lung epithelial cell types (Fig. 6f) , we recovered cathepsins (CTSB, CTSC, CTSD, CTSL, CTSS), proteasome subunits (e.g. PSMB2, PSMB4, PSMB5), and complement proteases (C1R, C2, CFI) (Fig. 6f, Extended Data Fig. 15 ), the latter also captured in our programs above (Fig. 4) .\n",
      "\n",
      "We performed integrative analyses of single-cell atlases in the lung and airways and across tissues to identify cell types and tissues that have the key molecular machinery required for SARS-CoV-2 infection. We then examined the relationship between specific cell types and three key covariates --age, sex, and smoking status --that have been related to disease severity. We further used the scale of these integrated atlases to identify gene programs in major epithelial cell subsets that may be infected by the virus, and search for other potential accessory proteases. Our hope is that this extensive analysis and resource will help with hypothesis generation (and refutation) towards better understanding of the molecular and cellular basis of COVID-19 infection, and the identification of putative therapeutic avenues.\n",
      "\n",
      "Our cross-tissue analysis substantially expands on our 45,46,58,112 and others' 113-115 earlier efforts, allowing us to identify cell subsets across diverse tissues that may be implicated in virus transmission, pathogenesis, or both. Focusing on pathogenesis, in addition to key subsets in the lung, airways and gut, we identified ACE2 + cells that co-express either TMPRSS2 or CTSL in diverse organs, many of which have been associated with severe disease. These include epithelial cells in the liver, kidney, pancreas, and olfactory epithelium, cardiomyocytes, pericytes and fibroblasts in the heart, and oligodendrocytes in the brain. For example, the presence of double positive cardiomyocytes and cardiac pericytes and fibroblasts may provide a pathological basis for the cardiac abnormalities noted in COVID-19 patients including elevated troponin, a signature of cardiomyocyte injury, myocarditis, and sudden cardiac death 5 . As the co-expression of genes . CC-BY-ND 4.0 International license author/funder. It is made available under a\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint involved in SARS-CoV2 infection are highest in cardiac pericytes in healthy heart, damage to vascular beds may trigger troponin release in otherwise normal hearts. Moreover, as myocardial ACE expression is increased in patients with existing cardiovascular diseases (Tucker et al. companion manuscript 58 ), SARS-CoV2 infection may result in greater damage to cardiomyocytes, and account for greater disease acuity and poorer survival in these patients.\n",
      "\n",
      "One intriguing clinical observation is that some COVID-19 patients display an array of neurologic symptoms 116, 117, 118 , reported as seizures and acute necrotizing encephalopathy, similar to that previously observed following other infections such as influenza 119, 120 . Neuroinflammation could result from direct viral infection of the brain, or a systemic cytokine storm 121-123,124-126,121-123 . Direct viral invasion of SARS-CoV and MERS-CoV is observed in multiple brain regions in human patients and mouse models 124, 125 , consistent with widespread ACE2 expression in numerous brain cell types. Furthermore, SARS-CoV has been shown to infiltrate the brain via the olfactory epithelium-olfactory bulb axis 127 ; olfactory transmission for SARS-CoV-2 has been recently proposed 128 . Other possible transmission routes could be through the infection of ACE2 + TMPRSS2 + enteric neurons synapsing with vagal afferents, or entry through blood-CNS interfaces such as the choroid plexus or meninges 121,129-131 . Profiling immune cells at these sites after infection is an important future step to better understand how the viral response may lead to encephalitis. One intriguing possibility is that encephalitis might arise as an autoimmune response to myelin antigens expressed by infected cells. Antibodies against peptides of myelin proteins have been clinically shown to be associated with autoimmune encephalitis and seizures [132] [133] [134] , and myelin peptides are targets of T cells 135 in demyelinating inflammatory neurological diseases such as acute demyelinating encephalomyelitis, Guillain-Barre Syndrome, and multiple sclerosis [135] [136] [137] [138] [139] [140] [141] [142] . Oligodendrocytes, the myelin-producing cells of the CNS, are the main ACE2 + TMPRSS2 + cell type in the brain, the myelin transcriptional regulator MYRF was enriched in certain ACE2 + TMPRSS2 + cell types as noted above, and myelin proteins MOG and MBP were coexpressed in numerous ACE2+ clusters across organs. MYRF and MBP were significantly differentially expressed in ACE2 + TMPRSS2 + subsets of the lung and gut (Supplementary Table  10 ); myelin targeting Th17 cells trained in the gut are able to infiltrate the CNS in a mouse model of experimental autoimmune encephalomyelitis 143 . Taken together, the expression of myelin proteins across multiple ACE2 + TMPRSS2 + cells may hypothetically contribute to antigen presentation and autoimmune response in the context of viral infection. A small number of COVID-19 patients were reported to have Guillain-Barre Syndrome 144 , with a demyelinating process in some; this is consistent with observations of Guillain-Barre Syndrome following other viral infections such as Zika, influenza, and Epstein-Barr virus 139, 145, 146 . Whether anti myelin specific immunity can be induced by virus infected cells expressing myelin proteins remains an area for future study.\n",
      "\n",
      "Our meta--analysis of scRNA-seq across studies provided the required statistical power to uncover population-level signals at a molecular level and at single-cell resolution. We found that the SARS-CoV-2 receptor and associated proteases were up-regulated in airway epithelial and AT2 cells with age and in males, an association that may shed light on the marked increase in mortality with age. Furthermore, ACE2 was up-regulated in airway epithelial cells (basal and multiciliated cells) in past or present smokers, but down-regulated in their AT2 cells; we have also confirmed this in an experimental system in a mouse model. These contrasting smoking associations show the importance of the single-cell resolution, as the down-regulation in AT2 cells will be masked by . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint the airway epithelial signal, leading to loss of association or misinterpretation of seemingly consistent ACE2 signals in bulk RNA-Seq 147 . Importantly, ACE2 is particularly lowly expressed in young pediatric samples, also mirrored by lack of chromatin accessibility in the ACE2 locus. ACE2 expression is known to be regulated in complex ways across different tissues and may be affected by both common therapies (ACEi/ARB) (Tucker et al., companion manuscript 58 ), and during infection 46 . Moreover, both higher ACE2 expression per cell and a higher fraction of ACE2 + cells can in principle have implications for infection, but may have conflicting effects on pathogenesis, as ACE2 knockout mice show more severe ARDS upon lung injury because of its role in the renin-angiotensin pathway, which seems to protect from consequences of lung injury and inflammation 148 (for potential roles of this pathway on CoV infection (pre-COVID) see the review in 149 ). As SARS-CoV2 binding will lead to internalization and therefore downregulation of ACE2 on the cell surface, the protective function of ACE2 via its proteolytic processing of Angiotensin-II may be lost. Thus, the smoking mediated downregulation of ACE2 in AT2 cells may not protect cells from being infected but rather may increase ARDS due to more severe loss of ACE2 from the cell surface upon infection. Other confounders, including ACEi which are not available to our meta-analysis may further impact our results.\n",
      "\n",
      "To the best of our knowledge, this study is the first single-cell meta-analysis (in any setting). To perform this meta-analysis, we used a model that included both the tested covariates (age, sex, and smoking status), technical covariates (dataset and the number of UMIs per cell), and several interaction terms. Including these interaction terms was crucial, as omission resulted in increased background variation and reversed effect estimates. Likewise, modeling the smoking status of a donor was important to reduce background variation and account for the unbalanced distribution of covariates in the dataset. For example, while we have similar numbers of male ascertained smokers and non-smokers (21 and 20 donors), there are three times as many female ascertained non-smokers as female smokers (27 and 9 donors), which is reflective of this bias in the population 150 . The addition of these terms increases the complexity of the model. Indeed, only one dataset (\"Seibold\") had sufficient numbers of donors of various ages, sex, and smoking status to fit the full model. Thus, performing the meta-analysis was only possible due to the aggregation of a large number of healthy single-cell datasets enabled by the HCA Lung Biological Network and a community-wide effort.\n",
      "\n",
      "A limitation of our expression model is that each cell is treated as an independent observation. Thus, the significance of association with traits such as sex, age, and smoking status may show inflated p-values, especially where the associations are determined from few donors. In this case, the variation between cells from a single donor dominates the variation between donors, background variation is underestimated and effect significance can be overestimated. Aggregating many datasets allows us to counteract this effect, yet p-value inflation may occur in cell types that are not as commonly shared across datasets. Our main conclusions are drawn on airway epithelial and AT2 cells, which are distributed widely across datasets and are modeled on the basis of many donors. Furthermore, we have confirmed significant associations by pseudo-bulk analysis and by holding out datasets. This confirmation ensures that associations are consistent when only considering donor variation, and we are aware if these associations are dataset dependent (often when one dataset is a particularly major source of a given cell type). Models that account for both single-cell count distributions, and population structure in the data have the potential to improve future meta-analyses across single-cell atlases.\n",
      "\n",
      ". CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint\n",
      "\n",
      "Having a cell type annotation with consistent resolution across datasets was instrumental for analyzing the association with clinical covariates up to the resolution of basal, secretory, or multiciliated cells. These cell type labels still aggregate over considerable diversity, which is the subject of ongoing scientific research. Importantly, the labeled subtypes of these cell clusters differ between datasets. Thus, in those cases where individual associations depend on a particular dataset not being held out, it may be the case that these associations become more robust at a higher level of cell type annotation. Future high-resolution cell annotation efforts have the potential to further consolidate our single-cell meta-analysis results.\n",
      "\n",
      "In addition to modeling associations between gene expression and clinical co-variates, we also examined whether the proportion of ACE2 + TMPRSS2 + cells per sample is associated with age or sex. While we can observe a trend of double positive cell proportions increasing with age (Extended Data Fig. 3a) , the high compositional diversity across samples and studies (Fig. 3a,  Extended Data Fig. 16 ), the potential confounders (total counts, dataset), and limited sample numbers are prohibitive to modeling these associations. Further metadata that describe the sample diversity such as harmonized annotations on anatomical location, sampling methods, and sample processing can help to capture this heterogeneity in ACE2 + TMPRSS2 + cell proportion models.\n",
      "\n",
      "The expression of ACE2 and TMPRSS2 in lung, nasal and gut epithelial cells is associated with expression programs with many shared features, involving key immunological genes and genes related to viral infection, raising many hypotheses for future studies, especially as more patient tissue samples are analyzed in the coming months. In the lung, epithelial cells express IL6, IL6R and IL6ST, which raises the hypothesis that infection may trigger cytokine expression from these cells and contribute to uncontrolled immunological responses. The immune-like programs in these cells are further reinforced by the accessibility of STAT and IRF binding sites in scATAC-Seq data, consistent with another study from our network showing the role of interferon in regulating ACE2 expression in epithelial cells 46 . Notably, scRNA-Seq analysis of immune cells from bronchoalveolar lavage fluid of COVID-19 patients identified high activity of transcription factors such as STAT1/2 and IRF1/2/5/7/8/9 in macrophage states increased in severe COVID-19 patients 151 . Other hypotheses for future studies include lysosomal genes in dual positive lung secretory and multiciliated cells, which may be consistent with putative \"viral entry\" cells, and RIPK3 expression in the cell programs of airway cells, which opens the hypothesis of necroptosis initiating a pro-inflammatory response. Interestingly, we observed relatively high enrichment of ACE2 in secretory cell types (mucous cells and AT2 cells). We speculate that viruses may take advantage of the rich secretory pathway components in these cells for their efficient dispersal. Additionally, SMGs of the airways are recently shown to serve as reservoirs of reserve stem cells 152, 153 . Therefore, we also speculate that SMGs similarly may serve as reservoirs for viruses where they can escape from muco-ciliary transport and mechanical expulsion associated with severe cough in the airway luminal surface.\n",
      "\n",
      "The gene programs of AT2 cells can also contribute to cross talk with alveolar macrophages. Our cell-cell interaction analysis suggests that AT2 cells engage with alveolar macrophages through Oncostatin M, CSF, IL1 and complement pathways, suggesting therapeutic hypotheses. The complement pathway is particularly intriguing in the context of COVID-19. First, viral protein glycosylation is a known trigger for the lectin pathway (LP) of the proteolytic complement . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint cascade, such that in addition to classical complement activation via antibody complexes, other coronavirus glycoproteins can be recognized by LP-inducing host collectin proteins 154, 155 . Moreover, excessive complement activation resulting in acute lung injury and cytokine storms were also implicated in the pathogenesis of SARS 156 , and complement inhibition using the anti-C5 antibody eculizumab 157 is currently evaluated as anti-inflammatory experimental emergency treatment for severe COVID-19 in clinical trials (ClinicalTrials.gov identifier NCT04288713). In our analysis, multiple complement pathway proteases (e.g. C1R, C2, CFI) are co-expressed with ACE2 across different lung cell subsets (Fig. 6f,g, Extended Data Fig. 15 ) and complement inhibitory factor CD55 and complement protease C3 were preferentially expressed by ACE2 + TMPRSS2 + DPs within lung tissue, in multiciliated and secretory cells, respectively (Fig.  4a,c, Supplementary Table 6, Supplementary Tables 7,8) . Moreover, cell-cell interaction analysis predicted cross-talk between AT2 cells expressing complement proteins C3 and C5 and macrophages expressing cognate receptors. AT2 cell expression of negative complement regulators CFI and CD55 might represent a strategy for SARS-CoV-2 to at least partially escape complement surveillance.\n",
      "\n",
      "Finally, to explore therapeutic hypotheses related to disruption of viral processing via protease inhibition, we explored the expression of other proteases across our integrated atlases. Although a multitude of different SARS-CoV-2 features likely account for its high pathogenicity and transmissivity, it has been speculated that the PRRAR loop might contribute to increased COVID-19 severity. Introduction of similar polybasic cleavage sites into avian influenza viruses and human coronaviruses was shown to render them more pathogenic, increasing mortality and viral spread 158, 159 . One hypothesis is that acquisition of a PCSK cleavage site would expand the number of cell types that can be directly infected by SARS-CoV-2. A recent report has started to address expression of FURIN in cells expressing SARS-CoV-2 host factors ACE2 or TMPRSS2 47 , and FURIN activity is inhibited by Guanylate-binding proteins (GBPs), a group of interferonstimulated genes, in order to restrict viral envelope processing 160 . However, the highly overlapping recognition sequence of PCSK family members (Extended Data Fig. 14d) suggests that multiple PCSKs in addition to FURIN could mediate cleavage at the S1/S2 PRRAR motif (Extended Data  Fig. 14e) . Our expression analysis confirms that PCSK family members, in particular FURIN, PCSK5 and PCSK7, are more broadly expressed than TMPRSS2 across lung cell types (Fig. 6d) , as well as across tissues (Extended Data Fig. 14i) . In the lung, we note the higher proportion of ACE2 + PCSK7 + basal cells and ACE2 + PCSK7/5 + fibroblasts (Fig. 6e, Extended Data Fig. 14h) . Interestingly, the host interactome of SARS-CoV-2 further suggests interaction of viral proteins with PCSK6 89 , which also showed significant ACE2 co-expression in AT2 cells (Fig. 6b,  Extended Data Fig. 14b) . Moreover, because PCSK localization is detected in different membrane compartments along the secretory and endocytic pathways 107 , it is conceivable that PCSKs could process SARS-CoV-2 S-proteins at different stages of the viral life cycle. Moreover, further analysis is required to assess the extent to which SARS-CoV-2 relies on proteolytic activity provided in trans either by neighboring cells or extracellularly localized proteases 111 . Altogether, this could provide SARS-CoV-2 with an immense flexibility in different entry and egress pathways.\n",
      "\n",
      "Taken together, our analyses provide a rich molecular and cellular map as context for the transmission, pathogenesis, clinical associations, and therapeutic hypotheses for COVID-19. As The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint new single cell atlases will be generated from COVID-19 tissues and experimental models, they will help further advance our understanding of this disease.\n",
      "\n",
      "Sample collection underwent IRB review and approval at the institutions where the samples were originally collected. \"Adipose_Healthy_Manton_unpublished\" was collected under IRB 2007P002165/1(ORSP-3877). Tissue samples from breast, esophagus muscularis, esophagus mucosa, heart, lung, prostate, skeletal muscle and skin referred to as \"Tissue_Healthy_Regev_snRNA-seq_unpublished\" were collected under ORSP-3635. Samples The studies relating to datasets \"Schultze\" and \"Schultze_Falk\" were approved by the ethics committees of the University of Bonn and University hospital Bonn (local ethics vote 076/16) and the Medizinische Hochschule Hannover (local ethics vote 7414/2017). Fifteen human tracheal airway epithelia in the \"Schultze\" dataset were isolated from de-identified donors whose lungs were not suitable for transplantation. Lung specimens were obtained from the International Institute for the Advancement of Medicine Table 1 \n",
      "\n",
      "Publicly available single-cell RNA-seq datasets were downloaded from Gene Expression Omnibus (GEO) 161 . We searched GEO for datasets that met all of the following three criteria: (1) provided unnormalized count data; (2) was generated using the 10X Genomics's Chromium platform 162 ; and (3) profiled human samples. These tissue samples spanned a wide range, including primary tissues, cultured cell lines, and chemically or genetically perturbed samples. Applying these filters increases standardization of sample as the vast majority were prepared using the same 10X Chromium instrument and Cell Ranger pipelines. Datasets comprise of one or more samples (individual gene expression matrices), which often correspond to individual experiments or patient samples. In total, this yielded 2,333,199 cells from 469 samples from 64 distinct datasets (Supplementary Table 1 ). To allow comparison across samples and datasets, we mapped through a common dictionary of gene symbols and excluded unrecognized symbols. If a gene from an aggregated master list was not found in a sample, the expression was considered to be zero for every cell in that sample.\n",
      "\n",
      "After all datasets were collected, we quantified the percentage of cells with >0 UMIs for both ACE2 and TMPRSS2 or ACE2 and CTSL. For further analyses with broad cell classes, we only used datasets with more than 15 double positive cells yielding 252,871 cells from 40 samples.\n",
      "\n",
      "For integration across datasets, we used two levels of annotations. When possible, every sample was annotated with its tissue of origin based on the available metadata from GEO. We excluded any sample for which tissue was not specified. For the smaller subset of 252,871 cells we then . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint manually annotated cell clusters with broad cell type classes using marker genes. These clusters were generated using the harmony-pytorch Python implementation (version 0.1.1 (https://github.com/lilab-bcb/harmony-pytorch 163 ) of the Harmony scRNA-seq integration method 164 for batch correction and leiden clustering from the Scanpy package (version 1.4.5) 165 . Clusters without clear markers distinguishing types were excluded from further analysis.\n",
      "\n",
      "Data was processed using Scanpy. Individual datasets were normalized log (UMIs/10,000 +1) by column sum and the log1p function (ln(10,000 * gij + 1) where a gene's expression profile, g, is the result of the UMI count for each gene, i, for cell j, normalized by the sum of all UMI counts for cell j. This data normalization step was only used for generating the clusters and cell type annotations.\n",
      "\n",
      "All other statistical tests for the integrated analysis were performed on the cell's binary classification as a double positive or not. For example, for a cell to be considered ACE2+, it has >0 ACE2 transcripts. Double positive cells have >0 transcripts for both genes of interest. We used Fisher's exact test to test for statistical dependence between the expression of ACE2 and TMPRSS2 or CTSL and corrected for multiple testing via Benjamini-Hochberg over all tests for each gene pair.\n",
      "\n",
      "We compiled a compendium of published and unpublished datasets consisting of 2,433,890 cells from 21 tissues and/or organs including adipose, bone marrow, brain, breast, colon, cord blood, enteric nervous system, esophagus mucosa, esophagus muscularis, anterior eye, heart, kidney, liver, lung, nasal, olfactory epithelium, pancreas, placenta, prostate, skeletal muscle and skin. After the harmonization of cell type annotations, ACE2-TMPRSS2 and ACE2-CTSL coexpression were assessed using a logistic mixed effect model:\n",
      "\n",
      "where Yi was the binarized expression level of either TMPRSS2 or CTSL, and covariates were binarized ACE2 expression in cell i and a sample-level random intercept.\n",
      "\n",
      "Models were fit separately for each cell type in each dataset. In order to avoid spurious associations in cell types with very few ACE2 + cells and due to very low expression of ACE2, we subsampled ACE2cells to the number of ACE2 + cells within each cell type and discarded cell types containing fewer than 5 cells expressing either ACE2 or fewer than 5 cells expressing the other gene being tested after the subsampling procedure. The significance of the association between ACE2 and TMPRSS2/CTSL is controlled for 10% FDR using the statsmodels Python package (version 0.11.1) 166 . Data processing was performed using Scanpy Python package (version 1.4.6) 165 and logistic models were fit using lme4 R package (version 1.1.21) 167 .\n",
      "\n",
      "Library Generation and Sequencing. Libraries were generated using the 10x Chromium Controller and the Chromium Single Cell ATAC Library & Gel Bead Kit (#1000111) according to the manufacturer's instructions (CG000169-Rev C; CG000168-Rev B) with unpublished modifications relating to cell handling and processing. Briefly, human lung derived primary cells were processed in 1.5ml DNA LoBind tubes (Eppendorf), washed in PBS via centrifugation at 400g, 5 min, 4C, lysed for 3 min on ice before washing via centrifugation at 500g, 5 min, 4C. The supernatant was discarded and lysed cells were diluted in 1x Diluted Nuclei buffer (10x Genomics) before counting using Trypan Blue and a Countess II FL Automated Cell Counter to validate lysis. If large cell clumps were observed, a 40µm Flowmi cell strainer was used prior to the tagmentation reaction, followed by Gel Bead-In-Emulsions (GEMs) generation and linear PCR as described in the protocol. After breaking the GEMs, the barcoded tagmented DNA was purified and further amplified to enable sample indexing and enrichment of scATAC-seq libraries. The final libraries were quantified using a Qubit dsDNA HS Assay kit (Invitrogen) and a High Sensitivity DNA chip run on a Bioanalyzer 2100 system (Agilent).\n",
      "\n",
      "All libraries were sequenced using Nextseq High Output Cartridge kits and a Nextseq 500 sequencer (Illumina). 10x scATAC-seq libraries were sequenced paired end (2 x 72 cycles).\n",
      "\n",
      "Initial data processing and QC. Fastq files were demultiplexed using 10x Genomics CellRanger ATAC mkfastq (version 1.1.0). We obtained peak-barcode matrices by aligning reads to GRCh38 (CR v1.2.0 pre-built reference) using CellRanger ATAC count. Peak-barcode matrices from six channels were normalized per sequencing depth and pooled using CellRanger ATAC aggr.\n",
      "\n",
      "The aggregated, depth-normalized, filtered dataset was analyzed with Signac (v0.1.6, https://github.com/timoast/signac), a Seurat 168 extension developed for the analysis of scATAC-seq data. All the analyses in Signac were run with a random number generator seed set as 1234. Cells that appeared as outliers in QC metrics (peak_region_fragments ≤ 750 or peak_region_fragments ≥ 20,000 or blacklist_ratio ≥ 0.025 or nucleosome_signal ≥ 10 or TSS.enrichment ≤ 2) were excluded from the analysis.\n",
      "\n",
      "Normalization and dimensionality reduction. The aggregated dataset was processed with Latent Semantic Indexing 169 , i.e. datasets were normalized using term frequency-inverse document frequency (TF-IDF), then singular value decomposition (SVD), ran on all binary features, was used to embed cells in low-dimensional space. Uniform Manifold Approximation and Projection (UMAP) 170 was then applied for visualization, using the first 30 dimensions of the SVD space.\n",
      "\n",
      "Gene activity matrix and differential motif activity analysis. A gene activity matrix was calculated as the chromatin accessibility associated with each gene locus (extended to include 2kb upstream of the transcription start site, as described in the vignette 'Analyzing PBMC scATAC-seq' (version: March 13, 2020, https://satijalab.org/signac/articles/pbmc_vignette.html), using as gene annotation the genes.gtf file provided together with Cellranger's atac GRCh38-1.2.0 reference genome.\n",
      "\n",
      "Clusters were annotated using label transfer from matching scRNA samples or by literature / expert search of marker \"active\" (i.e. accessible) genes. Differential motif activity analysis was performed using Signac's implementation of ChromVAR 93 , with motif position frequency matrices from . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint JASPAR2020 171 (http://jaspar.genereg.net/) selecting transcription factors motifs from human (species=9606), broadly following the vignette 'Motif analysis with Signac' (https://satijalab.org/signac/articles/motif_vignette.html). Cells were identified as positive for ACE2 and/or TMPRSS2 (i.e. with the loci accessible) if at least one fragment was overlapping with the gene locus or 2kb upstream. Differential activity scores between epithelial cells positive for ACE2 (with the above-mentioned definition of 'positive') and non-expressing ACE2 was performed with the FindMarkers function of Seurat (version 3.1.1), using as test 'LR' (i.e. logistic regression) and as latent variable the number of counts in peak.\n",
      "\n",
      "The following publically available bulk-RNAseq datasets were obtained from the ENCODE database: LID20728; LID20729; LID47030; LID47031; LID47036; LID47037; LID21040; LID21041; LID47032; LID47033; LID20870; LID20871; LID20872; LID20873; LID21183; LID21184; LID21042; LID21043; LID20920; LID20921; LID20924; LID20925; LID20821; LID20822; LID46983; LID46984; LID20819; LID20820; LID21038; LID21039; LID20732; LID20733; LID20868; LID20869; LID20922; LID20923, generated by the Gingeras lab 172 . These FASTQ datasets were aligned to the MM9 annotation build using STAR and processed using the standard Tuxedo Suite to yield a normalized FPKM matrix.\n",
      "\n",
      "Immunohistochemistry analysis was performed on 4% PFA fixed, OCT embedded tissue sections from human explant donors. Briefly, cells were permeabilized with 5% Triton-X in PBS. Slides were either not treated with antigen retrieval or antigen retrieval was performed with the Tris-Citrate buffer as needed (Supplementary Table 4) . Slides were incubated overnight with primary antibodies at indicated concentrations (Supplementary Table 4 ) in donkey serum with 5% Triton-X in PBS. Slides were treated with Alexa-Fluor secondary antibodies mixed with DAPI at 1:500 concentration in donkey serum with 5% Triton-X in PBS for 1 hour at room temperature. Slides were mounted and imaged on a confocal microscope.\n",
      "\n",
      "Proximity ligation in situ hybridization (PLISH) was performed as described previously 173 . Briefly, frozen human trachea and distal lung sections were fixed with 4.0% paraformaldehyde for 20 min, treated with protease (20 µg/mL proteinase K for lung or Pepsin for trachea for 9 min) at 37°C, and dehydrated with up-series of ethanol. The sections were incubated with gene-specific oligos (Supplementary Table 11 ) in hybridization buffer (1 M sodium trichloroacetate, 50 mM Tris [pH 7.4], 5 mM EDTA, 0.2 mg/mL heparin) for 2 h at 37°C. Common bridge and circle probes were added to the section and incubated for 1 h followed by T4 ligase reaction for 2 h. Rolling circle amplification was performed by using phi29 polymerase (#30221, Lucigen) for 12 hours at 37°C. Fluorophore-conjugated detection probe was applied and incubated for 30 min at 37°C followed by mounting in medium containing DAPI.\n",
      "\n",
      "To assess the association of age, sex, and smoking status with the expression of ACE2, TMPRSS2, and CTSL, we aggregated 22 scRNA-seq datasets of healthy human nasal and lung cells, as well as fetal samples. Aggregation of these datasets was enabled by harmonizing the cell type labels of . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint individual datasets within Scanpy 165 (version 1.4.5.1). We harmonized annotations together with data contributors using a preliminary ontology generated on the basis of 5 published datasets 63-66,174 with 3 levels of annotations (level 1 -lowest resolution; Supplementary Table 2) . We further harmonized metadata by collapsing the smoking covariate into \"has smoked\" and \"has never smoked\" and by taking mean ages where only age ranges were given. This endeavor produced a dataset of 1,176,683 cells in 282 samples from 164 donors (Supplementary Data D3) . We divided the data into fetal (136, To get an overview of sample diversity, we clustered the samples using the proportion of cells in level 2 cell types as features. Clustering was performed using louvain clustering (resolution 0.3; louvain package version 0.6.1) on a knn-graph (k=15) computed on Euclidean distances over the top 5 principal components of the cell type proportion data within Scanpy. This produced five clusters. Sample cluster labels were assigned based on metadata for anatomical location that was obtained from the published datasets and via input from the data generators.\n",
      "\n",
      "Within adult datasets we modeled the association of age, sex, and smoking status with gene expression for ACE2, TMPRSS2, and CTSL using a generalized linear model with the log total counts per cell as offset and Poisson noise as implemented in statsmodels 166 (version 0.11.1) and using a Wald test from Diffxpy (www.github.com/theislab/diffxpy; version 0.7.3, batchglm version 0.7.4). Specifically we used the model:\n",
      "\n",
      "Here, \"5 denotes the raw count expression of gene i in cell j and age:sex, sex:smoking, and age:smoking are interaction terms of the three modeled covariates. These terms model whether there is a difference in the smoking effect in men and women, and likewise whether the age effect is different for smokers and non-smokers. While we model these interaction terms, we only tested age, sex, and smoking effects individually to reduce the multiple testing burden. We included the dataset term to model the batch effects between the diverse datasets we obtained, and the log total counts per cell was used as an offset. Here, the total counts were scaled to have a mean of 1 across all cells before the log was taken. In order to fit this model we pruned the data to contain only datasets that have at least 2 donors and for which smoking status metadata was provided. This resulted in a dataset of 761,693 cells and 165 samples from 77 donors for adult lung data. Only a single dataset remained for adult nasal data after this filtering on which the model could not be fit. To obtain cell-type specific associations the above model was fit within each cell type for all cell types with at least 1,000 cells. We performed Wald tests over the age and sex covariates independently and corrected for multiple testing via Benjamini-Hochberg over all tests within a cell type.\n",
      "\n",
      "As metadata on smoking status was only available for a subset of the data, we also fitted a simpler model on a larger dataset to confirm sex and age associations. The simplified model:\n",
      "\n",
      ". CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint was fit on 986,342 cells in 225 samples from 125 donors of adult lung data. Again, log total counts (scaled) was used as an offset.\n",
      "\n",
      "The above models treat cells as independent observations and thus model cellular and donor variation jointly. As donor variation tends to be larger than single-cell variation, when most cells come from few donors (either there are few donors, or few donors contribute most of the cells), this can lead to an inflation of p-values. To counteract this effect, we verified that significant associations are consistent when modeling only donor variation via pseudo-bulk analysis, and we tested whether effects are dependent on few donors by holding out datasets.\n",
      "\n",
      "Pseudo-bulk data was generated by computing the mean for each gene expression value and nUMI covariate for cells in the same cell type and donor. After filtering as described above, models (2) and (3) were fit to the data. In contrast to the single-cell model, pseudo-bulk analysis underestimates certainty in modeled effects as uncertainty in the pseudo-bulk means are not taken into account. Thus, we used only effect directions from pseudo-bulk analysis to validate singlecell associations. We regarded only those associations as significant, where the FDR-corrected pvalue in the single-cell model is below 0.05, and the sign of the estimated effect is consistent in both the single-cell and the pseudo-bulk analysis.\n",
      "\n",
      "We further separated significant associations into robust trends and indications depending on the holdout analysis. A significant association was regarded as a robust trend if the effect direction is consistent when holding out any dataset when fitting the model irrespective of p-value. In the case that holding out one dataset caused the maximum likelihood estimate of the coefficient to be reversed, we denote this as the effect no longer being present, which characterized the association as an indication.\n",
      "\n",
      "For each of the lung, nasal, and gut datasets, we labeled the cells with non-zero counts for both ACE2 and TMPRSS2 as dual-positive cells (DPs), and the cells with zero counts for both ACE2 and TMPRSS2 as dual-negative cells (DNs). Within each tissue, we identified cell types with greater than 10 DPs, and for each of these cell types, we selected the genes with increased expression (log fold change greater than 0) in DPs vs DNs (so that we focus on important \"positive\" features). We trained a classifier with 75:25 train:test split to classify the DPs from DNs within each of these cell types using the sklearn (version 0.21.3) 175 RandomForestClassifier function with the following parameters: n_estimators set to 100, the criterion as gini, and the class_weight parameter set to balanced_subsample. We first trained individual classifiers separately for each of the cell types, and pooled genes with positive feature importance values (using the feature_importance 176 field in the trained RandomForestClassifier object) to train a final DP vs DN classifier across each tissue. We used the top 500 genes, as ranked by their feature importance scores, to define the signature for the gene expression program of DPs for the tissue. This procedure was carried out in lung, nasal, and gut datasets, yielding tissue-specific signatures for gene expression programs of DPs from each tissue. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint\n",
      "\n",
      "For visualization purposes only, we generated network diagrams using the networkx (version 2.2) 177 tool with the ForceAtlas2 graph layout algorithm 178 . We scored genes that appeared in signatures for multiple tissues by their aggregated feature importance (using a plotting heuristic that used the sum of importance ranks for genes in individual tissues and by assigning a large valued rank (10000) to a gene that did not appear in a particular tissue) and selected the top 10 genes that were shared by each pair of tissues or shared by all tissues along with additional genes that included the ones unique to each tissue's signature to plot in the network visualization. The GO terms enriched in the gene expression programs shared by DPs across tissues were found using gprofiler (version 1.0.0) 179 using the scanpy.queries.enrich tool.\n",
      "\n",
      "This analysis was performed in two ways: on the original data, as well as after accounting for differences in distribution of the number of UMIs (nUMI) per cell between DPs and DNs. This was done by binning the nUMI distribution in the DPs for each tissue into a 100 bins and then randomly sampling from the nUMI distribution for the DNs in each bin to match the distribution of the DPs in that bin. The nUMI distributions before and after the matching are shown in Extended Data Fig. 10b .\n",
      "\n",
      "In parallel, we used a regression framework to recover gene modules enriched in DP vs. DN cells (Fig. 4c,d, Extended Data Fig. 11a,b) in the nasal, lung, and gut datasets. We first restricted our analysis to cell subsets derived from at least two donor individuals that each contained a mixture of DN and DP cells (Nawijn Nasal: multiciliated, Goblet; Regev/Rajagopal Lung: AT1, AT2, Basal, multiciliated, Secretory; Aggregated Lung: AT2, multiciliated, Secretory; Regev/Xavier Colon: BEST4 + Enterocytes, Cycling TA (Transit Amplifying), Enterocytes, Immature Enterocytes 2, TA-2). For each of these cell subsets, we then used MAST (version 1.8.2) 180 to fit the following regression model to every gene with cells as observations:\n",
      "\n",
      "where Yi is the expression level of gene i in cells, measured in units of log2(TP10K+1), X is the binary co-expression state of each cell (i.e. DP vs. DN), and S is the donor that each cell was isolated from. To control for donor-specific effects (i.e. batch effects), we used a mixed model with a random intercept that varies for each donor. To fit this model, we subsampled cells from DP and DN groups to ensure that both the donor distribution and the cell complexity (i.e. the number of genes per cell) were evenly matched between the two groups, as follows. First, for each subset, we restricted our analysis to donors containing at least two DN and two DP cells. Using these samples, we partitioned the cells into 10 equally-sized bins based on cell complexity and subsampled DN cells from each bin to match the cell complexity distribution of the DP cells. Finally, we fit the mixed model (above), controlling for both donor and cell complexity.\n",
      "\n",
      "To build gene modules for DP cells, we prioritized genes by requiring that they be expressed in at least 10% of DP cells, and to have a model coefficient greater than 0 with an FDR-adjusted pvalue less than 0.05 (for the combined coefficient in the hurdle model). After this filtering step, genes were ranked by their model coefficient (i.e. estimated effect size). The top 12 genes were selected for network visualization within each cell type (Fig. 4c,d, Extended Data Fig. 11a,b) . In The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint three cases (gut Cycling TA, TA-2 and BEST4 + cells), RP11-* antisense genes were flagged and excluded from visualizations. To visualize overlap across each network, we indicated whether each gene was among the top 250 genes from each of the other cell types. Putative drug targets were identified by querying the Drugbank database 88 . Gene set enrichment analysis was performed using the R package EnrichR (version 1.0) 181 , selecting the top 25 genes from each cell type for the pan-tissue analysis (\"All\" category; Fig. 4e ), and the top 50 genes from each cell type for the tissue-specific analyses (\"Gut\", \"Nasal\", and \"Lung\" categories; Fig. 4e ). We note a few caveats/challenges/limitations that may influence our results, including non uniform sampling across donors; variation in cell compositions across regions (e.g., distal lung vs carina), and additional cellular heterogeneity that the current level of broad subset annotation may not have been captured.\n",
      "\n",
      "CellphoneDB 96 v.2.0.0 was run with default parameters on the 10 human lung samples of the Regev/Rajagopal dataset, analyzing the cells from each dissected region separately. For each sample (patient/location combination), for each cell type we distinguished double positive cells (ACE2 > 0 and TMPRSS2 > 0) from all others. Only interactions highlighted as significant, i.e. present in the \"significant means\" output from CellphoneDB were considered.\n",
      "\n",
      "ACE2-protease co-expression (Figure 6, Extended Data Fig. 14) and ACE2-IL6/IL6R/IL6ST coexpression (Extended Data Fig. 12) were tested via the logistic mixed-effects model described in \"Integrated co-expression analysis of high resolution cell annotations across tissues\" (Equation 1, above).\n",
      "\n",
      "Data and an interactive analysis examining the co-expression of genes across datasets can be accessed via the open-source data platform, Terra at https://app.terra.bio/#workspaces/kcoincubator/COVID-19_cross_tissue_analysis.\n",
      "\n",
      "Interactive visualization and download of gene expression data can be accessed on the Single Cell Portal at https://singlecell.broadinstitute.org/single_cell?scpbr=hca-covid-19-integrated-analysis\n",
      "\n",
      "N.K. was a consultant to Biogen Idec, Boehringer Ingelheim, Third Rock, Pliant, Samumed, NuMedii, Indaloo, Theravance, LifeMax, Three Lake Partners, Optikira and received nonfinancial support from MiRagen. All of these outside the work reported. J.L. is a scientific . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint consultant for 10X Genomics Inc A.R. is a co-founder and equity holder of Celsius Therapeutics, an equity holder in Immunitas, and an SAB member of ThermoFisher Scientific, Syros Pharmaceuticals, Asimov, and Neogene Therapeutics O.R.R., is a co-inventor on patent applications filed by the Broad Institute to inventions relating to single cell genomics applications, such as in PCT/US2018/060860 and US Provisional Application No. 62/745,259. A.K.S. compensation for consulting and SAB membership from Honeycomb Biotechnologies, Cellarity, Cogen Therapeutics, Orche Bio, and Dahlia Biosciences. S.A.T. was a consultant at Genentech, Biogen and Roche in the last three years. F.J.T. reports receiving consulting fees from Roche Diagnostics GmbH, and ownership interest in Cellarity Inc. L.V. is funder of Definigen and Bilitech two biotech companies using hPSCs and organoid for disease modelling and cell based therapy.\n",
      "\n",
      ". CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint (orange), and CTSL (green) in epithelial cells. Effect size (x axis) of the association, in log fold change (sex, smoking status) or slope of log expression with age. Colored bars: associations with an FDR-corrected p-value<0.05, where pseudo-bulk analysis shows a consistent effect direction. Error bars: standard errors around coefficient estimates. (e) Distribution of ACE2 and TMPRSS2 expression across level 3 lung cell types. Red shading indicates the main cell types that express ACE2 and TMPRSS2. (f) Hold out analysis shows the robustness of associations to holding out a dataset. The values show the number of held-out datasets that result in loss of association between a given covariate (rows) and ACE2, TMPRSS2, or CTSL expression in a given cell type (columns). Robust trends are determined by significant effects that are robust to holding out any dataset (0 values). (g) Low expression in pediatric samples. Mean expression level (log CPM, y axis) of ACE2 (blue), TMPRSS2 (orange), and CTSL (green) across age bins (x axis) in AT2 (left) and ciliated (right) cells. Pediatric samples: 0-10 years. Samples from past or current smokers were removed from this plot to avoid smoking confounders. Error bars are omitted due to y-axis limitations. They are typically 10-fold the mean value (Supplementary Table 5 ). Multiciliated and AT2 cells are shown as these cell types are present in fetal data, and show significant age associations with ACE2 expression. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint UMAP of scRNA-seq profiles (dots) colored by experimental group (c) or by Ace2 + cells and indicated double positive cells (d). Alveolar epithelial cells (AT1 and AT2) and airway epithelial secretory and ciliated cells are marked. (e) The relative frequency of Ace2 + cells is increased by smoking in airway secretory cells but not AT2 cells. Relative proportion (y axis) of Ace2 + (red) and Ace2 -(grey) cells in smoking and control mice of different cell types (x axis). (f, g) Expression of Ace2 is increased in airway secretory cells, but not in AT2 cells. Distribution of Ace2 expression (y axis) in secretory (f) and AT2 (g) cells from control and smoking mice (x axis). (h-j) Re-analysis of published bulk mRNA-Seq 101 of lungs exposed to different daily doses of cigarette smoke show increased expression of (h) Ace2, (i) Tmprss2, and (j) Ctsl after five months of chronic exposure. Extended Data Figure 6 . Age, sex, and smoking status associations with expression of ACE2, TMPRSS2, and CTSL across level 2 cell type annotations. Effect size (y axis) of association as log fold changes (sex, smoking status) and slope of log expression with age. Bars that are colored in indicate associations with an FDR-corrected p-value of < 0.05 where the pseudo-bulk analysis shows a consistent effect direction. Error bars represent model uncertainties. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint Extended Data Figure 7 . Age, sex, and smoking status associations with expression of ACE2, TMPRSS2, and CTSL across level 3 cell type annotations. Effect size (y axis) of the association as log fold changes (sex, smoking status) and slope of log expression with age. Bars that are colored in indicate associations with an FDR-corrected p-value of < 0.05 where the pseudo-bulk analysis shows a consistent effect direction. Error bars represent model uncertainties. Comparison of signature scores of cell programs between DP and DN cells for each cell type stratified by gene complexity bin. Cells were partitioned into 10 gene complexity bins for every cell type. (d,e) IL6 and its receptor's expression in specific cell types in lung and heart. (d) Significance (dot size) and fold change (dot color) of differential expression between DP and DN cells within different types (rows) for IL6 and its receptors IL6R and IL6ST (columns) across tissues. (e) Top: Significance (dot size) and fold change (dot color) of differential expression between DP and DN cells within different cell types in the heart (rows)for IL6 and its receptors IL6R and IL6ST (columns). Bottom: Significance (dot size) and effect size (dot color) from a mixed . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint effects model of co-expression of IL6, IL6R, or IL6ST (columns) coexpression with ACE2. (f) Distribution of number of counts in peaks (y axis) in ACE2 + epithelial cells (having at least 1 fragment in the ACE2 gene locus) and ACE2cells. Figure 12. Co-expression of ACE2 and IL6,IL6R,IL6ST . Co-expression of ACE2 and IL6,IL6R,IL6ST in select single-cell datasets. P-values and significance (FDR 10%) derived from the logistic mixed-effects model. Figure 13 . Expression of Ace2, Tmprss2 and Ctsl in mouse placenta. UMAP embedding (as in Fig. 5k ) of scRNA-seq profiles of placenta cells collected at E.14.5 (top) or along a time course (bottom), colored by expression level ofAce2, Tmprss2, and Ctsl. Extended Data Figure 14 . Additional analyses to identify other proteases that may have a role in infection. (a) Multiple proteases are co-expressed with ACE2 in another human lung scRNA-seq (\"aggregated lung\"). Scatter plot of significance (y axis, -log10(adjusted p value)) and effect size (x axis) of co-expression of each protease gene (dot) with ACE within each indicated epithelial cell type (color). Dashed line: significance threshold. TMPRSS2 and PCSKs that significantly coexpressed with ACE2 are marked. (b) ACE2-protease co-expression with PCSKs, TMPRSS2 and CTSL across lung cell types (\"aggregated lung\"). Significance (dot size, -log10(adjusted p value)) and effect size (color) for co-expression of ACE2 with selected proteases (columns) across cell types (rows). (c-d) Predicted cleavage sites in the SARS-CoV-2 S-protein S1/S2 region. (c) Multiple amino acid sequence alignment of SARS-CoV-2 S-protein S1/S2 region with orthologous sequences from other betacoronaviruses (top) and polybasic cleavage sites of other human pathogenic viruses (bottom). The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint Significance (dot size) and effect size (dot color) of co-expression of each protease (columns) with ACE2 in each cell subset (rows). Figure 16 . Variation in fraction of ACE2 + TMPRSS2 + cells The normalized fraction of ACE2 + TMPRSS2 + cells in 282 lung and nasal samples from 164 donors, subdivided by cell type. Normalization was performed by dividing the fraction by the median total counts per cell in the cell type of the sample. Samples are grouped by dataset and ordered by donor age within each dataset (blue bars at the top). Datasets are ordered by mean age of donors. White patches indicate the cell type annotation was not observed in the sample. This occurs either due to coarseness of annotation, or absence of cell type in the sample.\n",
      "\n",
      "Kenji The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.19.049254 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 5th best search\n",
      "Title : Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China\n",
      "Although the outbreak of Coronavirus disease 2019 has caused over 2200 deaths in China, there was no study about death yet. We aimed to describe the clinical characteristics of non-survivors with COVID-19.\n",
      "\n",
      "For this retrospective, single-center study, we included 36 non-survivors with COVID-19 in the Fifth Hospital of Wuhan. Cases were confirmed by real-time RT-PCR between Jan 21 and Feb 10, 2020 according to the recommended protocol. The epidemiological, demographic, clinical, laboratory, radiological and treatment data were collected and analyzed. Outcomes were followed up until Feb 14, 2020 . This study was approved by the ethics commissions of the Fifth Hospital of Wuhan, with a waiver of informed consent due to a public health outbreak investigation.\n",
      "\n",
      "We included 36 patients who died from COVID- 19 author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the \n",
      "\n",
      "Lots of patients died from COVID-19 till now. The median survival time of these non-survivors from onset to death was about 2 weeks. Most patients were older males with comorbidities. They finally progressed to ARDS. The median time from onset to ARDS was 11 days. Gradually decreased lymphocytes and increased inflammation biomarkers were common, and need to be monitored in the routine treatment.\n",
      "\n",
      "There is no any funder involved in this study.\n",
      "\n",
      "The outbreak of Coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) happened since December 2020 in Wuhan 1 . 2 SARS-CoV-2 has been spreading in China as well as in other countries [3] [4] [5] [6] . The outbreak of SARS-CoV-2 infection has been declared as a Public Health Emergency of International Concern by the World Health Organization (WHO) on 30 January 2020. According to the situation report from WHO, as of 21 February 2020, SARS-Cov-2 infection has caused 76769 confirmed cases in 27 countries and 2247 deaths globally 7 .\n",
      "\n",
      "Severity of COVID-19 varied from mild to severe. 8 Mortality rate of COVID-19 can be reduced, by timely identifying those who are at higher risk of developing into critically ill patients, closely monitoring changes of their disease course, and applying intensive care treatment to these patients. However, reports about dead cases of COVID-19 were limited but warranted, considering the large amount of confirmed cases, which is still increasing. In this single-centered study, we retrospectively analysed specific clinical information of 36 non-survivors infected with SARS-CoV-2.\n",
      "\n",
      "For this retrospective, single-center study, we included 36 non-survivors with COVID-19 in the Fifth Hospital of Wuhan. Cases were confirmed between Jan 21 and Feb 10, 2020 according to All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint the recommended protocol. This study was approved by the ethics commissions of hospital, with a waiver of informed consent due to a public health outbreak investigation.\n",
      "\n",
      "COVID-19 was confirmed by real-time RT-PCR using the same protocol described previously 9 .\n",
      "\n",
      "Additionally, all patients were given chest computed tomography (CT). We obtained epidemiological, demographic, clinical, laboratory, treatment and outcome data from patients' medical records. Clinical outcomes were followed up to Feb 14, 2020.\n",
      "\n",
      "We analysed epidemiological, demographics, exposure history, smoking history, chronic medical illness, signs and symptoms on admission, comorbidity, chest CT findings, laboratory findings and treatment measures.\n",
      "\n",
      "Categorical variables were described as count (%). Continuous variables were described using mean (SD) if they are normally distributed, median (IQR) if they are not. All statistical analyses were performed using R (version 3.6.1).\n",
      "\n",
      "All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint\n",
      "\n",
      "We included 36 patients who died from COVID-19 in the Fifth Hospital of Wuhan. One (2.78%) patients had a history of exposure to the Huanan seafood market in Wuhan. The mean age of the patients was 69.22 years (SD 9.64, range 50-90). Twenty-five (69.44%) patients were males, and 11 (30.56%) female. Four (11.11%) patients have smoking history. Twenty-six (72.22%) patients had chronic diseases. The most common chronic diseases were hypertension (21, 58.33%), cardiovascular (8, 22 .22%) and diabetes (7, 19.44%) ( Table 1 ).\n",
      "\n",
      "On admission, most patients had fever, cough and more than half of them had shortness of breath (Table 2 ). Other symptoms included fatigue, dyspnea, sputum production, disturbance of consciousness, diarrhea, hemoptysis and myalgia. (Table 2) According to chest CT scan, 31 (96.88%, N=32) patients showed bilateral pneumonia (75%). On admission, leucocytes were below the normal range in 5 (14.71%) patients and above the normal range in 11 (32.35%) patients (Table 3 ). Neutrophils showed above the normal range in 16 (47.06%) patients. Lymphocytes and hemoglobin were below the normal range in many patients (Table 3) . Twelve (35.39%) patients' platelets were below the normal range. 22 patients had varying degrees of liver function abnormality, with alanine aminotransferase (ALT) or aspartate aminotransferase (AST) rose above the normal range ( Table 3) . Elevation of lactate dehydrogenase were showed in 26 (100%, N=26) patients. About 30% of these patients had different degrees of renal function damage, with elevated blood urea nitrogen or serum creatinine. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint\n",
      "\n",
      "Regarding the infection index, procalcitonin (PCT) rose above the normal range in 28 (100%)\n",
      "\n",
      "patients. 11 patients were tested for IL-6, most of whose index were above the normal range (Table 3) . 32 patients had increased C-reactive protein (CRP).\n",
      "\n",
      "Thirty-five (35, 97.22%) patients received antiviral treatment, including oseltamivir, ganciclovir, ribavirin or umifenovir hydrochloride. All patients were given antibiotic treatment ( Table 2) Table   2 ). The median time from onset to death was 17 days (Figure 1 ).\n",
      "\n",
      "We reported on 36 patients who died from SARS-CoV-2 infection in a single hospital of Wuhan.\n",
      "\n",
      "All these patients had ARDS and 28 (77.78%) patients needed mechanical ventilation. Bilateral pneumonia was detected in most patients on admission. Previous studies have shown that elder patients and males who were confirmed with coronavirus infection are at higher risk of ARDS, 10 All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint which was consistent with our study. Antibiotic treatment was used to prevent secondary infections since these patients were critically ill. Antiviral regimes were also used in most patients which we hoped may have roles but still need more data. However, there were no specific remedies for SARS-CoV-2 yet 11, 12 . Therefore, most patients were treated with supportive treatment and intensive care. Unfortunately, due to restrictions of hospital's conditions and the heavy burden of COVID-19 in Wuhan City, neither were we able to provide patients in need with advanced life support systems like Extracorporeal Membrane Oxygenation (ECMO), nor were able to transport all serious patients to superior hospital for better medical treatments in time. The median survival time of the patients in our study was only 2 weeks approximately, which was shorter compared to previous reports 13 will decrease as the clinical capacity improves.\n",
      "\n",
      "All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint\n",
      "\n",
      "As reported in previous studies, patients who are male, elder or have a history of cerebrovascular disease are at higher risk of SARS-CoV-2 infection 2,14,15 . In our study, we found that the majority of coronavirus non-survivors were male patients, elder patients and patients with chronic diseases, which indicated that these patients might be also at increased risk of becoming critically ill or death. Cerebrovascular diseases were the most common comorbidities. Twentytwo (61.11%) non-survivors had cerebrovascular diseases in our report, and the percentage of cerebrovascular diseases among non-survivors was higher than that reported by Yang et al 13 .\n",
      "\n",
      "Besides, there were 4 (11.11%) non-survivors who had smoking history and 4 (11.11%) had preexisting COPD. According to previous studies on middle east respiratory syndrome coronavirus (MERS-CoV), smokers and COPD patients might be susceptible to MERS. Further, compared to non-smokers, smokers and COPD patients had a higher dipeptidyl peptidase IV (DPP4) expression, which was inversely correlated with lung function and diffusing capacity parameters 16 . Although the relationship between smoking history and susceptibility as well as worse outcomes in COVID-19 remains unclear, we cautioned that the prognosis of COVID-19 in patients with smoking history might be more severe.\n",
      "\n",
      "In our cohort, lymphocytopenia occurred in more than 70% of patients at admission, which is a main laboratory feature in COVID-19 patients 15, 17 . Lymphocytopenia have been identified in the critically ill patients with SARS-CoV and MERS infection 18, 19 . As mentioned in previous studies, the severity of lymphocytopenia might indicate the severity of COVID-19, under the assumption of SARS-CoV-2 viral could attack and destroy the lymphocyte targetedly 2 . Further studies are warranted to confirm these findings. Increased levels of serum CRP, PCT, IL-6 were also found.\n",
      "\n",
      "It indicated the obvious inflammatory response among these patients. In addition, we noticed that All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint the increased CRP, PCT concentration and decreased lymphocyte count from admission to death (Supplementary Figure 2) , which may represent more prominent inflammation in severe patients. Therefore, intravenous glucocorticoids therapy, intravenous immunoglobulin therapy and interferon-alpha (α-IFN) aerosol inhalation were also used to restore homeostasis, without solid evidence. To our knowledge, there is still no specific medicine for COVID-19 till now. Clinical trials on promising regimens for COVID-19, such as remdesivir, lopinavir, and chloroquine phosphate are ongoing, which shed light on conquering the COVID-19 epidemic 12 .\n",
      "\n",
      "This study has several limitations. First, only 36 dead cases were included. However, this is the largest cohort about the non-survivors of COVID-19 up to now. Second, some specific clinical information was insufficient, such as mechanical ventilation settings, oxygen concentration, and detailed medication history. Third, this is a retrospective, single-centered study. It may be limited to the hospital critical care resources. Further studies are still needed.\n",
      "\n",
      "In conclusion, most non-survivors are older men with comorbidities (especially cardiovascular diseases). They finally progressed to ARDS. The median time from onset to ARDS was 11 days.\n",
      "\n",
      "The mean survival time in our cohort of COVID-19 non-survivors was about 2 weeks. Gradually decreased lymphocytes and increased inflammation biomarkers were found in these patients, and need to be monitored in the routine treatment. Early detection and intervention of patients are especially important which can delay the development from mild to severe cases.\n",
      "\n",
      "All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint YH, RY,YX collected the epidemiological and clinical data. YH summarised all data. YH, RY, YX, and PG drafted the manuscript. YH and PG revised the final manuscript.\n",
      "\n",
      "We declare no competing interests.\n",
      "\n",
      "The data will be made available to others on reasonable requests to the corresponding author.\n",
      "\n",
      "Deidentified participant data will be provided after approval from the corresponding author and the Fifth Hospital of Wuhan. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint author/funder, who has granted medRxiv a license to display the preprint in perpetuity. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.27.20029009 doi: medRxiv preprint\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 6th best search\n",
      "Title : Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2\n",
      "In December 2019, a novel respiratory disease emerged in a seafood market in Wuhan, China 1 . Genomic sequencing demonstrated that the causative agent was a highly-contagious coronavirus, since named SARS-CoV-2 2,3 . The disease, called COVID-19, rapidly spread worldwide, and as of March 2020, more than 600,000 people have been infected and more than 25,000 people have died 4 . No clinically-validated treatment or vaccine for COVID-19 is currently available. Thus, understanding the factors that mediate susceptibility to SARS-CoV-2 is crucial for controlling disease transmission. Molecular analysis has begun to shed light on how SARS-CoV-2 infections occur. Like a related coronavirus that emerged in 2003 5 , SARS-CoV-2 enters human cells by binding to the extracellular domain of Angiotensin Converting Enzyme 2 (ACE2) 3,6 . Importantly, ACE2 is both necessary and sufficient for infection by SARS-CoV-2: ACE2-targeting antibodies block viral uptake in permissive cells while transgenic expression of human ACE2 allows viral entry in nonhuman cells. ACE2 normally functions in the renin-angiotensin system (RAS) by cleaving the vasoconstrictive hormone angiotensin-II into the vasodilator angiotensin 1-7 7 . Sequestration of ACE2 by coronavirus dysregulates the RAS pathway, contributing to morbidity 8 . Additionally, ACE2 levels are capable of influencing disease progression: among a cohort of mice engineered to express human ACE2, mice expressing the highest levels of ACE2 mRNA exhibited the shortest survival time following coronavirus exposure 9 . Thus, the regulation of ACE2 expression likely has a significant effect on SARS-CoV-2 susceptibility.\n",
      "\n",
      "Epidemiological studies have identified several demographic features that correlate with the severity of clinical COVID-19 cases. While fewer than 5% of SARS-CoV-2 infections are fatal 10 , men and elderly patients are particularly at risk of developing severe disease 11-14 . Additionally, cigarette smokers are highly susceptible to SARS-CoV-2 and are significantly more likely to require aggressive clinical interventions: in a study of 1,099 patients with laboratory-confirmed COVID-19, 12.3% of current smokers required mechanical ventilation, were admitted to an ICU, or died, compared to only 4.7% of non-smokers 12 . The causes underlying these differences in clinical outcome are at present unknown.\n",
      "\n",
      "In order to study factors that could potentially influence susceptibility to SARS-CoV-2 infection, we investigated the expression of the coronavirus receptor ACE2. We first assessed the expression of ACE2 in a variety of post-mortem rodent and human tissues ( Figure S1 and Table  S1 ) 15-18 . ACE2 was consistently expressed at high levels in mouse, rat, and human kidneys, consistent with its role as a regulator of the RAS pathway. ACE2 was also robustly expressed in rodent and human lungs, the predominant location of coronavirus infections. Interestingly, significant ACE2 expression was evident in the mouse and human small intestine. Viral RNA has been detected in stool samples from patients with COVID-19 19 , and gastrointestinal symptoms have been reported in a subset of affected individuals 12 , suggesting a potential alternate route for SARS-CoV-2 transmission. However, as SARS-CoV-2 is primarily spread through viral inhalation, we focused our study on factors that affect ACE2 expression in the lung and associated respiratory tissue.\n",
      "\n",
      "Age and male sex are significant risk factors for SARS-CoV-2 infections. We therefore investigated whether either feature was associated with increased ACE2 expression. Young mice (<26 weeks) and elderly mice (>78 weeks) displayed equivalent levels of ACE2 expression in the lung, as did young rats (6 weeks) and elderly rats (104 weeks)( Figure 1A-B) 17, 20 . Similarly, ACE2 expression in rodent lungs was not significantly different between sexes ( Figure 1C -D) 17,21 . We next assessed the expression of ACE2 in two different human cohorts: 1) lung tissue from the Genotype-Tissue Expression project (GTEx) 15, 16 and 2) whole-lung tissue samples from organ donors 22 . These datasets yielded results that were consistent with our rodent analyses: ACE2 expression was equivalent between men and women and between young individuals (<29 years) and elderly individuals (>70 years)( Figure 1E -H). In total, these findings suggest that the increased morbidity of men and older patients with COVID-19 is unlikely to result from inherent differences in ACE2 expression in the respiratory tract.\n",
      "\n",
      "Cigarette smoking is strongly associated with adverse outcomes from COVID-19 12,23 . To investigate whether smoking could affect ACE2 levels, we first assessed gene expression in mouse lungs. We analyzed a cohort of mice exposed to diluted cigarette smoke for 2, 3, or 4 hours per day for five months 24 . Strikingly, we found a dose-dependent increase in ACE2 expression according to smoke exposure ( Figure 2A ). Mice exposed to the highest dose of cigarette smoke expressed ~80% more ACE2 in their lungs compared to sham-treated mice. To determine whether this association was present in humans as well, we assessed three cohorts of current smokers and never-smokers 25-27 . For these analyses, lung epithelial cells were sampled from either the small or large airways by fiberoptic bronchoscopy. In each cohort, we observed that tissue collected from smokers exhibited ~40%-50% more ACE2 compared to tissue from nonsmokers ( Figure 2B -D).\n",
      "\n",
      "Next, we sought to determine whether human ACE2 expression showed a dose-dependent relationship with cigarette smoke, as we had observed in mice. To investigate this, we analyzed two human datasets: 1) lung tissue from a cohort of smokers undergoing thoracic surgery for transplantation, lung volume reduction, or nodule resection 28 and 2) lung tissue from a cohort of patients analyzed as part of The Cancer Genome Atlas (TCGA) 29 . For patients undergoing cancer-related surgeries, a sample of pathologically-normal tissue distant from the suspected tumor was used for analysis. In both cohorts, lung samples from patients who reported smoking the greatest number of pack-years also expressed the highest levels of ACE2 ( Figure 2E -F). For instance, among smokers undergoing thoracic surgery, patients who had smoked more than 80 pack-years exhibited a ~100% increase in ACE2 expression relative to patients who had smoked less than 20 pack-years ( Figure 2E ). Multivariate linear regression on this dataset further confirmed that smoking history was a significant predictor of ACE2 expression even when controlling for a patient's age, sex, race, and body-mass index (Table 1) . Lastly, we compared the expression level of ACE2 in lung epithelial cells from current smokers and former smokers 30 , and we found that quitting smoking was associated with a 30% decrease in ACE2 expression ( Figure 2F ). In total, our results demonstrate that exposure to cigarette smoke increases the expression of the coronavirus receptor ACE2 in rodent and human respiratory tissue, and this upregulation is potentially reversible.\n",
      "\n",
      "To follow up on these observations, we considered the possibility that ACE2 expression was commonly upregulated by lung diseases and/or carcinogen exposure. Indeed, we observed increased ACE2 expression in two cohorts of patients with idiopathic pulmonary fibrosis (IPF) 31,32 , a lung disease strongly associated with prior cigarette exposure ( Figure S2A -B) 33 . However, ACE2 was not up-regulated in lung cells from a large cohort of patients with asthma or from patients with the lung disease sarcoidosis ( Figure S2C-D) 34, 35 . Similarly, ACE2 expression was unaffected in lung tissue from a mouse model of cystic fibrosis or in mice exposed to a variety of carcinogens, including arsenic, ionizing radiation (IR), and 1,3-butadiene ( Figure S2E -G) 36-39 . We conclude that ACE2 upregulation in the lung is tightly associated with a history of cigarette smoking and is not a universal response to pathological insults.\n",
      "\n",
      "Coronavirus infections are facilitated by a set of host proteases that cleave and activate the viral spike (S) protein 40 . SARS-CoV-2 primarily relies on the serine protease TMPRSS2 but can also utilize an alternate pathway involving Cathepsin B/L in TMPRSS2-negative cells 6 . Interestingly, we observed that Cathepsin B expression, but not TMPRSS2 or Cathepsin L expression, was consistently increased in mice and humans exposed to cigarette smoke ( Figure S3 ). Thus, smoking can upregulate both the coronavirus receptor as well as a protease that SARS-CoV-2 uses for viral activation.\n",
      "\n",
      "Mammalian lungs harbor more than 30 distinct cell types representing a variety of epithelial, endothelial, stromal, and immune compartments 41 . Of note, the upper respiratory epithelium is comprised of mucociliary cells, including goblet cells, club cells, and ciliated cells, that secrete protective fluids and remove inhaled particles from the airways 42 . The lower respiratory epithelium includes alveolar type 1 cells, which allow gas exchange with the blood, and alveolar type 2 cells, which regulate alveolar fluid balance and can differentiate into type 1 cells following injury 43 .\n",
      "\n",
      "To gain further insight into coronavirus infections, we profiled multiple single-cell RNA-Seq experiments to identify the cell type(s) that express ACE2. We first examined a dataset containing 13,822 cells from normal mouse lungs 44 . We performed unsupervised Leiden clustering to separate the cells into distinct populations and then we assigned cell types to major clusters using established markers 45-48 . ACE2 was expressed solely in the EpCAM+ clusters that comprise the lung epithelium 49 , and was not detected in CD45+ immune cells 50 , PDGFRA+ mesenchymal cells 51 , or TMEM100+ endothelial cells ( Figure 3A ) 52 . We therefore focused on localizing ACE2 within the epithelial lineage. We found that ACE2 was strongly expressed in a cluster of cells that express secretory markers including MUC5AC, GABRP, and SCGB1A1 that we identified as being comprised of closely-related goblet and club cells ( Figure 3B -C) 48,53-55 . Additionally, ACE2 expression was observed in a subset of LAMP3+ alveolar type 2 cells 56 , but was largely absent from RTKN2+ alveolar type 1 cells 48 and FOXJ1+ ciliated cells 57 . We next identified the transcripts whose expression correlated with ACE2. Across all cells, ACE2 levels exhibited the strongest correlation with CLDN10 (Claudin-10), a tight-junction protein that contributes to the formation of the bronchial epithelium and that is expressed by club cells and goblet cells ( Figure 3C ) 58,59 . Gene ontology analysis revealed that ACE2-correlated transcripts were enriched for genes involved in xenobiotic metabolism, antioxidant activity, and cellular detoxification, consistent with the bronchial epithelium's role as a barrier against toxins and foreign matter ( Figure 3D and Table S2A ) 60 .\n",
      "\n",
      "To confirm our findings regarding ACE2's localization, we subsequently repeated this analysis on an independent lung single-cell dataset ( Figure S4 ). Consistent with our initial observations, we found that ACE2+ cells localized to clusters of epithelial cells that expressed the goblet and club cell markers CLDN10, GABRP, and SCGB1A1 or the alveolar type 2 marker LAMP3 ( Figure S4A -C). Gene ontology analysis on ACE2-correlated transcripts revealed a similar enrichment for xenobiotic metabolism and cellular detoxification ( Figure S4D and Table S2B ). In total, our combined analysis demonstrates that ACE2 is expressed in secretory club and goblet cells as well as alveolar type 2 cells within the lung epithelium.\n",
      "\n",
      "Our findings suggested a possible explanation for the upregulation of ACE2 in the lungs of cigarette smokers. Chronic exposure to cigarette smoke has been reported to trigger the expansion of secretory goblet cells, which produce mucous to protect the respiratory epithelium from inhaled irritants 61-63 . Thus, the increased expression of ACE2 in smokers' lungs could be a byproduct of smoking-induced secretory cell hyperplasia. To investigate this hypothesis, we examined a dataset of single-cell transcriptomes collected by fiberoptic bronchoscopy from current smokers and never-smokers 59 . Notably, these cells were sorted and enriched for an epithelial cell marker (ALCAM) prior to sequencing. Additionally, they were collected from the main stem bronchus, and subsequently harbor few RTKN2+ or LAMP3+ alveolar cells 59 . This population was therefore highly enriched for upper-respiratory epithelial cells, and allowed us to localize ACE2 within this population with increased precision.\n",
      "\n",
      "Using this purified dataset, we could more easily differentiate between goblet cells (marked by MUC5AC) 53 , club cells (marked by SCGB1A1) 54 , ciliated cells (marked by FOXJ1) 57 , ionocytes (marked by CFTR) 64 , and basal cells (which serve as respiratory epithelial stem cells and are marked by KRT5) 65 . We found that ACE2 expression was observed in both MUC5AC+ goblet cells and SCGB1A1+ club cells, but was largely absent from the basal cell, ciliated cell, and ionocyte clusters ( Figure 4A -B). Next, we separated the cells harvested from current smokers and never-smokers and then analyzed each population separately. Consistent with previous reports 61-63 , we detected an increase in goblet cells in the lungs of current smokers: while 17% of cells from non-smokers expressed the canonical goblet marker MUC5AC 53 , 47% of cells collected from smokers were positive for MUC5AC ( Figure 4C -D). Cells that expressed other mucus-and goblet cell-related genes, including MUC1 67 , AGR2 68 , and SPDEF 69 , were similarly over-represented in the lungs of smokers. KRT5+ basal stem cells were depleted in the smokers' epithelia, likely reflecting their differentiation into secretory cells ( Figure 4C ). However, we note that in this dataset, we observed an increase in the percentage of ACE2+ cells between non-smokers and smokers that was not statistically-significant. This may reflect the small number of cells that were analyzed (~1,000) or the small number of smokers that were profiled (six).\n",
      "\n",
      "Secretory cell differentiation of lung epithelium can be modeled in vitro by culturing cells at an airliquid interface (ALI) 70, 71 . Under appropriate conditions, primary respiratory cells growing at an ALI will undergo mucociliary differentiation into a stratified epithelium consisting of ciliated cells, goblet cells, and club cells 72 . As our single-cell analysis suggested that the coronavirus receptor ACE2 is expressed in secretory goblet and club cells, we investigated whether in vitro mucociliary differentiation increases ACE2 expression. Indeed, in mouse tracheal extracts 73 and primary human lung cells 74 , mucociliary differentiation resulted in a highly-significant upregulation of ACE2 ( Figure 4E -F). Finally, to investigate the link between smoking, differentiation, and ACE2 expression, we examined data from human bronchial epithelial cells cultured at an ALI in which cells were either exposed to clean air or to diluted cigarette smoke 75 . Remarkably, treatment with cigarette smoke during in vitro differentiation resulted in a significant upregulation of ACE2, MUC5AC, MUC1, and other secretory cell markers, relative to cells that were differentiated in clean air ( Figure 4G ). ACE2 expression was increased by ~42% in smoke-exposed cells, comparable to the increases that we observed between the lungs of non-smokers and smokers ( Figure 2 ). In total, our results demonstrate that a subset of lung secretory cells express the coronavirus receptor ACE2, and cigarette smoke promotes the expansion of this cell population.\n",
      "\n",
      "the SARS-CoV-2 receptor ACE2 in their lungs. This upregulation is likely mediated at least in part by the expansion of ACE2+ mucus-secreting goblet cells triggered by chronic smoke exposure. The overabundance of ACE2 in the lungs of smokers may partially explain why smokers are significantly more likely to develop severe SARS-CoV-2 infections that require aggressive medical interventions 12,23 . Furthermore, as quitting smoking is associated with a decrease in ACE2 expression, we speculate that giving up cigarettes may reduce susceptibility to COVID-19.\n",
      "\n",
      "Several contrasting findings exist in the literature on ACE2 and cigarette exposure [76] [77] [78] [79] [80] [81] . In particular, it has been reported that nicotine and/or cigarette smoke has the potential to downregulate ACE2 expression in certain tissues or cell types [76] [77] [78] [79] . In this manuscript, we focused our analysis on factors affecting ACE2 expression in the mammalian lungs and associated respiratory epithelia. We observed a consistent correlation between smoking history and ACE2 expression that was dose-dependent ( Figure 2E -F), that could be recapitulated in mice ( Figure  2A ) and in vitro ( Figure 4G ), and that remained significant when controlling for other demographic variables (Table 1) . Thus, we propose that cigarette smoke causes an upregulation of ACE2 expression in the respiratory tract, though we recognize that smoking may have different effects on ACE2 levels in the heart, kidney, or other organs.\n",
      "\n",
      "Our results concerning the localization of ACE2 agree with and expand on several unpublished reports [82] [83] [84] . Consistent with our findings, these analyses identified a population of ACE2+ cells in the alveolar type 2 compartment. However, as these previous reports predominantly analyzed tissue from the lower respiratory tract, we speculate that they were enriched for alveolar cells relative to cells from the upper respiratory epithelium. By analyzing datasets from both mice and humans, and by including cells collected from bronchial brushing, we also demonstrate the presence of ACE2 in the secretory cells that participate in mucociliary clearance 42 . As these cells line the upper respiratory tract, they may represent the initial site of coronavirus infections, followed by an eventual spread and migration into the alveoli. Moreover, secretory cell hyperplasia is a well-described consequence of prolonged smoke exposure 61-63 , potentially explaining the consistent up-regulation of ACE2 in smokers' lungs. Interestingly, ACE2-knockout mice have been reported to be particularly vulnerable to lung failure following forced acid inhalation 85 . Thus, the ACE2 that we uncovered in mucociliary cells could potentially play a direct role in protecting the airways from inhaled toxins and foreign matter.\n",
      "\n",
      "The factors that mediate susceptibility to SARS-CoV-2 infections are poorly understood. We speculate that the increased expression of ACE2 in the lungs of smokers could partially contribute to the severe cases of COVID-19 that have been observed in this population. In support of this hypothesis, mice that were engineered to express high levels of human ACE2 succumbed to infections with a related coronavirus more quickly than mice that expressed low levels of human ACE2 9 . Nonetheless, the relevance of increased ACE2 expression as a driver of disease susceptibility in humans or for SARS-CoV-2 remains to be demonstrated. Chronic smokers may exhibit a number of co-morbidities, including emphysema, atherosclerosis, and decreased immune function 86 , that are also likely to affect COVID-19 progression. While the effects of smoking can last for years, smoking cessation causes an improvement in lung function and an overall decrease in disease burden 86 . Interestingly, quitting smoking also leads to a normalization of respiratory epithelial architecture 87 , a decrease in hyperplasia 88 , and a downregulation of ACE2 levels. Thus, for multiple reasons, smoking cessation could eventually lessen the risks associated with SARS-CoV-2 infections.\n",
      "\n",
      "The analysis described in this paper was performed using Python, Excel, and Graphpad Prism. Gene expression data was acquired from the Gene Expression Omnibus (GEO) 89 , the GTEx portal 15 , the Broad Institute TCGA Firehose 90 , the Human Cell Atlas 91 , and the Single-Cell Expression Atlas 92 , as described below. For microarray datasets, probeset definitions were downloaded from GEO, and probes mapping to the same gene were collapsed by averaging. For each gene expression comparison, a control population was identified (e.g., young rats, shamtreated mice, non-smokers, etc.), and gene expression values were log2-transformed and normalized by subtraction so that the mean expression of a gene of interest in the control population was 0. Graphs of gene expression values were then generated using Graphpad Prism; all data points are displayed and no outliers were excluded from analysis.\n",
      "\n",
      "The data sources used in this paper are listed in Table S1 . In general, pre-processed microarray and RNA-Seq datasets were downloaded. Additional notes on sample selection and processing are included in Table S1 .\n",
      "\n",
      "Multivariate regression to investigate the relationship between ACE2 expression and smoking history was performed on the GSE76925 lung tissue dataset. Regressions were performed in Python using ordinary least squares from the statsmodels package 93 . Results reported include the standard errors ('bse'), betas, and p-values.\n",
      "\n",
      "Single-cell clustering and analysis on the datasets listed in Table S1 was performed in Python using the Scanpy and Multicore-TSNE packages 45,94 . To filter out low-quality cells, only cells in which 500 or more genes were detected were included in this analysis. Before clustering, transcript counts were log2 transformed. Highly variable genes were selected using the Seurat approach in Scanpy, and these highly variable genes were used to produce the principal component analysis. A t-SNE projection and unsupervised Leiden clustering were then performed on each dataset using nearest neighbors, as described in the associated code.\n",
      "\n",
      "In order to label each cluster, a gene ranking analysis was obtained using Scanpy. The 10 most highly-ranked genes from each cluster (as determined by t-test with overestimated variance) were identified. These genes were then compared against gold-standard marker lists from multiple sources to produce the cluster labels 45-48 .\n",
      "\n",
      "In order to identify the genes whose expression correlates with ACE2, pairwise Pearson correlations coefficients (PCCs) were calculated between ACE2 and every other expressed gene. Genes whose PCC were more than four standard deviations greater than the average gene's PCC were classified as strongly correlated with ACE2. Gene ontology terms enriched in this group were then identified with GProfiler against the background list of non-strongly correlated genes using a Benjamini-Hochberg FDR of .05 95 . (B) Log2-normalized expression of ACE2 in the lungs of young rats (6 weeks old) and old rats (104 weeks old).\n",
      "\n",
      "(C) Log2-normalized expression of ACE2 in the lungs of female mice and male mice.\n",
      "\n",
      "(D) Long2-normalized expression of ACE2 in the lungs of female rats and male rats.\n",
      "\n",
      "(E) Log2-normalized expression of ACE2 in lungs from the GTEx cohort by age.\n",
      "\n",
      "(F) Log2-normalized expression of ACE2 in lungs from the GTEx cohort by sex.\n",
      "\n",
      "(G) Log2-normalized expression of ACE2 in lungs from a cohort of organ donors by age.\n",
      "\n",
      "(H) Log2-normalized expression of ACE2 in lungs from a cohort of organ donors by sex.\n",
      "\n",
      "(A) Log2-normalized expression of ACE2 in the lungs of mice that were sham-treated or that were exposed to diluted cigarette smoke for two, three, or four hours a day (low, medium, and high smoke exposure, respectively).\n",
      "\n",
      "(B) Log2-normalized expression of ACE2 in human small airway epithelia collected by fiberoptic brushing and analyzed according to smoking history (GSE52237).\n",
      "\n",
      "(C) Log2-normalized expression of ACE2 in human small airway epithelia collected by fiberoptic brushing and analyzed according to smoking history (GSE3320).\n",
      "\n",
      "(D) Log2-normalized expression of ACE2 in human large airway epithelia collected by fiberoptic brushing and analyzed according to smoking history.\n",
      "\n",
      "(E) Log2-normalized expression of ACE2 in human lung tissue in a cohort of smokers and analyzed according to the number of pack-years smoked.\n",
      "\n",
      "(F) Log2-normalized expression of ACE2 in human lung tissue in a cohort of lung cancer patients and analyzed according to the number of pack-years smoked.\n",
      "\n",
      "(G) Log2-normalized expression of ACE2 in bronchial epithelia collected by fiberoptic brushing among either current smokers or former smokers. *, p < .05; **, p < .005; ***, p < .0005 (Student's t-test).\n",
      "\n",
      "(A) T-SNE clustering of cells from the mouse lung. Cells expressing ACE2 and various lineage markers (endothelial, epithelial, mesenchymal, and immune) are highlighted.\n",
      "\n",
      "(B) T-SNE clustering of cells from the mouse lung. Cells expressing markers for various epithelial lineages are highlighted.\n",
      "\n",
      "(C) Violin plots display the expression of ACE2 and several lung-related genes in different cell populations obtained from Leiden clustering.\n",
      "\n",
      "(D) Gene ontology enrichment analysis on transcripts whose expression correlates with ACE2. (E) Log2-normalized expression of ACE2 in mouse cells prior to differentiation or following mucociliary differentiation at an air-liquid interface. ***, p < .0005 (Student's t test).\n",
      "\n",
      "(F) Log2-normalized expression of ACE2 in human cells prior to differentiation or following mucociliary differentiation at an air-liquid interface. ***, p < .0005 (Student's t test).\n",
      "\n",
      "(G) Log2-normalized expression of the indicated genes in human cells differentiated in the presence of diluted cigarette smoke or in the presence of clean air. *, p < .05; ***, p < .0005 (Student's t test). (B) The expression of ACE2 in 11 different rat tissues, sorted according to mean ACE2 expression.\n",
      "\n",
      "(C) The expression of ACE2 in 30 different human tissues or cell types, sorted according to mean ACE2 expression. Figure S2 . ACE2 is not commonly upregulated by lung diseases or carcinogen exposure.\n",
      "\n",
      "(A) The expression of ACE2 in archived lung tissue from patients with idiopathic pulmonary fibrosis or from histologically-normal lung tissue from patients with lung cancer (GSE2052).\n",
      "\n",
      "(B) The expression of ACE2 in tissue from the lungs of transplant recipients with idiopathic pulmonary fibrosis or from control donor lungs (GSE47460).\n",
      "\n",
      "(C) The expression of ACE2 in bronchial brushing from individuals with normal lung function, mild asthma, or severe asthma.\n",
      "\n",
      "(D) The expression of ACE2 in disease-free lung tissue resected from patients with sarcoidosis or control tissue from healthy lung donors.\n",
      "\n",
      "(E) The expression of ACE2 in the lungs of a cystic fibrosis mouse model (Cftr -/-) or wild-type littermate controls.\n",
      "\n",
      "(F) The expression of ACE2 in the lungs of mice provided with normal water, water with 10 parts per billion sodium arsenite, or water with 100 parts per billion sodium arsenite (low and high arsenic, respectively).\n",
      "\n",
      "(G) The expression of ACE2 in the lungs of mice exposed to normal air or exposed to the carcinogen 1,3-butadiene.\n",
      "\n",
      "(H) The expression of ACE2 in the lungs of mice that were sham-treated, exposed to 5 Gy of ionizing radiation to the thorax, or exposed to 17.5 Gy of ionizing radiation to the thorax (low and high IR, respectively). *, p < .05; **, p < .005 (Student's t-test).\n",
      "\n",
      "(A) The expression of TMPRSS2, Cathepsin B, and Cathepsin L in the lungs of mice that were sham-treated or that were exposed to diluted cigarette smoke for two, three, or four hours a day (low, medium, and high smoke exposure, respectively). (D) Gene ontology enrichment analysis on transcripts whose expression correlates with ACE2.\n",
      "\n",
      ". CC-BY-NC 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.28.013672 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.28.013672 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.28.013672 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.28.013672 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.28.013672 doi: bioRxiv preprint\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 7th best search\n",
      "Title : Bulk and single-cell transcriptomics identify tobacco-use disparity in lung gene expression of ACE2, the receptor of 2019-nCov\n",
      "In the past two decades, pathogenic coronaviruses (CoVs) have caused epidemic infections, including the server acute respiratory syndrome (SARS)-CoV outbreak in 2003, the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) outbreak in 2012 and the current Wuhan 2019 Novel Coronavirus (2019-nCov) outbreak. They typically affect the respiratory tract and cause severe respiratory illnesses. We have learned from SARS-Cov and MERS-Cov that human populations showed disparities in susceptibility to these viruses. For example, epidemiology studies found that males had higher incidence and mortality rates than females. 1, 2 We believe that the susceptibility to the novel 2019-nCov is also different among population groups. In current severe global emergency situation of 2019-nCov outbreak, it is imperative to identify vulnerable and susceptible groups for effective protection and care.\n",
      "\n",
      "Recently, Xu et.al. computationally modelled protein interactions and identified a putative cell entry receptor of 2019-nCov, angiotensin-converting enzyme 2 (ACE2), which is also a receptor for SARS-nCov. 3 Zhou et.al. further confirmed this virus receptor in the HELA cell line. 4 Interestingly, Zhao et al. found ACE2 is specifically expressed in a subset of type II alveolar cells (AT2), in which genes regulating viral reproduction and transmission are highly expressed. 5 They also found that an Asian male had much higher ratio of ACE2-expressing cell than other seven white and African American donors, which may indicate the higher susceptibility of Asian. However, the sample size was too small to draw conclusion on this racial disparity. Here, we analyzed four large-scale bulk transcriptomic datasets of normal lung tissue to investigate the disparities related to race, age, gender and smoking status in ACE2 gene expression. Also, we analyzed two lung tissue single-cell transcriptomic datasets to investigate the distribution of ACE2 gene expression among cell types, which will provide new knowledge for understanding the mechanism and population disparities of 2019-nCov infection.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint Two RNA-seq datasets and two DNA microarray datasets from lung cancer patients were analyzed in this study, including a Caucasian RNA-seq dataset from TCGA (https://www.cancer.gov/tcga), an Asian RNA-seq dataset from Gene Expression Omnibus (GEO) with the accession number GSE40419 6 , an Asian microarray dataset from GEO with the accession number GSE19804 7 and a Caucasian microarray dataset from GEO with the accession number GSE10072 8 . Both RNA-seq datasets were generated with the Illumina HiSeq platform and both microarray datasets were generated with the Affymetrix GeneChip Human Genome U133 Array. The details and processing of data were described in our previous study 9 . All these datasets contain samples from tumor and normal pairs and we only use the normal samples in this study. In addition, we analyzed a GSE34450 10 microarray dataset of gene expression from small airway epithelium and large airway epithelium of 50 healthy nonsmokers and 71 healthy smokers. In total, 54 samples in the TCGA dataset, 77 samples in the GSE40419 dataset, 60 samples in the GSE19804 dataset, 33 samples in the GSE10072 dataset and 121 samples in the GSE34450 dataset were analyzed. We studied the Reads Per Kilobase per Million mapped reads (RPKM) values for RNA-seq data and Robust Multi-Array Average (RMA) 11 values for microarray data. All data were log2 transformed to improve normality. The means of data values across samples in datasets from the same platform were highly correlated (Pearson correlation coefficient r=0.9 for microarray datasets and r=0.97 for RNA-seq datasets, Fig. S1 ), indicating no significant system variation in datasets from the same platform.\n",
      "\n",
      "Simple linear regressions were used to test the association of ACE2 gene expression with each single variable of age, gender, race and smoking status. And, multiple linear regression was used to test the association of ACE2 expression with multiple factors (age, gender, race, smoking status and data platform). Also, ordinal regression was performed to investigate the association between ACE2 expression and ordinal categorical smoking history. All data management, statistical analyses and visualizations were accomplished using R 3.6.1.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint\n",
      "\n",
      "Two single-cell RNA sequencing (scRNA-seq) datasets available in GEO with accession numbers GSE122960 12 and GSE131391 13 were downloaded and analyzed. The GSE122960 dataset was generated from lung tissue of 8 lung transplant donors (including 1 Asian former smoker, 2 Caucasian current smoker and 5 African American non-smokers), using 3' V2 chemistry kit on 10x Genomics Chromium single cell controller. The GSE131391 study profiled bronchial epithelial cells from 6 never and 6 current smokers using CEL-Seq 14 .\n",
      "\n",
      "Counts of single cells were downloaded, and subsequent data analyses were performed using the Seurat 3.0 package 15 , including data normalization, high variable feature selection, data scaling, dimension reduction and cluster identification. We also used SCANNER 16 to assist the data visualization and cell type identification.\n",
      "\n",
      "Inconsistent with the study of Zhao et al. 5 , we observed no significant difference in ACE2 expression in Caucasian lung tissue samples compared to Asian lung tissue samples in the RNAseq datasets (p-value=0.45, Fig 1A) . In the microarray datasets, a higher ACE2 expression was observed in Caucasian samples compared to Asian samples (p-value=0.001, Fig 1A) . Given that the GSE19804 microarray study focused on female nonsmokers while the GSE10072 dataset includes samples from both males and females and both smokers and non-smokers, we believe that the observed disparity may be due to other factors other than race, such as smoking, gender and unknown factors. Therefore, we performed multiple linear regression on multiple independent variables (age, gender, race, smoking status and platform) and found no significant difference between racial groups (p-value=0.36, Fig. 1B ).\n",
      "\n",
      "Furthermore, we didn't observe a disparity between age groups (>60 vs <60) or gender groups (male vs female) in ACE2 gene expression in each available dataset (Fig.   1C, D) . Consistently, multivariate analysis didn't detect a significant difference between . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint groups of age or gender after other variables (age/gender, race, smoking status and platforms) were adjusted (p-value=0.90 for age, p-value=0.35 for gender, Fig. 1B) . We also consistently found no difference between male and female healthy lung tissue samples from GTEx 17 (Fig. S2 ).\n",
      "\n",
      "We found a significant higher ACE2 gene expression in smoker (including current smoker and former smoker) samples compared to non-smoker samples in the TCGA (p-value=0.05) and GSE40419 RNA-seq datasets (p-value=0.01, Fig. 2A ). Smokers in GSE10072 showed a higher mean of ACE2 gene expression than non-smokers. The difference is not significant (p-value=0.18), which may be due to the small sample size of this study (n=33) with insufficient power to detect the difference. The GSE19804 data which has only non-smoker samples available was not included into the analysis. Adjusted by other factors (age, gender, race and platforms) in multivariate analysis, smoking still shows a significant disparity in ACE2 gene expression (p-value=0.01, Fig. 1B) . These data were from the normal lung tissue of patients with lung adenocarcinoma, which may be different with the lung tissue of healthy people. Therefore, we also analyzed a gene expression dataset of airway epithelium from healthy smokers and healthy non-smokers. Consistently, we observed a significant upregulation of ACE2 gene expression in both large airway epithelium and small airway epithelium of smokers (p-value=4.99E-4 and 7.02E-3, respectively, Fig. 2C ). Furthermore, we investigated ordinal categorized smoking history (non-smoker, former smoker quit more than 15yrs, former smoker quit less than15yrs, and current smoker) and showed results in Figure 2B . In the TCGA dataset, we found a significant trend of ACE2 gene expression regulation associated with the smoking history that current smokers had the highest expression, non-smokers had the lowest expression and former smokers had that inbetween (ordinal regression p-value=0.01, Fig. 2B ). We found a similar but non-significant trend in the GSE10072 due to its small size (ordinal regression p-value=0.11). We didn't observe such a tread in the TCGA dataset. Instead, we found a higher average expression in recent quitters (<=15 years) compared to non-smokers, current smokers and former . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint smokers who have quit for longer durations (>15 years). This may indicate a difference in ACE2 gene expression between Caucasian and Asian current smokers, but it is not statistically significantly detected due to the limited sample size in this study (p-value=0.43).\n",
      "\n",
      "Compared to non-smokers, multivariate analysis on all data showed a significant higher ACE2 expression in former smokers (p-value=0.04) and a higher mean of ACE2 expression in current smokers, which did not reach statistical significance from current analysis though (p-value=0.11).\n",
      "\n",
      "Duclos G et.al. studied human bronchial epithelial cells using single-cell RNA sequencing and found smokers showed a remodeled cell composition in bronchial epithelium with a loss of club cells and extensive hyperplasia of goblet cells. 14 We confirmed their analysis in this study based on the same set of cell makers, including KRT5 for basal cells, FOXJ1 for ciliated cells, SCGB1A1 for club cells, MUC5AC for goblet cells and CD45 for WBCs (white blood cells) (Fig. 3A,C, Fig. S3) . We also identified a smoking related basal cell subpopulation which might be pro-goblet precursor cells and has been discussed in the study of Duclos G et.al. 14 Interestingly, we found ACE2 is mainly expressed in goblet cells in current smokers and club cells in non-smokers (Fig. 3B) , indicating 2019-nCov may infect different cell types in bronchial epithelium of current smokers and non-smokers.\n",
      "\n",
      "Further, we analysed a scRNA-seq dataset of whole-lung tissue. Based on the expression of a set of markers shown in Figure 4C and Figure S4 , we identified 13 distinct cell populations including alveolar type I (AT1) cells, alveolar type II (AT2) cells, endothelial cells, ciliated cells, club cells, fibroblast, monocytes, macrophages, B, T cell or natural killer T cell (T/NKT), dendritic cells, a former smoker-specific subpopulation of AT2 cells (AT2reformed) and a current smoker-specific subpopulation of AT2 cells (AT2-smoking) (Fig. 4A) .\n",
      "\n",
      "Significantly, current smoker and former smoker showed differently remodelled AT2 cells (AT2-smoking and AT2-reformed, respectively). Also, we found ACE2 is most actively expressed in AT2-reformed cells in former Asian smokers but not in Caucasian current smokers and African American non-smokers (Fig. 4B, D) , which is consistent with our finding . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint from above large-scale bulk transcriptome analysis. In addition, we observed an expression of MUSC5AC (the marker of goblet cells) in a subpopulation of club cells of current smokers, indicating the smoking related tissue remodelling with club cell loss and goblet cell hyperplasia (Fig. S5 top) . This is consistent with our finding in bronchial epithelium.\n",
      "\n",
      "However, we failed to observe an active expression of ACE2 in the goblet cells of current smokers in a way similar to what we found in bronchial epithelial cells (Fig. S5 bottom) . The limited cell number of this cluster in this dataset might be the reason.\n",
      "\n",
      "In this study, we investigated the disparities related to race, age, gender and smoking status in ACE2 gene expression by analyzing bulk and single-cell transcriptome data. We found significantly higher ACE2 gene expression in lung tissue of former smokers compared to that of non-smokers. This may explain the reason why more males (56% of 425 cases) were found in a recent epidemiology report of 2019-nCov early transmission by China CDC 18 . It also consistent with the epidemiology study of 24,554 cases that former smokers (49%) have the higher risk to develop severe disease than non-smokers (14.5%) and current smokers (21.7%) 19 . We didn't observe significant disparities in ACE2 gene expression between racial groups (Asian vs Caucasian), age groups (>60 vs <60) or gender groups (male vs female). However, Asian current smokers may have higher ACE2 gene expression than Caucasian current smokers. The different is not statistically significant in this study but may indicate an existence of gene-smoking interaction.\n",
      "\n",
      "We also found that ACE2 gene is expressed in specific cell types related to smoking history and location. In bronchial epithelium, ACE2 is actively expressed in goblet cells of current smokers and club cells of non-smokers. In alveoli, ACE2 is actively expressed in remodelled AT2 cells of former smokers. This may indicate that 2019-nCov infect respiratory tract through different paths in smokers, former smokers and non-smokers, and this may partially lead to different susceptibility, disease severity and treatment outcome.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint One limitation of this study is that the small sample size of current single-cell transcriptome datasets has limited power in studying multiple factors involved in this question.\n",
      "\n",
      "Whether ACE2 is the only or major receptor of 2019-nCov is unknown. The reason(s) for the tobacco-related disparity in ACE2 expression is unknown. Studies found smoke significantly increased ACE2 expression in the lung of rats 20 \n",
      "\n",
      "There is no direct involvement of human subjects in this study. All the data use existing deidentified biological samples and data from prior studies. Therefore, ethical oversight and patient consent were not handled in this project. Figure   Figure 1 . ACE2 gene expression profiles in groups.\n",
      "\n",
      "A, C and D shows groups in race (Caucasian vs Asian), age (>60 vs <60) and gender (male vs female). B shows the result from multivariate analysis with all factors including age, gender, race, smoking and platforms.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint A shows expression in normal lung tissues of smoker and non-smoker with lung adenocarcinoma. B shows expression in normal lung tissues of never-smoker/non-smoker, reformed/former smoker and current smoker with lung adenocarcinoma. TCGA dataset has more categories of smoking history, including never-smoker, smoker reformed more than 15 years, smoker reformed less than 15 years and current smoker. C shows expression in healthy lungs of smoker and non-smoker.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.05.20020107 doi: medRxiv preprint\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 8th best search\n",
      "Title : The relationship of COVID-19 severity with cardiovascular disease and its traditional risk factors: A systematic review and meta-analysis\n",
      "Cases of coronavirus disease 2019 (COVID-19) are rapidly increasing globally. As of April 5, 2020, more than 1.2 million cases have been confirmed and ~70,000 deaths have been reported in ~180 countries. 1 Several studies have rapidly provided crucial data (e.g., incubation period) related to various aspects of the novel coronavirus (SARS-CoV-2 : severe acute respiratory syndrome coronavirus 2) infection. 2 However, risk factors for the severity and prognosis of COVID-19 are poorly understood. Such information is critical to identify high risk patients and to facilitate planning (e.g., forecasting the need for hospital beds and mechanical ventilators). These risk factors will also have implications for workforce allocation (e.g., assignment of healthcare providers with specific risk factors to positions with reduced risk of exposure to .\n",
      "\n",
      "To date, several studies have reported that a history of cardiovascular disease (CVD) and traditional CVD risk factors, e.g., age, male sex, current smoking, hypertension, and diabetes, are associated with severe COVID-19. However, other than age, results have been inconsistent. Furthermore, few studies accounted for potential confounding by age and sex when they evaluated other potential risk factors. For example, some studies reported that hypertension is a risk factor of severe COVID-19, but their observations may simply reflect the fact that hypertension is more common in older adults. 3 Nonetheless, despite the lack of robust evidence, this observation, together with the fact SARS-CoV-2 uses angiotensin-converting enzyme 2 as an entry to human body, 4 has raised a concern about continued use of reninangiotensin system increase among some clinicians and researchers. 5, 6 In this context, we conducted a systematic review of studies reporting cardiovascular risk factors and their relation to severe manifestation of COVID-19 (i.e., death, acute respiratory distress syndrome [ARDS] , the need of mechanical ventilator support, and admission to an intensive care unit [ICU]), with a particular interest on studies that adjusted for key confounders such as age and sex.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint\n",
      "\n",
      "We conducted this systematic review following the PRISMA Statement. According to the predetermined protocol, we systematically searched PubMed and Embase for eligible reports (search terms are listed in Web Appendix 1). We included full reports or letters with original data written in English. Eligible study designs were cohort study, cross-sectional, case series, and clinical trials. We conducted the literature search on March 20, 2020 and restricted to publications after December 1, 2019.\n",
      "\n",
      "Our review included studies that reported adult patients, aged 18 years or older. There was no restriction with respect to gender, race/ethnicity, and comorbidities. The primary outcome of interest was severe COVID-19 defined by any of the following: all-cause mortality, ICU admission, ARDS, or the need for mechanical ventilation. We included studies reporting at least 10 cases of severe COVID-19. To obtain reliable estimates with enough number of outcomes and considering clinical cascade (e.g., death as the final outcome), when one study reported results for multiple outcomes, we prioritized any composite outcome followed by ICU admission, ARDS, the need of mechanical ventilation, and mortality.\n",
      "\n",
      "Potential risk factors of interest were pre-existing CVD (including cardiac disease and cerebrovascular disease) and its traditional risk factors recognized in major CVD clinical guidelines: age, sex, smoking, hypertension, and diabetes. We found only one study reporting severity of COVID-19 by lipids (low-density lipoprotein). We categorized risk factors into sociodemographic factors (age, sex, and smoking) and clinical factors (hypertension, diabetes, and pre-existing CVD).\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint \n",
      "\n",
      "The same eight reviewers collected relevant data elements from each identified publication and recorded in Microsoft Excel (Microsoft Corporation, Redmond, Washington, USA). Overall quality was based on the Newcastle Ottawa Quality Assessment Scale (NOS), 7 which includes eight items about selection, comparability, and outcome (Web Appendix 2). The NOS score for cohorts studies ranges from 0 to 9; a score greater than 6 was considered high-quality. For cross-sectional studies, we applied an adapted form of the NOS. 8 The maximum score was 10, and 7 points were used to identify studies with high quality.The paired reviewers resolved conflicts related to their own data collection and quality assessment.\n",
      "\n",
      "We summarized relative risk estimates (odds ratios or hazard ratios) of the association between each risk factor and the primary outcome from the relevant studies. We pooled these estimates using random-effects meta-analysis. When studies did not report these measures of association but the prevalence of risk factors of interest by the outcome status (e.g., survivors vs. nonsurvivors), we calculated crude odds ratios and their 95% CIs. In this process, when there was any cell with zero count, we added 0.5 to each cell, as appropriate. 9 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint Potential confounding by age and sex is relevant to prior CVD, hypertension, and diabetes, since these comorbidities become more prevalent with increased age. 3 Because most studies did not report adjusted risk estimates for these comorbidities, we ran meta-regression with random-effects for log odds ratio or log hazard ratio for these comorbidities by the difference in mean or median age between those with vs. without primary outcome across eligible studies. To obtain reliable estimates, we conducted meta-regression for any analyses with at least five studies. We also depicted funnel plots and visually checked the possibility of publication bias. Heterogeneity of study estimates was assessed by I 2 statistic, and I 2 >75% was considered high heterogeneity. 10 A p-value <0.05 was considered statistically significant. All analyses were conducted with STATA 14 or 15 (StataCorp, LLC, College Station, Texas, USA).\n",
      "\n",
      "Our systematic review identified 373 potentially eligible publications after removing duplicate publications ( Figure 1 ). Of these, 322 publications were excluded after screening titles and abstracts. Of the remaining 51 publications reviewed with full-text screening, we excluded 36 publications that did not meet our inclusion criteria, leaving 15 publications 11-25 for our qualitative and quantitative analyses. Most of these publications were considered high quality (Web Table   1 ). Of the included studies, death was reported in 13 studies, ICU admission in 9 studies, ARDS in 7 studies, and mechanical ventilation in 6 studies. Two studies reported a composite outcome.\n",
      "\n",
      "Most studies reported COVID-19 patients from China (14 studies) and were small with sample size <300 (10 studies) ( Table 1 ). All studies included confirmed COVID-19 patients with laboratory tests. A total of 51,845 COVID-19 patients were included in these studies, with 9,066 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint patients manifesting severe disease. Most studies exclusively investigated hospitalized COVID-19 cases. Average or median age ranged from 38 to 63 years, and 45% to 73% of participants were male. Most studies reported the prevalence of CVD risk factors; the prevalence of hypertension ranged from 12.8% to 35%, diabetes from 5.3% to 22%, and pre-existing CVD from 2.5% to 21.7%.\n",
      "\n",
      "Older age was associated with higher risk of severe COVID-19 in all eight studies with relevant estimates ( Table 2 ). This pattern was confirmed in national surveys database in the US, China, and Italy 26 (published after our literature search), without any clear thresholds (Web Table 2 ).\n",
      "\n",
      "The case fatality rate in those three databases exceeded 1% around age of 50-55 years and 10% above 80-85 years (above 70 years in Italy). Only two studies reported the relative risk by age in both unadjusted and multivariable models; both studies observed that the effect size of age was not materially changed after adjustment for comorbidities. 12, 25 Most studies showed a higher risk of severe COVID-19 in men than in women, with a pooled crude relative risk estimate of severe COVID-19 between men and women of 1.70 (95% CI 1.52-1.89) (Figure 2A ). Meta-regression demonstrated consistent results regardless of differences in mean or median age between those with vs. without severe COVID-19 (Web Figure 1) . A funnel plot did not indicate major publication bias (Web Figure 2A) .\n",
      "\n",
      "Only three studies reported associations of current smoking with severe COVID-19, with only one study reaching statistical significance ( Figure 2B ). The pooled estimate of relative risk for severe COVID-19 was 2.01 (95% CI 0.83-4.86). Only one study reported the association across three categories of current smoking (odds ratio 2.84 [1.57-5.14]), former smoking (6.27 [2.20-17 .90]) vs. never smoking. 15 The corresponding funnel plot is shown in Web Figure 2B .\n",
      "\n",
      "Clinical factors: hypertension, diabetes, and prior CVD . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. for diabetes. 17 In that study, cancer and chronic obstructive pulmonary disease had similar independent associations with severe COVID-19.\n",
      "\n",
      "Meta-regression analyses demonstrated that studies with greater age difference between those with vs. without severe COVID-19 tended to have greater relative risk according to the presence of hypertension, diabetes, and pre-existing CVD, indicating some levels of potential confounding by age (Figure 4) , although none of the analyses reached statistical significance. A similar pattern was seen for CVD and the difference in the proportion of male sex between those with vs. without severe COVID-19 (Web Figure 3) . On the other hand, meta-. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint regression did not indicate that a higher proportion of male sex confounded the association of hypertension and diabetes with severe COVID-19.\n",
      "\n",
      "To our knowledge, this is the first systematic review and meta-analysis focusing on the relationship of severe COVID-19 with CVD and its risk factors. We confirmed a robust association of age and male sex with severe COVID-19. Their contributions are likely to be independent of each other. A few studies demonstrated positive associations of current smoking with severe COVID-19. Several studies reported that pre-existing CVD, hypertension, and diabetes were also associated with severe COVID-19. However, only two studies that reported estimates for these comorbidities adjusted for age and/or sex. One study found that the association between coronary heart disease and severe COVID-19 was no longer statistically significant after age adjustment, 25 whereas the other showed independent associations of hypertension and diabetes with severe COVID-19 in analyses that adjusted for age and a few other comorbidities. 17 Although the primary estimate was not statistically significant, our metaregression analyses indicated some degree of confounding by age, but not necessarily by sex, for the associations of hypertension, diabetes, and prior CVD with severe COVID-19.\n",
      "\n",
      "The positive association of sociodemographic factors (age, male sex, and smoking) with severe COVID-19 is consistent with reports of other infectious diseases (e.g., influenza virus and SARS in 2003). [27] [28] [29] There are several plausible mechanisms. Older age is linked to reduced immune reaction, more comorbidities, and limited organ reserve. 3, 27, 30 Male sex is related to higher prevalence of comorbidities, less frequency of washing hands, and immunological disadvantage given X-chromosome coding proteins in the immune system, 2,31,32 whereas smoking can damage respiratory system. 33 In our meta-analysis, we confirmed overall positive crude associations of CVD, hypertension, and diabetes with severe COVID-19, with pooled relative risk estimates around 3-. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint 3.5. An important question is whether these associations are independent of major confounders, particularly age. Although we could not obtain a definite answer as very few studies ran multivariable models, our meta-regression analysis indicated some degree of confounding by age. One study by Zhou et al showed substantial attenuation of the association between coronary heart disease and severe COVID-19. On the other hand, Liang et al showed independent associations of hypertension and diabetes with severe COVID-19 in a relatively large sample size of 1,590 patients. These is also biologic plausibility. Hypertension and diabetes are leading risk factors for CVD and kidney diseases, and there is evidence that COVID-19 damages these organs. 34, 35 Available studies provide scant evidence related to the hypothesized adverse effect of renin-angiotensin system inhibitors on severity of COVID-19 infection. As noted above, there is biological plausibility for potential associations of COVID-19 with hypertension and diabetes, two comorbidities associated with use of renin-angiotensin system inhibitors. Only one study reported the prevalence of renin-angiotensin system inhibitors use. 13 In this study, ~30% of patients reported prevalent hypertension, but only ~5% of patients were taking renin-angiotensin system inhibitors. 13 Thus, it is reasonable that many expert organizations recommend continuing this category of medications until better evidence becomes available. 4 Our results can potentially be used to guide decision-making. While the lack of discrete age thresholds for severe COVID-19 complicates this process, data on case fatality rates from three countries (Web Table 2 ) suggest that age older than 60 or 65 years confers high risk of severe COVID-19 (relative risk > ~5 compared to <50 years). Our results also indicate male sex is an independent risk factor with a pooled relative risk of ~1.7. Although it is very likely that the pooled crude relative risk of ~3-3.5 for hypertension, diabetes, and CVD overestimate their impact beyond age and sex, all or some of them may each confer 1.5-2 times greater risk. Thus, using these factors, it is possible to estimate, at least crudely, the risk of severe COVID-19. For example, a man aged 60-65 years old, with either hypertension, diabetes, or prior CVD would . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint have a risk of ~15 fold higher risk (approximation of 5 x 1.7 x 2) compared to a woman younger than 50 years without any of these clinical factors. Such information could be used to inform decisions on testing for COVID-19, clinical management of COVID-19, and workforce planning.\n",
      "\n",
      "Our study has some limitations. First, reflecting the fact that the outbreak started from China, most studies were from China. However, given similar case-fatality rates and clinical manifestations across different countries, it seems likely that these results are largely generalizable. Nonetheless, we need to acknowledge regional variations of some risk factors (e.g., ~25-fold difference in the prevalence of smoking in men vs. women in China 36 ) and thus future investigations in different regions would be valuable. Second, we did not include non-English publications. Third, most studies reported odds ratios, which are known to overestimate risk ratio when the prevalence of exposures is relatively high. Fourth, we cannot deny the possibility that some patients were included in multiple studies especially in the China CDC report 14 and other Chinese studies. Nonetheless, the pooled estimates were largely similar in analyses that excluded the China CDC data (data not shown). Finally, the literature of COVID-19 is growing rapidly, and thus there is a lag time from our literature search and publication. On the other hand, our systematic review has several strengths: in-depth review of CVD and its risk factors, a clinically relevant definitions of severe COVID-19 that minimize subjective reporting, meta-regression to explore potential confounding, and relatively short elapsed time of ~2 weeks between the literature search and manuscript submission.\n",
      "\n",
      "In conclusion, our systematic review and meta-analysis found robust associations of older age and male sex as risk factors of severe COVID-19. Few studies reported the association between current smoking and severe COVID-19. In unadjusted analyses, hypertension, diabetes, and prior CVD were significantly associated with severe COVID-19.\n",
      "\n",
      "However, only two-studies conducted age-adjusted analyses, with one relatively large study showing independent contributions of hypertension and diabetes to severe COVID-19. No study explicitly assessed the relationship of renin-angiotensin system inhibitors with COVID-19. Our . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint results suggest that the combination of age, male sex, and CVD risk factors identify a substantial gradient in the risk of developing severe COVID-19 compared to those without any of these factors.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org /10.1101 /10. /2020 This project is supported by Resolve to Save Lives, which is funded by Bloomberg Philanthropies, the Bill and Melinda Gates Foundation, and Gates Philanthropy.\n",
      "\n",
      "None.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint All studies were published in 2020.\n",
      "\n",
      "Abbreviations: COVID-19: coronavirus disease 2019; CVD: cardiovascular disease; HTN: hypertension; NR, not reported. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.05.20054155 doi: medRxiv preprint\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 9th best search\n",
      "Title : Prevalence, Severity and Mortality associated with COPD and Smoking in patients with COVID-19: A Rapid Systematic Review and Meta-Analysis Authors contributions\n",
      "The emergence in Wuhan City, Hubei Province of China of a novel pneumonia of unknown origin on the 31 st of December, 2019 was the start of an outbreak which would later be declared a pandemic by the World Health Organization (WHO) (1). The name COVID-19 (acronym for \"coronavirus disease 2019\") was coined on the 11 th of February 2020 to describe presentation with severe acute respiratory disease (2). COVID-19 is caused by a novel strain of coronavirus, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This virus belongs to the family of single stranded RNA viruses some of which have been previously described to be responsible for the Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS) (3, 4) . Although the symptoms and clinical presentation of COVID-19 is similar to SARS and MERS, the rate of spread is greater (5) . As of 25 th of March 2020, the total number of confirmed case of COVID-19 stands at 459,419 with 20,818 deaths globally and the number is projected to increase (6) .\n",
      "\n",
      "Chronic Obstructive Pulmonary Disease (COPD) is a common, persistent and preventable dysfunction of the lung associated with limitation in airflow. COPD is a complex disease associated with abnormalities of the airway and/ or alveoli which is predominantly caused by exposure to noxious gases and particulates over a long period (7). With a global prevalence of 251 million cases in 2016, and 3.17 million (5%) deaths in 2015 alone, COPD was ranked by the WHO as the third leading cause of death especially with a particular burden in low-and middle-income countries (8, 9) . This burden is predicted to grow due mainly to increased global exposure to tobacco, aging populations, poor awareness and inadequate access to diagnosis (10) . COPD exacerbations are a major event in the natural history of the disease associated with worsening of symptoms often resulting in hospitalisation and poor prognosis (11) . Various factors have been described to contribute to acute worsening of COPD, however, viral infection remains the main trigger, including seasonal coronaviruses (12) (13) (14) .\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 25.20043745 doi: medRxiv preprint Since the emergence of COVID-19, scientists around the world continue to better understand   the clinical, diagnostic and prognostic characteristic of the disease. While over 150 papers,   editorials and comments have been written about COVID-19 as of 24 th of March 2020, there is none dedicated to the specific risk posed to patients with a previous history of COPD. This review addresses this gap in knowledge with the aim of assisting clinicians to assess the prognosis of COVID-19 infection in patients with COPD and those with smoking history.\n",
      "\n",
      "This systematic review was conducted in accordance with the Preferred Reporting in Systematic Reviews and Meta-Analyses (PRISMA) guidelines (15) . We prospectively submitted this review to Prospero (ID: 175518).\n",
      "\n",
      "We searched MEDLINE and Google scholar from inception date to March 16, 2020. The search was updated on March 24, 2020. We used an extensive search strategy developed by a specilaised librarian for retrieving this type of evidence, which included the reference list of eligible papers and published pre-print papers (see supplementary Table S1 ). All retrieved studies were exported into EndNote to remove duplicates. The remaining studies were exported to Rayyan software for title, abstracts and full text screening by two independent reviewers.\n",
      "\n",
      "Eligible studies were those that investigated: epidiomolgical, clinical charcterstics and features of COVID-19 and prevelance of chronic diseases specifically COPD in their analysis. We excluded the following: studies that did report COPD, studies that included respiratory diseases but did not specifically analyse COPD, only children, editorials, correspondence letters, reviews, qualitative studies, theses, non-English manuscripts and non-full text articles.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 25.20043745 doi: medRxiv preprint\n",
      "\n",
      "Two authors independently screened titles and abstracts of potential studies and conflicts were resolved through discussion between the two. Full-text articles of potential studies were then independently read by two authors to identify studies meeting the inclusion criteria. The reference lists from all identified studies and reviews were scrutinized for eligible articles.\n",
      "\n",
      "Disagreement on selected papers was resolved through discussion with a third author.\n",
      "\n",
      "Two authers independently evaluated the methodological quality of included studies using a modified version of the Newcastle-Ottawa Scale (NOS) (16). It includes seven domains, each one of these domains was scored from 0 (high risk of bias) to 3 (low risk of bias) and we took a mean of the domains to result in a score between 0 and 3, where a higher score represents a lower risk of bias. Any disagreement in the quality assessment was resolved by discussion with a third author.\n",
      "\n",
      "We completed meta-analysis to calculate the pooled prevalence of COPD and current smokers among those patients with confrmed COVID-19. The output was generated using the Stata procedure Metaprop. Owing to heterogeneity within and between studies, we used the random-effects model in Stata/SE 15. Data were displayed using forest plots. We examined between-study heterogeneity using the I 2 statistic. A narrative synthesis of the results was conducted considering the prevelance, disease severity and mortality among COVID-19 COPD patients and smoking status. We defined COVID-19 severity as those who were admitted to intensive care unit (ICU), had severe, oxygenation, needed mechanical ventilation or death.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 25.20043745 doi: medRxiv preprint\n",
      "\n",
      "As this is a rapidly evolving area, we also searched pre-print literature and whilst not incorporating such data in the formal analysis, provide narrative synthesis where such preprint data are relevant to our main findings.\n",
      "\n",
      "An initial search generated 123 potentially relevant papers, of which two were immediately excluded due to duplication. After the first screening of title and abstracts, 61 papers were potentially relevant according to the inclusion criteria. An additional 46 papers were excluded after full-text review, which resulted in 15 studies that satisfied all criteria. The reference list of the relevant papers was also examined (see Figure 1 , PRISMA flow diagram).\n",
      "\n",
      "A summary of the included studies is presented in Table 1 , which included a total of 2473 confirmed COVID-19 patients. Of those patients, only 58 (2.3%) had COPD as a comorbidity. The sample size of the included studies ranged from 21 to 1099 patients. Most of the studies were retrospectively conducted and all were studied in China, except one in the United States. The risk of bias ranged from 0.4 to 2.7; nine studies scored ≥ 2, which indicates low risk of bias (see, Table S2 ). The overall crude mortality rate for COVID-19 was 7.4% (184/2473). There were no reports of the baseline characteristics of the 58 COPD patients that described their age, or severity of COPD airflow limitation. Smoking status was reported in eight studies, in which 221 confirmed COVID-19 cases were current smokers.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "1 0\n",
      "\n",
      "The pooled prevalence of COVID-19 patients who had COPD from all studies was 2% (95% CI, 1%-3%), Figure 2 . \n",
      "\n",
      "Seven studies that included 35 COPD patients reported COVID severity in their analysis. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "Smoking exposure including (current and ex-smokers) was reported in eight studies, with 221 confirmed COVID-19 cases. We assessed the prevalence of current smokers in all studies using meta-analysis of proportions of current smokers that had confirmed COVID-19. There was a pooled prevalence of 9%, (95% CI, 4%-14%), Figure 3 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 25.20043745 doi: medRxiv preprint\n",
      "\n",
      "To the best of our knowledge, this is the first systematic review and meta-analysis to develop an informed understanding of the prevalence, severity and mortality of COPD patients diagnosed with COVID-19. We provide an updated report in relation to smokers (32) . Our main outcomes show that the prevalence of COPD in COVID-19 patients was low, but that the risk of severity (63%) and mortality (60%) were high, which indicates COPD patients with confirmed COVID-19 are at a greater risk of severe complications and death.\n",
      "\n",
      "Furthermore, the prevalence of current smokers in COVID-19 patients was 9% (95% CI, 4%-14%), and this was also associated with greater severity (22.30%) and mortality (38.5%).\n",
      "\n",
      "We report a low prevalence of COPD patients in COVID-19 case series compared to the latest COPD prevalence rate in China, which was 13.6% (95% CI 12·0-15·2) and the global prevalence of COPD (9-10%) (33, 34) . We speculate that patients may have not been diagnosed. Having a reliable estimate of the prevalence of COPD in COVID-19 cases, and likely outcomes, is crucial to ensure specific successful global preventive and treatment strategies for COPD patients. Bearing this in mind, in the included studies there was no report on COPD severity data and COPD-related comorbidities, which prevents us from assessing the impact of such essential information.\n",
      "\n",
      "Although the COPD prevalence was not high in the included confirmed COVID-19 cases,\n",
      "\n",
      "This summary agrees with other results currently available only in pre-print (35, 36) . Viral infections in COPD patients increase systemic inflammation with slow recovery of reported symptoms (37, 38) . In addition to the effects of COVID-19, patients with COPD have various comorbidities, some of which are associated with increased risk of hospitalisation (39) (40) (41) .\n",
      "\n",
      "According to recent systematic reviews, the prevalence of comorbidities in COVID-19\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 25.20043745 doi: medRxiv preprint patients was high and these comorbidities were associated with increased disease severity (42, 43) . Most of the studies that reported COPD severity defined severe cases as those who were admitted to intensive care unit (ICU), had severe oxygenation, needed mechanical ventilation or death (18, 19, 22, 23, 27, 29) . These studies did not give details about the underlying COPD. It would have been interesting to know if the frequency of previous exacerbations was linked to increased complications in COPD patients diagnosed with COVID-19. In general, those with severe case of COVID-19 were older and had more coexisting comorbidities than those with mild illness.\n",
      "\n",
      "Data from two studies (28, 30) describing COPD patients with confirmed COVID-19 show a higher mortality rate at 60%. This concurs with pre-print studies that found similarly high mortality rates (35, (44) (45) (46) (47) . Despite the small number of patients that were analysed, this increases concerns about the prognosis of this vulnerable population. However, this high mortality rate could be attributed to several factors. The majority of COPD patients have various comorbidities that may also be associated with mortality and related conditions may have been underreported because of the difficulties finding the specific cause of mortality (48) . Moreover, in patients with severe COPD, respiratory failure is the principal cause of mortality and this demands ICU intervention. It is possible that limited access to respiratory support as part of COVID-19 management may be contributing to this mortality, dependent on critical care capacity in each hospital or region. According to a recent COVID-19 report from Italy, the surge in patients requiring intensive care has been unmanageable, with 12% of positive cases requiring ICU admission (49) , more than that reported in China (18, 28). As a consequence, patients were dying because mechanical ventilation could not be offered, on top of acute shortage of clinicians who were able to manage those patients (50, 51) . Until now, the particular mechanisms of how COVID-19 increases COPD severity and mortality is . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "unknown, and undoubtedly more research is needed to find the possible mechanisms that linked COVD-19 and increased severity and mortality of COPD patients.\n",
      "\n",
      "Concerning smoking and COVID-19, our data showed a pooled prevalence of 9% current smokers, (95% CI, 4%-14%), lower than the reported prevalence of smoking in China that was 25.2% (25. (18, 29) . The impact of smoking history on vulnerability to COVID-19 has been explored but there is limited data on the contribution of tobacco smoking to the spread of and poor outcome in COVID-19. A recent systematic review on COVID-19 and smoking including five studies found that smoking was most likely associated with the negative outcomes. This recommended further research to explore this in more detail due to limited studies (32) . Evidence from other respiratory viruses, respiratory syncytial virus, has shown that inhaled tobacco smoke raises the transmission rate and severity of viral respiratory tract infections (53) . It seems there is underlying mechanisms behind this prevalence, as smoking has been related to higher expression of ACE2 (the receptor for SARS-CoV-2) (pre-print) (54) . However, as more reports globally from diverse racial and genetic contexts become available, differences in the production of ACE2 can be further evaluated and linked to how they lead to COVID-19 vulnerability in different groups (55) .\n",
      "\n",
      "To our knowledge, the present study is the first to systematically evaluate existing literature with a focus on risk of COVID-19 on COPD. For the first time, we conducted a meta-analysis using a random effects model to calculate the pooled prevalence of COPD in confirmed COVID-19 and examined outcomes. This increased the generalisability of our findings, as . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 25.20043745 doi: medRxiv preprint heterogeneity was addressed by incorporating between-study variability of effect sizes. We showed that COPD and smoking in COVID-19 is associated with greater disease severity and higher mortality. This review has some limitations. Few studies were eligible for inclusion and most of them come from China. Second, heterogeneity exists in location, setting, and design. The reported clinical characteristics were not available in most of the studies at the time of analyses. This work has a number of clinical and research implications. It highlights the global prevalence and the clinical effects of COVID-19 on COPD patients and smokers.\n",
      "\n",
      "As COPD patients are at an increased risk of severe outcomes if they became infected with COVID-19, it is recommended that patients and clinicians establish effective plans for ensuring prevention, such as using tele-medicine to ensure that COPD receive the best care (56, 57) . We strongly advocate public awareness campaigns concentrating on ways to achieve smoking cessation among smokers, and it is possible that an improvement in cessation rates will help to reduce the spread of SARS-CoV-2. Future studies should investigate the mechanisms between COPD, smoking and COVID-19 infection.\n",
      "\n",
      "Though COPD prevalence in reported COVID-19 cases is low, COVID-19 infection is associated with significant severity and mortality in COPD. There was also increased risk of severe disease and mortality in current smokers. Effective preventive measures are urgently required to reduce COVID-19 risk on COPD patients and current smokers.\n",
      "\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03. 25.20043745 doi: medRxiv preprint 1 1 9\n",
      "\n",
      "Appendix: \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The 10th best search\n",
      "Title : Smoking is Associated with COVID-19 Progression: A Meta-Analysis\n",
      "COVID-19, the coronavirus-transmitted infectious disease, has caused a worldwide pandemic. Smoking 1 2 and e-cigarette use 3 increase risk and severity of pulmonary infections because of damage to upper airways and a decrease in pulmonary immune function. In particular, smokers have a higher risk of infection and mortality from Cov-MERS. 4 Two reviews 5 6 of the first 5 papers presenting data on smoking and COVID-19 reached different conclusions. Another review described 6 published case series presenting data on smoking among COVID-19 patients but did not draw a conclusion about the association of severity of COVID-19 with smoking. 7 We reviewed and summarized 12 papers presenting data on the association between smoking and severity of COVID-19.\n",
      "\n",
      "We conducted a systematic search using PubMed database on April 6, 2020, with the search term: ((smoking) OR (characteristics) OR (risk factors) OR (outcomes) OR (smoker*)) AND ((COVID-19) OR (COVID) OR (coronavirus) OR (sars cov-2) OR (sars cov 2)) for studies published between January 1, 2020 and April 6, 2020. There were no language restrictions. A total of 396 studies were retrieved through the search, of which 12, 10 from China, [8] [9] [10] [11] [12] [13] [14] [15] [16] [17] 1 from Korea, 18 and 1 from the US, 19 included data on smoking behavior and COVID-19 disease progression (Table 1 ). Ten studies 8 10-18 were based on hospitalized patients and two 9 19 included both hospitalized patients and outpatients.\n",
      "\n",
      "The exposure group is those who had a history of smoking (current smokers or former smokers) and unexposed group was never smokers. Five studies 8 10 12 14 17 assessed whether the patient was a \"current smoker,\" three studies 9 16 19 assessed whether the patient was a current or All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04. 13.20063669 doi: medRxiv preprint former smoker (as separate categories), and four studies 11 13 15 18 assessed whether the patent had a \"history of smoking\" (current or former).\n",
      "\n",
      "Outcome is progression of COVID-19 to more severe or critical conditions or death (Table 1) . Two studies 14 16 categorized the outcome as severe (respiratory distress with respiratory rate ≥30/min, or oxygen saturation ≤93% at rest, or oxygenation index ≤300 mmHg, based on the diagnostic and treatment guideline for SARS-CoV-2 issued by Chinese National Health Committee 16 ) or mild, three 8 11 12 categorized the outcome as progression or improvement, two 10 19 categorized the outcome as ICU admission or non-ICU admission, one 9 categorized the outcome as the primary composite end point (ICU admission, the use of mechanical ventilation, or death) or not, two 15 17 categorized the outcome as death or survivor, one 13 categorized the outcome as the occurrence of severe cases (without defining severe) or death or mild, and one 18 categorized the outcome as clinical deterioration during the hospitalization and needed supplemental oxygen therapy.\n",
      "\n",
      "We computed unadjusted odds ratios (OR) and 95% confidence interval (CI) for each study using the number of smokers (current and former) and never smokers with and without disease progression. Random effects meta-analysis was performed using the Stata 14.0 metan command and using metabias command with Harbord and Peters to test for the presence of publication bias.\n",
      "\n",
      "Patients or the public were not involved in the design, or conduct, or reporting of our research. We will widely disseminate our findings working with the media and interested clinical and public health groups.\n",
      "\n",
      "All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.13.20063669 doi: medRxiv preprint\n",
      "\n",
      "A total of 9,025 COVID-19 patients included in our meta-analysis, 878 of whom (9.7%) experienced disease progression and 495 with a history of smoking (5.5%). A total of 88 patients with a history of smoking (17.8%) experienced disease progression, compared with 9.3% of never smoking patients.\n",
      "\n",
      "The meta-analysis showed an association between smoking and COVID-19 progression (OR 2.25, 95% CI 1.49-3.39, p=0.001) ( Figure 1 ). There was not significant heterogeneity among the studies (I 2 =28.9%, p=0.162) or publication bias (Harbord's p=0.155, Peters' p=0.668).\n",
      "\n",
      "Our analysis confirms that smoking is a risk factor for progression of COVID-19, with smokers having 2.25-times the odds of severe COVID-19 outcomes than never smokers. This finding contrasts with an earlier meta-analysis, 6 which included only 5 studies and used a nonstandard method to compute the meta-analysis. The finding that smoking is associated with COVID-19 progression is not surprising because of the adverse effects of smoking on pulmonary immune function. 1 2 Our study has several limitations.\n",
      "\n",
      "The definition of \"smoking\" sometimes includes former smokers and sometimes does not. Only three studies 9 16 19 separated current and former smokers in different categories, which was not enough data to do a meta-analysis for current and former smokers separately. Because the lung recovers after someone stops smoking, including former smokers in the exposed group biases the effect estimate to the null.\n",
      "\n",
      "All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04. 13 20 18.5% in the Korean patients vs. 21.1% 21 (37.0% for men and 5.2% for women) in 2017, and 3.6% in the US patients vs. 13.7% 22 (15.6% for men and 12.0% for women) in 2018. It is highly likely that many smokers were misclassified as nonsmokers, which also biases the risk estimate toward the null.\n",
      "\n",
      "We computed and assessed unadjusted odds ratios based on the numbers of patients reported in the studies. Only one 11 None of these studies assessed e-cigarette use.\n",
      "\n",
      "All these limitations suggest that this analysis underestimates the risk of smoking in terms of increasing COVID-19 severity.\n",
      "\n",
      "All 12 studies were of patients who had already developed COVID-19, so the risk estimate we report does not represent the effect of smoking on the risk of contracting COVID-19 in the general population. As population-level testing ramps up, it would be useful to collect data on smoking and e-cigarette use to determine what risks these behaviors impose in terms of infection.\n",
      "\n",
      "Smoking is associated with COVID-19 disease progression. Physicians and public health professionals should collect data on smoking and, given the pulmonary effects of e-cigarettes, 3 e-All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04. 13.20063669 doi: medRxiv preprint cigarette use, as part of routine clinical assessments and add smoking cessation to the list of practices to blunt the COVID-19 pandemic. \n",
      "\n",
      "from the corresponding author) and declare that the work was supported as described in the funding statement above. The authors have no financial relationships with any organisations that might have an interest in the submitted work in the previous three years and no other relationships or activities that could appear to have influenced the submitted work.\n",
      "\n",
      "RP developed the idea for the study, collected, analyzed the data, and wrote the first draft of the manuscript. SAG assisted with revising and refining the manuscript. Both authors affirm that the manuscript is an honest, accurate, and transparent account of the study being reported; that no important aspects of the study have been omitted; and that any discrepancies from the study as planned have been explained.\n",
      "\n",
      "All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04. 13.20063669 doi: medRxiv preprint ETHICAL APPROVAL Not required.\n",
      "\n",
      "All data used to prepare this paper are available from the cited sources.\n",
      "\n",
      "All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.13.20063669 doi: medRxiv preprint Smoking and COVID-19 Progression All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04. 13.20063669 doi: medRxiv preprint \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "#return the tite and the fullbody documents of the top 10 similarities\n",
    "ordinal = lambda n: \"%d%s\" % (n,\"tsnrhtdd\"[(n/10%10!=1)*(n%10<4)*n%10::4])\n",
    "arrFinal = dfinal.to_numpy()\n",
    "arrayDocuementsid  = np.argsort(sims[query_doc_tf_idf])[-10:]\n",
    "for i in range (len(arrayDocuementsid)):\n",
    "    index = arrayDocuementsid[i]\n",
    "    print (\"The \" + ordinal(i+1) + \" best search\" )\n",
    "    if(arrFinal[index][0]==''):\n",
    "        print(\"Title : not Mentioned\" + arrFinal[index][0])\n",
    "    else:\n",
    "        print(\"Title : \" + arrFinal[index][0])\n",
    "        \n",
    "        \n",
    "    \n",
    "    print(arrFinal[index][2])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " * Serving Flask app \"__main__\" (lazy loading)\n",
      " * Environment: production\n",
      "   WARNING: This is a development server. Do not use it in a production deployment.\n",
      "   Use a production WSGI server instead.\n",
      " * Debug mode: off\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)\n",
      "127.0.0.1 - - [14/May/2020 01:32:17] \"GET / HTTP/1.1\" 200 -\n",
      "127.0.0.1 - - [14/May/2020 01:32:21] \"GET / HTTP/1.1\" 200 -\n",
      "127.0.0.1 - - [14/May/2020 01:32:24] \"GET / HTTP/1.1\" 200 -\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of documents:2\n",
      "Comparing Result: [0.         0.         0.         ... 0.00148189 0.         0.        ]\n",
      "4.6586294\n",
      "Maximum Similiarity\n",
      "0.1647856\n",
      "[0.05636115 0.05703232 0.05949826 0.06513985 0.09237453 0.11066813\n",
      " 0.12400745 0.12550147 0.1512416  0.1647856 ]\n",
      "[1266  623 2254 1773  708 1048  931 1337 1041  644]\n",
      "Maximum Similiarity of current Document0.05636115\n",
      "The 1st best search\n",
      "Title : Epidemiological and ecological modelling reveal diversity in upper respiratory tract microbial population structures from a cross-sectional community swabbing study Key words Ecology, Epidemiology, Respiratory Infection Running Title Application of epidemiological and ecological modelling to microbial populations Corresponding Author\n",
      "Maximum Similiarity of current Document0.057032317\n",
      "The 2nd best search\n",
      "Title : An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors' CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design\n",
      "Maximum Similiarity of current Document0.059498258\n",
      "The 3rd best search\n",
      "Title : Computational vaccinology approach: Designing an efficient multi-epitope peptide vaccine against Cryptococcus neoformans var. grubii's heat shock 70KDa protein\n",
      "Maximum Similiarity of current Document0.065139845\n",
      "The 4th best search\n",
      "Title : Genome based Evolutionary study of SARS-CoV-2 towards the Prediction of Epitope Based Chimeric Vaccine\n",
      "Maximum Similiarity of current Document0.092374526\n",
      "The 5th best search\n",
      "Title : Shotgun proteomics of SARS-CoV-2 infected cells and its application to the optimisation of whole viral particle antigen production for vaccines\n",
      "Maximum Similiarity of current Document0.11066813\n",
      "The 6th best search\n",
      "Title : Original Article\n",
      "Maximum Similiarity of current Document0.12400745\n",
      "The 7th best search\n",
      "Title : Estimating Probabilities of Success of Clinical Trials for Vaccines and Other Anti-Infective Therapeutics 1\n",
      "Maximum Similiarity of current Document0.12550147\n",
      "The 8th best search\n",
      "Title : Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study 1 2 AUTHORS 3\n",
      "Maximum Similiarity of current Document0.1512416\n",
      "The 9th best search\n",
      "Title : The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19\n",
      "Maximum Similiarity of current Document0.1647856\n",
      "The 10th best search\n",
      "Title : Financing Vaccines for Global Health Security\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "127.0.0.1 - - [14/May/2020 01:32:26] \"POST / HTTP/1.1\" 200 -\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0\n",
      "Number of documents:3\n",
      "Comparing Result: [0.         0.00239954 0.00657212 ... 0.00462065 0.00140222 0.        ]\n",
      "16.598541\n",
      "Maximum Similiarity\n",
      "0.31352836\n",
      "[0.11928368 0.1196094  0.12171915 0.13552599 0.13590983 0.1370548\n",
      " 0.15111834 0.16413192 0.17724138 0.31352836]\n",
      "[2253  265 1368 1824  978   52  160 2144 1696 2092]\n",
      "Maximum Similiarity of current Document0.119283676\n",
      "The 1st best search\n",
      "Title : Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole\n",
      "Maximum Similiarity of current Document0.1196094\n",
      "The 2nd best search\n",
      "Title : not Mentioned\n",
      "Maximum Similiarity of current Document0.12171915\n",
      "The 3rd best search\n",
      "Title : Title: Is the spread of COVID-19 across countries influenced by environmental, economic and social factors?\n",
      "Maximum Similiarity of current Document0.13552599\n",
      "The 4th best search\n",
      "Title : Is a 14-day quarantine period optimal for effectively controlling coronavirus disease 2019 (COVID-19)?\n",
      "Maximum Similiarity of current Document0.13590983\n",
      "The 5th best search\n",
      "Title : Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study\n",
      "Maximum Similiarity of current Document0.1370548\n",
      "The 6th best search\n",
      "Title : Quantifying SARS-CoV-2 transmission suggests epidemic control with digital contact tracing\n",
      "Maximum Similiarity of current Document0.15111834\n",
      "The 7th best search\n",
      "Title : Title: The incubation period of 2019-nCoV infections among travellers from Wuhan, China\n",
      "Maximum Similiarity of current Document0.16413192\n",
      "The 8th best search\n",
      "Title : Community Memory of COVID-19 Infections Post Lockdown as a Surrogate for Incubation Time\n",
      "Maximum Similiarity of current Document0.17724138\n",
      "The 9th best search\n",
      "Title : not Mentioned\n",
      "Maximum Similiarity of current Document0.31352836\n",
      "The 10th best search\n",
      "Title : Clinical Data on Hospital Environmental Hygiene Monitoring and Medical Staff Protection during the Coronavirus Disease 2019 2 Outbreak\n",
      "1\n",
      "Number of documents:4\n",
      "Comparing Result: [0.         0.         0.         ... 0.         0.         0.01395048]\n",
      "19.637815\n",
      "Maximum Similiarity\n",
      "0.20015937\n",
      "[0.1182963  0.12141868 0.12199586 0.12389287 0.13505365 0.13603243\n",
      " 0.15917094 0.16453102 0.1820992  0.20015937]\n",
      "[1911 1711  609  863 1035  461 2108 1168  972  789]\n",
      "Maximum Similiarity of current Document0.1182963\n",
      "The 1st best search\n",
      "Title : BURDEN AND PREVALENCE OF PROGNOSTIC FACTORS FOR SEVERE COVID-19 DISEASE IN SWEDEN\n",
      "Maximum Similiarity of current Document0.12141868\n",
      "The 2nd best search\n",
      "Title : How many are at increased risk of severe COVID-19 disease? Rapid global, regional and national estimates for 2020\n",
      "Maximum Similiarity of current Document0.12199586\n",
      "The 3rd best search\n",
      "Title : Risk assessment of progression to severe conditions for patients with COVID-19 pneumonia: a single-center retrospective study\n",
      "Maximum Similiarity of current Document0.12389287\n",
      "The 4th best search\n",
      "Title : U.S. county-level characteristics to inform equitable COVID-19 response * shared first authorship † shared last authorship\n",
      "Maximum Similiarity of current Document0.13505365\n",
      "The 5th best search\n",
      "Title : Estimating the Risks from COVID-19 Infection in Adult Chemotherapy Patients\n",
      "Maximum Similiarity of current Document0.13603243\n",
      "The 6th best search\n",
      "Title : Testing the association between blood type and COVID-19 infection, intubation, and death\n",
      "Maximum Similiarity of current Document0.15917094\n",
      "The 7th best search\n",
      "Title : History of coronary heart disease increases the mortality rate of COVID-19 patients: a nested case-control study\n",
      "Maximum Similiarity of current Document0.16453102\n",
      "The 8th best search\n",
      "Title : Title: A COVID-19 Risk Assessment for the US Labor Force\n",
      "Maximum Similiarity of current Document0.1820992\n",
      "The 9th best search\n",
      "Title : Modeling risk of infectious diseases: a case of Coronavirus outbreak in four countries\n",
      "Maximum Similiarity of current Document0.20015937\n",
      "The 10th best search\n",
      "Title : The relationship of COVID-19 severity with cardiovascular disease and its traditional risk factors: A systematic review and meta-analysis\n",
      "2\n",
      "Number of documents:5\n",
      "Comparing Result: [0.00085062 0.00437647 0.         ... 0.00373709 0.         0.00016688]\n",
      "7.758667\n",
      "Maximum Similiarity\n",
      "0.10850376\n",
      "[0.04421194 0.04437233 0.04532223 0.05047332 0.0520763  0.05245346\n",
      " 0.05274791 0.06073821 0.08673877 0.10850376]\n",
      "[1391 1400  841 1499  910 1441  329  829 1304  925]\n",
      "Maximum Similiarity of current Document0.04421194\n",
      "The 1st best search\n",
      "Title : Parametric analysis of early data on COVID-19 expansion in selected European countries\n",
      "Maximum Similiarity of current Document0.044372328\n",
      "The 2nd best search\n",
      "Title : ACE2 variants underlie interindividual variability and susceptibility to COVID-19 in Italian population ACE2 gene variants in Italian population\n",
      "Maximum Similiarity of current Document0.04532223\n",
      "The 3rd best search\n",
      "Title : Recombinant rotaviruses rescued by reverse genetics reveal the role of NSP5 hyperphosphorylation in the 2 assembly of viral factories 3 4\n",
      "Maximum Similiarity of current Document0.050473318\n",
      "The 4th best search\n",
      "Title : A minimal and adaptive prediction strategy for critical resource planning in a pandemic\n",
      "Maximum Similiarity of current Document0.052076295\n",
      "The 5th best search\n",
      "Title : Recombinant vector vaccines and within-host evolution 1\n",
      "Maximum Similiarity of current Document0.052453462\n",
      "The 6th best search\n",
      "Title : The 2019-new coronavirus epidemic: evidence for virus evolution\n",
      "Maximum Similiarity of current Document0.052747905\n",
      "The 7th best search\n",
      "Title : The origin and underlying driving forces of the SARS-CoV-2 outbreak Running Title: Evolution at the early stage of SARS-CoV-2 spread\n",
      "Maximum Similiarity of current Document0.060738206\n",
      "The 8th best search\n",
      "Title : A study of SARS-CoV-2 evolution in Italy: from early days to secondary effects after social distancing\n",
      "Maximum Similiarity of current Document0.08673877\n",
      "The 9th best search\n",
      "Title : Modelling the COVID-19 epidemics in Brasil: Parametric identification and public health measures influence\n",
      "Maximum Similiarity of current Document0.10850376\n",
      "The 10th best search\n",
      "Title : Viral long-term evolutionary strategies favor stability 1 over proliferation\n",
      "3\n",
      "Number of documents:6\n",
      "Comparing Result: [0. 0. 0. ... 0. 0. 0.]\n",
      "7.1120257\n",
      "Maximum Similiarity\n",
      "0.21853589\n",
      "[0.07498843 0.07584663 0.07862637 0.08062869 0.08070454 0.10638057\n",
      " 0.11199588 0.12574022 0.13197447 0.21853589]\n",
      "[2158  603 1895 1334 1154 1484 1383 1091 1118   42]\n",
      "Maximum Similiarity of current Document0.07498843\n",
      "The 1st best search\n",
      "Title : The impact of social distancing and epicenter lockdown on the COVID-19 epidemic in mainland China: A data-driven SEIQR model study\n",
      "Maximum Similiarity of current Document0.07584663\n",
      "The 2nd best search\n",
      "Title : A mechanistic population balance model to evaluate the impact of interventions on infectious disease outbreaks: Case for COVID19\n",
      "Maximum Similiarity of current Document0.07862637\n",
      "The 3rd best search\n",
      "Title : The Effectiveness of Social Distancing in Mitigating COVID-19 Spread: a modelling analysis\n",
      "Maximum Similiarity of current Document0.08062869\n",
      "The 4th best search\n",
      "Title : Department of Occupational and Environmental Health\n",
      "Maximum Similiarity of current Document0.08070454\n",
      "The 5th best search\n",
      "Title : The effect of control strategies that reduce social mixing on outcomes of the COVID-19 epidemic in Wuhan, China\n",
      "Maximum Similiarity of current Document0.106380574\n",
      "The 6th best search\n",
      "Title : Patterns of the COVID19 epidemic spread around the world: exponential vs power laws\n",
      "Maximum Similiarity of current Document0.111995876\n",
      "The 7th best search\n",
      "Title : Evaluating the effectiveness of social distancing interventions against COVID-19\n",
      "Maximum Similiarity of current Document0.12574022\n",
      "The 8th best search\n",
      "Title : The effect of non-pharmaceutical interventions on COVID-19 cases, deaths and demand for hospital services in the UK: a modelling study\n",
      "Maximum Similiarity of current Document0.13197447\n",
      "The 9th best search\n",
      "Title : CAN-NPI: A Curated Open Dataset of Canadian Non-Pharmaceutical Interventions in Response to the Global COVID-19 Pandemic\n",
      "Maximum Similiarity of current Document0.21853589\n",
      "The 10th best search\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Title : If long-term suppression is not possible, how do we minimize mortality for COVID-19 and other emerging infectious disease outbreaks?\n",
      "4\n",
      "Number of documents:7\n",
      "Comparing Result: [0.         0.00186851 0.         ... 0.00599675 0.         0.        ]\n",
      "6.157095\n",
      "Maximum Similiarity\n",
      "0.10613852\n",
      "[0.06528234 0.06807995 0.0707794  0.07773875 0.0802798  0.08870413\n",
      " 0.09033341 0.09592824 0.10429777 0.10613852]\n",
      "[ 708 1903 1337 1855  353  492 1041  644  931  538]\n",
      "Maximum Similiarity of current Document0.06528234\n",
      "The 1st best search\n",
      "Title : Shotgun proteomics of SARS-CoV-2 infected cells and its application to the optimisation of whole viral particle antigen production for vaccines\n",
      "Maximum Similiarity of current Document0.06807995\n",
      "The 2nd best search\n",
      "Title : COMPUTATIONAL PREDICTION OF THE COMPREHENSIVE SARS-COV-2 VS. HUMAN INTERACTOME TO GUIDE THE DESIGN OF THERAPEUTICS A PREPRINT -NOT PEER-REVIEWED\n",
      "Maximum Similiarity of current Document0.0707794\n",
      "The 3rd best search\n",
      "Title : Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study 1 2 AUTHORS 3\n",
      "Maximum Similiarity of current Document0.077738754\n",
      "The 4th best search\n",
      "Title : Understand research hotspots surrounding COVID-19 and other coronavirus infections using topic modeling\n",
      "Maximum Similiarity of current Document0.0802798\n",
      "The 5th best search\n",
      "Title : Therapeutic Management of COVID-19 Patients: A systematic review Corresponding Author\n",
      "Maximum Similiarity of current Document0.08870413\n",
      "The 6th best search\n",
      "Title : Bayesian Adaptive Clinical Trials for Anti-Infective Therapeutics during Epidemic Outbreaks *\n",
      "Maximum Similiarity of current Document0.09033341\n",
      "The 7th best search\n",
      "Title : The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19\n",
      "Maximum Similiarity of current Document0.09592824\n",
      "The 8th best search\n",
      "Title : Financing Vaccines for Global Health Security\n",
      "Maximum Similiarity of current Document0.10429777\n",
      "The 9th best search\n",
      "Title : Estimating Probabilities of Success of Clinical Trials for Vaccines and Other Anti-Infective Therapeutics 1\n",
      "Maximum Similiarity of current Document0.10613852\n",
      "The 10th best search\n",
      "Title : More than efficacy revealed by single-cell analysis of antiviral therapeutics\n",
      "5\n",
      "Number of documents:8\n",
      "Comparing Result: [0.         0.01257118 0.         ... 0.00349872 0.         0.        ]\n",
      "9.9341\n",
      "Maximum Similiarity\n",
      "0.09164327\n",
      "[0.0624399  0.06291097 0.06332081 0.06580196 0.06844728 0.07283895\n",
      " 0.08342224 0.08581458 0.09128709 0.09164327]\n",
      "[1084 1552  915 1390 1249 2000 1431  970  193 1705]\n",
      "Maximum Similiarity of current Document0.0624399\n",
      "The 1st best search\n",
      "Title : Social distancing strategies for curbing the COVID-19 epidemic\n",
      "Maximum Similiarity of current Document0.062910974\n",
      "The 2nd best search\n",
      "Title : Mathematical modeling of COVID-19 transmission and mitigation strategies in the population of Ontario, Canada\n",
      "Maximum Similiarity of current Document0.06332081\n",
      "The 3rd best search\n",
      "Title : Which Measures are Effective in Containing COVID-19? --Empirical Research Based on Prevention and Control Cases in China\n",
      "Maximum Similiarity of current Document0.06580196\n",
      "The 4th best search\n",
      "Title : SOCRATES: An online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19\n",
      "Maximum Similiarity of current Document0.06844728\n",
      "The 5th best search\n",
      "Title : A Social Network Model of the COVID-19 Pandemic\n",
      "Maximum Similiarity of current Document0.07283895\n",
      "The 6th best search\n",
      "Title : Manchester Academic Health Science Centre, Manchester, M13 9PL 5 World Health Organisation\n",
      "Maximum Similiarity of current Document0.08342224\n",
      "The 7th best search\n",
      "Title : Social Isolation and Mortality Social Isolation as a predictor for mortality: Implications for COVID-19 prognosis\n",
      "Maximum Similiarity of current Document0.08581458\n",
      "The 8th best search\n",
      "Title : Mathematical modeling of COVID-19 containment strategies with considerations for limited medical resources\n",
      "Maximum Similiarity of current Document0.09128709\n",
      "The 9th best search\n",
      "Title : not Mentioned\n",
      "Maximum Similiarity of current Document0.09164327\n",
      "The 10th best search\n",
      "Title : Optimal timing for social distancing during an epidemic\n",
      "6\n",
      "Number of documents:9\n",
      "Comparing Result: [0.         0.00186423 0.         ... 0.00465674 0.         0.        ]\n",
      "8.630498\n",
      "Maximum Similiarity\n",
      "0.28283948\n",
      "[0.07856452 0.07977203 0.09504896 0.0953633  0.11074258 0.1170142\n",
      " 0.11909722 0.13446721 0.20330507 0.28283948]\n",
      "[ 120  145 1570  176  494  217   81 1611  784  785]\n",
      "Maximum Similiarity of current Document0.078564525\n",
      "The 1st best search\n",
      "Title : Understanding the B and T cells epitopes of spike protein of severe respiratory syndrome 1 coronavirus-2: A computational way to predict the immunogens\n",
      "Maximum Similiarity of current Document0.07977203\n",
      "The 2nd best search\n",
      "Title : Surveillance by age-class and prefecture for emerging infectious febrile diseases with respiratory symptoms, including COVID-19\n",
      "Maximum Similiarity of current Document0.095048964\n",
      "The 3rd best search\n",
      "Title : A model to predict SARS-CoV-2 infection based on the first three- month surveillance data in Brazil\n",
      "Maximum Similiarity of current Document0.0953633\n",
      "The 4th best search\n",
      "Title : not Mentioned\n",
      "Maximum Similiarity of current Document0.110742584\n",
      "The 5th best search\n",
      "Title : CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design\n",
      "Maximum Similiarity of current Document0.1170142\n",
      "The 6th best search\n",
      "Title : Clinical evaluation of a SARS-CoV-2 RT-PCR assay on a fully automated system 1 for rapid on-demand testing in the hospital setting\n",
      "Maximum Similiarity of current Document0.11909722\n",
      "The 7th best search\n",
      "Title : Massive and rapid COVID-19 testing is feasible by extraction- free SARS-CoV-2 RT-qPCR\n",
      "Maximum Similiarity of current Document0.13446721\n",
      "The 8th best search\n",
      "Title : not Mentioned\n",
      "Maximum Similiarity of current Document0.20330507\n",
      "The 9th best search\n",
      "Title : Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts\n",
      "Maximum Similiarity of current Document0.28283948\n",
      "The 10th best search\n",
      "Title : Towards a simulation framework for optimizing infectious disease surveillance: An information theoretic approach for surveillance 2 system design 3 4\n",
      "7\n",
      "Number of documents:10\n",
      "Comparing Result: [0.         0.         0.         ... 0.00018825 0.         0.        ]\n",
      "19.334593\n",
      "Maximum Similiarity\n",
      "0.229061\n",
      "[0.14132077 0.14190473 0.14311564 0.14786948 0.16482048 0.16874427\n",
      " 0.17192052 0.18304916 0.21029364 0.229061  ]\n",
      "[ 444  882 1536  487  583 1084  224  593 1370 1128]\n",
      "Maximum Similiarity of current Document0.14132077\n",
      "The 1st best search\n",
      "Title : Preventing respiratory illness in older adults aged 60 years and above living in long-term care A rapid overview of reviews\n",
      "Maximum Similiarity of current Document0.14190473\n",
      "The 2nd best search\n",
      "Title : Using ICU data to improve the real-time estimation of the effective reproductive number of the COVID-19 epidemic in 9 European countries\n",
      "Maximum Similiarity of current Document0.14311564\n",
      "The 3rd best search\n",
      "Title : Confronting COVID-19: Surging critical care capacity in Italy\n",
      "Maximum Similiarity of current Document0.14786948\n",
      "The 4th best search\n",
      "Title : \"The more I fear about COVID-19, the more I wear medical masks\": A survey on risk perception and medical masks' uses\n",
      "Maximum Similiarity of current Document0.16482048\n",
      "The 5th best search\n",
      "Title : Title: Effects of medical resource capacities and intensities of public 1 mitigation measures on outcomes of COVID-19 outbreaks 2\n",
      "Maximum Similiarity of current Document0.16874427\n",
      "The 6th best search\n",
      "Title : Social distancing strategies for curbing the COVID-19 epidemic\n",
      "Maximum Similiarity of current Document0.17192052\n",
      "The 7th best search\n",
      "Title : Study of the mental health status of medical personnel dealing with new coronavirus pneumonia\n",
      "Maximum Similiarity of current Document0.18304916\n",
      "The 8th best search\n",
      "Title : Estimating the maximum daily number of incident COVID-19 cases manageable by a healthcare system\n",
      "Maximum Similiarity of current Document0.21029364\n",
      "The 9th best search\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Title : Epidemiological, Clinical Characteristics and Outcome of Medical Staff Infected with COVID-19 in Wuhan, China: A Retrospective Case Series Analysis\n",
      "Maximum Similiarity of current Document0.229061\n",
      "The 10th best search\n",
      "Title : Flattening the curve before it flattens us: hospital critical care capacity limits and mortality from novel coronavirus (SARS-CoV2) cases in US counties\n",
      "8\n",
      "Number of documents:11\n",
      "Comparing Result: [0.         0.00505337 0.00230032 ... 0.00383356 0.         0.        ]\n",
      "5.5600777\n",
      "Maximum Similiarity\n",
      "0.22963426\n",
      "[0.04037378 0.04175462 0.0477051  0.04829083 0.04935154 0.0546926\n",
      " 0.07486511 0.08427481 0.14867748 0.22963426]\n",
      "[ 731  903  724 1276  577  820  131  311 1270  665]\n",
      "Maximum Similiarity of current Document0.040373784\n",
      "The 1st best search\n",
      "Title : Differential ventilation using flow control valves as a potential bridge to full ventilatory support during the COVID-19 crisis: From bench to bedside\n",
      "Maximum Similiarity of current Document0.04175462\n",
      "The 2nd best search\n",
      "Title : Title page Running Title: Health Education for Parents During the COVID-19 Outbreak Public Health Education for Parents During the Outbreak of COVID-19: A Rapid Review\n",
      "Maximum Similiarity of current Document0.047705103\n",
      "The 3rd best search\n",
      "Title : International Electronic Health Record-Derived COVID-19 Clinical Course Profiles: The 4CE Consortium\n",
      "Maximum Similiarity of current Document0.048290834\n",
      "The 4th best search\n",
      "Title : Systematic review and critical appraisal of prediction models for diagnosis and prognosis of COVID-19 infection\n",
      "Maximum Similiarity of current Document0.04935154\n",
      "The 5th best search\n",
      "Title : Building an International Consortium for Tracking Coronavirus Health Status\n",
      "Maximum Similiarity of current Document0.0546926\n",
      "The 6th best search\n",
      "Title : 2 A systematic review and evaluation of Zika virus forecasting and prediction research during a 3 public health emergency of international concern 4 5 Kobres P\n",
      "Maximum Similiarity of current Document0.07486511\n",
      "The 7th best search\n",
      "Title : Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype 1 sharing in a Neotropical bat community 2 3\n",
      "Maximum Similiarity of current Document0.08427481\n",
      "The 8th best search\n",
      "Title : Distinct spread of DNA and RNA viruses among mammals amid prominent role of domestic species\n",
      "Maximum Similiarity of current Document0.14867748\n",
      "The 9th best search\n",
      "Title : A flexible load sharing system optimising ICU demand in the context of COVID-19 pandemic\n",
      "Maximum Similiarity of current Document0.22963426\n",
      "The 10th best search\n",
      "Title : Predicting the global mammalian viral 1 sharing network using phylogeography\n",
      "FAQ HERE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "127.0.0.1 - - [14/May/2020 01:32:33] \"GET /faq HTTP/1.1\" 200 -\n"
     ]
    }
   ],
   "source": [
    "from flask import Flask, render_template, request, redirect, url_for\n",
    "\n",
    "tasks=['transmission, incubation, and environmental stability',\n",
    "       'COVID-19 risk factors', \n",
    "       'virus genetics, origin, and evolution', \n",
    "       'non-pharmaceutical interventions',\n",
    "       'vaccines and therapeutics', \n",
    "       'ethical and social science considerations', \n",
    "       'diagnostics and surveillance',\n",
    "       'medical care', \n",
    "       'information sharing and inter-sectoral collaboration']\n",
    "#print(len(tasks))\n",
    "   \n",
    "def GetArticle(question):\n",
    "    question=question\n",
    "    tokens = sent_tokenize(question)\n",
    "    for line in tokens:\n",
    "        file2_docs.append(line)\n",
    "    for line in file2_docs:\n",
    "        query_doc = [w.lower() for w in word_tokenize(line)]\n",
    "        query_doc_bow = dictionary.doc2bow(query_doc) #update an existing dictionary andcreate bag of words\n",
    "    print(\"Number of documents:\"+str(len(file2_docs)))\n",
    "    query_doc_tf_idf = tf_idf[query_doc_bow]\n",
    "    # print(document_number, document_similarity)\n",
    "    print('Comparing Result:', sims[query_doc_tf_idf])\n",
    "    sum_of_sims =(np.sum(sims[query_doc_tf_idf], dtype=np.float32))\n",
    "    print(sum_of_sims)\n",
    "    #the maximum value\n",
    "    print(\"Maximum Similiarity\")\n",
    "    print(np.amax(sims[query_doc_tf_idf]))\n",
    "    #the top 10 similarities\n",
    "    print (sims[query_doc_tf_idf][np.argsort(sims[query_doc_tf_idf])[-10:]])\n",
    "    #return the id of the top 10 similarities\n",
    "    similartiesArray=sims[query_doc_tf_idf][np.argsort(sims[query_doc_tf_idf])[-10:]];\n",
    "    print( np.argsort(sims[query_doc_tf_idf])[-10:])\n",
    "    #return the tite and the fullbody documents of the top 10 similarities\n",
    "    ordinal = lambda n: \"%d%s\" % (n,\"tsnrhtdd\"[(n/10%10!=1)*(n%10<4)*n%10::4])\n",
    "    arrFinal = dfinal.to_numpy()\n",
    "    arrayDocuementsid  = np.argsort(sims[query_doc_tf_idf])[-10:]\n",
    "    Top10=[]\n",
    "    for i in range (len(arrayDocuementsid)):\n",
    "        print('Maximum Similiarity of current Document'+str(similartiesArray[i]));\n",
    "        index = arrayDocuementsid[i]\n",
    "        print (\"The \" + ordinal(i+1) + \" best search\" )\n",
    "        if(arrFinal[index][0]==''):\n",
    "            print(\"Title : not Mentioned\" + arrFinal[index][0])\n",
    "            title=\"Title not Mentioned\" + str(arrFinal[index][0])\n",
    "        else:\n",
    "            print(\"Title : \" + arrFinal[index][0])\n",
    "            title=arrFinal[index][0]\n",
    "        Top10.append([arrFinal[index][2],title,similartiesArray[i]])\n",
    "    return Top10\n",
    "def MultipleQuestions():\n",
    "    AnswersArray=[]\n",
    "    for i in range (0,len(tasks)):\n",
    "        print(i)\n",
    "        Answer=GetArticle(tasks[i])[9];\n",
    "        AnswersArray.append(Answer)\n",
    "    return (AnswersArray);\n",
    "\n",
    "app = Flask(__name__)\n",
    "@app.route('/')\n",
    "def index():\n",
    "    return render_template(\"index.html\")\n",
    "\n",
    "@app.route('/', methods=['POST'])\n",
    "def my_form_post():\n",
    "    text = request.form['text']\n",
    "    Answers =GetArticle(text)\n",
    "    article1,title1,similiarity1=Answers[9]\n",
    "    article2,title2,similiarity2=Answers[8]\n",
    "    article3,title3,similiarity3=Answers[7]\n",
    "    article4,title4,similiarity4=Answers[6]\n",
    "    article5,title5,similiarity5=Answers[5]\n",
    "    article6,title6,similiarity6=Answers[4]\n",
    "    article7,title7,similiarity7=Answers[3]\n",
    "    article8,title8,similiarity8=Answers[2]\n",
    "    article9,title9,similiarity9=Answers[1]\n",
    "    article10,title10,similiarity10=Answers[0]\n",
    "\n",
    "    return render_template(\"answer/index.html\",Question=text\n",
    "                           ,article1=article1,title1=title1,similiarity1=similiarity1\n",
    "                           ,article2=article2,title2=title2,similiarity2=similiarity2\n",
    "                          ,article3=article3,title3=title3,similiarity3=similiarity3\n",
    "                          ,article4=article4,title4=title4,similiarity4=similiarity4\n",
    "                          ,article5=article5,title5=title5,similiarity5=similiarity5\n",
    "                          ,article7=article7,title7=title7,similiarity7=similiarity7\n",
    "                          ,article6=article6,title6=title6,similiarity6=similiarity6\n",
    "                          ,article8=article8,title8=title8,similiarity8=similiarity8\n",
    "                          ,article9=article9,title9=title9,similiarity9=similiarity9\n",
    "                           ,article10=article10,title10=title10,similiarity10=similiarity10\n",
    "                          \n",
    "                          \n",
    "                          )\n",
    "\n",
    "@app.route('/faq')\n",
    "def faq():\n",
    "    Answers =MultipleQuestions(); \n",
    "    article1,title1,similiarity1=Answers[0]\n",
    "    article2,title2,similiarity2=Answers[1]\n",
    "    article3,title3,similiarity3=Answers[2]\n",
    "    article4,title4,similiarity4=Answers[3]\n",
    "    article5,title5,similiarity5=Answers[4]\n",
    "    article6,title6,similiarity6=Answers[5]\n",
    "    article7,title7,similiarity7=Answers[6]\n",
    "    article8,title8,similiarity8=Answers[7]\n",
    "    article9,title9,similiarity9=Answers[8]\n",
    "\n",
    "    print(\"FAQ HERE\")\n",
    "    #print(similiarity1)\n",
    "    return render_template(\"faq/index.html\",\n",
    "                           article1=article1,title1=title1\n",
    "                           ,article2=article2,title2=title2\n",
    "                          ,article3=article3,title3=title3\n",
    "                          ,article4=article4,title4=title4\n",
    "                          ,article5=article5,title5=title5\n",
    "                          ,article7=article7,title7=title7\n",
    "                          ,article6=article6,title6=title6\n",
    "                          ,article8=article8,title8=title8\n",
    "                          ,article9=article9,title9=title9)\n",
    " \n",
    "\n",
    "if __name__ == '__main__':\n",
    "    app.run(debug=False)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
